{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "f9b2506d",
   "metadata": {},
   "source": [
    "# BioMed: Information Retrieval - BioMedical Information Retrieval System\n",
    "\n",
    "---\n",
    "\n",
    "**Group:**\n",
    "- Reyes Castro, Didier Yamil (didier.reyes.castro@alumnos.upm.es)\n",
    "- Rodriguez Fernández, Cristina ()\n",
    "\n",
    "**Course:** BioMedical Informatics - 2025/26\n",
    "\n",
    "**Institution:** Polytechnic University of Madrid (UPM)\n",
    "\n",
    "**Date:** November 2026\n",
    "\n",
    "---\n",
    "\n",
    "## Goal\n",
    "\n",
    "To develop an Information Retrieval system — specifically, a **binary text classifier** — to identify scientific articles in the PubMed database that are related to a given set of abstracts within a defined research topic. In this case, the focus is on a collection of 1,308 manuscripts containing information on the polyphenol composition of various foods."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2ce86cc1",
   "metadata": {},
   "source": [
    "## Setup and Installation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "5e05604d",
   "metadata": {},
   "outputs": [],
   "source": [
    "# !pip install scikit-learn pandas requests transformers pytorch datasets numpy"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "3aa50003",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/didierrc/miniconda3/envs/biomed_ir/lib/python3.13/site-packages/tqdm/auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
      "  from .autonotebook import tqdm as notebook_tqdm\n"
     ]
    }
   ],
   "source": [
    "import requests\n",
    "import time\n",
    "import torch\n",
    "import pandas as pd\n",
    "from transformers import AutoTokenizer, AutoModelForSequenceClassification, TrainingArguments, Trainer, EarlyStoppingCallback\n",
    "from datasets import Dataset\n",
    "import numpy as np\n",
    "from sklearn.model_selection import train_test_split\n",
    "from sklearn.metrics import accuracy_score, precision_recall_fscore_support, confusion_matrix, classification_report"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "e505653b",
   "metadata": {},
   "outputs": [],
   "source": [
    "BASE_URL = \"https://eutils.ncbi.nlm.nih.gov/entrez/eutils/\"\n",
    "ESEARCH_URL = BASE_URL + \"esearch.fcgi\"\n",
    "FETCH_URL = BASE_URL + \"efetch.fcgi\"\n",
    "DS_WITH_PMID = 'publications_pmid.csv'\n",
    "PMID_ABSTRACTS = 'publications_pmid_abstract.csv'\n",
    "ELSEVIER_SEARCH_URL = \"https://api.elsevier.com/content/search/scopus\"\n",
    "ELSEVIER_ABSTRACTS = 'publications_abstract_pubmed_elsevier.csv'\n",
    "IRRELEVANT_PUBLICATIONS = 'irrelevant_publications.csv'\n",
    "COMBINED_PUBLICATIONS = 'combined_publications.csv'\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ad6641b8",
   "metadata": {},
   "source": [
    "## **Task 1:** \n",
    "\n",
    "Retrieve from PubMed the abstracts associated with each publication in publications.xlsx\n",
    "\n",
    "(21 minutes with API KEY)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "413867ee",
   "metadata": {},
   "outputs": [],
   "source": [
    "dataset = pd.read_csv('publications.csv')\n",
    "dataset"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "7f7c3c61",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Step 1: Search for the PMID of the articles by title\n",
    "def search_pmid(article):\n",
    "    \n",
    "    title = article['title']\n",
    "    params = {\n",
    "        \"db\": \"pubmed\",\n",
    "        \"term\": f\"{title}\",\n",
    "        \"retmode\": \"json\",\n",
    "        \"field\": \"title\"\n",
    "    }\n",
    "\n",
    "    try:\n",
    "\n",
    "        # Trying to find the PMID\n",
    "        response = requests.get(ESEARCH_URL, params=params)\n",
    "        response.raise_for_status()\n",
    "        data = response.json()\n",
    "\n",
    "        if len(data[\"esearchresult\"][\"idlist\"]) >= 1:\n",
    "            pmid = data['esearchresult']['idlist'][0]\n",
    "            print(f\"> Found PMID for article: {pmid}\")\n",
    "            return pmid\n",
    "        \n",
    "        print(f\"> No PMID found for article.\")\n",
    "        return None\n",
    "\n",
    "    except requests.exceptions.RequestException as e:\n",
    "        print(f\"> ERROR during request for article: {e}\")\n",
    "        return None\n",
    "    \n",
    "ds_pmid = dataset.copy()\n",
    "for idx, article in ds_pmid.iterrows():\n",
    "    print(f\"[{idx + 1}/{len(ds_pmid)}] Searching PMID for: {article['title']}\")\n",
    "    pmid = search_pmid(article)\n",
    "    ds_pmid.at[idx, 'pmid'] = pmid\n",
    "\n",
    "ds_pmid.to_csv(DS_WITH_PMID, index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "7a4cf173",
   "metadata": {},
   "outputs": [],
   "source": [
    "print(\"Number of articles with PMID found:\", ds_pmid['pmid'].notnull().sum())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "447a6a14",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Step 2: Fetch article abstract by PMID\n",
    "def fetch_abstract_by_pmid(pmid):\n",
    "    params = {\n",
    "        \"db\": \"pubmed\",\n",
    "        \"id\": f\"{pmid}\",\n",
    "        \"retmode\": \"text\",\n",
    "        \"rettype\": \"abstract\",\n",
    "    }\n",
    "\n",
    "    try:\n",
    "        response = requests.get(FETCH_URL, params=params)\n",
    "        response.raise_for_status()\n",
    "        print(f\"> Fetched abstract!!\")\n",
    "        return response.text\n",
    "    except requests.exceptions.RequestException as e:\n",
    "        print(f\"> ERROR fetching abstract for PMID '{pmid}': {e}\")\n",
    "        return None\n",
    "    \n",
    "ds_pmid_abstract = ds_pmid.copy()\n",
    "for idx, article in ds_pmid_abstract.iterrows():\n",
    "    pmid = article['pmid']\n",
    "    if pd.notnull(pmid):\n",
    "        print(f\"[{idx + 1}/{len(ds_pmid_abstract)}] Fetching abstract for PMID: {pmid}\")\n",
    "        abstract = fetch_abstract_by_pmid(pmid)\n",
    "        ds_pmid_abstract.at[idx, 'abstract'] = abstract\n",
    "    else:\n",
    "        print(f\"[{idx + 1}/{len(ds_pmid_abstract)}] No PMID available, skipping abstract fetch.\")\n",
    "        ds_pmid_abstract.at[idx, 'abstract'] = None"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "ab6b2b87",
   "metadata": {},
   "outputs": [],
   "source": [
    "ds_pmid_abstract.to_csv(PMID_ABSTRACTS, index=False)\n",
    "print(\"Number of articles with abstract fetched:\", ds_pmid_abstract['abstract'].notnull().sum())\n",
    "print(\"Number of articles without abstract fetched:\", ds_pmid_abstract['abstract'].isnull().sum())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "38cfa3cf",
   "metadata": {},
   "outputs": [],
   "source": [
    "def search_scopus(article_row, api_key):\n",
    "\n",
    "    try:\n",
    "\n",
    "        title = article_row.get('title', '').replace('\"', '') # Remove quotes for query\n",
    "        headers = {\"Accept\": \"application/json\"}\n",
    "        params = {\n",
    "            \"query\": f\"TITLE-ABS-KEY(\\\"{title}\\\")\",\n",
    "            \"apiKey\": api_key\n",
    "        }\n",
    "        \n",
    "        response = requests.get(ELSEVIER_SEARCH_URL, \n",
    "                                headers=headers, params=params, timeout=10)\n",
    "        response.raise_for_status() \n",
    "        data = response.json()\n",
    "\n",
    "        if 'search-results' in data and data['search-results']['entry']:\n",
    "            try:\n",
    "                return data['search-results']['entry'][0]['prism:url']\n",
    "            except KeyError:\n",
    "                return None\n",
    "        return None\n",
    "\n",
    "    except requests.exceptions.RequestException as e:\n",
    "        print(f\"> Scopus ERROR: {e}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "65686071",
   "metadata": {},
   "outputs": [],
   "source": [
    "ds_elsevier = pd.read_csv(PMID_ABSTRACTS)\n",
    "\n",
    "counters = {\n",
    "    'total_missing': ds_elsevier['abstract'].isnull().sum(),\n",
    "    'elsevier_found': 0,\n",
    "    'failed': 0\n",
    "}\n",
    "\n",
    "for i, row in ds_elsevier.iterrows():\n",
    "\n",
    "    if not pd.isnull(row['abstract']):\n",
    "        continue\n",
    "\n",
    "    print(f\"[{i + 1}/{len(ds_elsevier)}] Searching ELSEVIER for abstract of article: {row['title']}\")\n",
    "    abstract_url = search_scopus(row, \"3f5ff36eb8d1d409e3befea2ed2aa2cc\")\n",
    "    \n",
    "    if abstract_url:\n",
    "        response = requests.get(abstract_url, \n",
    "                                headers={\"Accept\": \"application/json\", \n",
    "                                         \"X-ELS-APIKey\": \"3f5ff36eb8d1d409e3befea2ed2aa2cc\"})\n",
    "        if response.status_code == 200:\n",
    "            data = response.json()\n",
    "            try:\n",
    "                abstract_text = data['abstracts-retrieval-response']['coredata']['dc:description']\n",
    "                ds_elsevier.at[i, 'abstract'] = abstract_text\n",
    "                print(\"> Found abstract via ELSEVIER!\")\n",
    "                counters['elsevier_found'] += 1\n",
    "            except KeyError:\n",
    "                print(\"> Abstract not found in ELSEVIER response.\")\n",
    "                counters['failed'] += 1\n",
    "    else:\n",
    "        print(\"> Nope :(\")\n",
    "        counters['failed'] += 1\n",
    "\n",
    "    time.sleep(1) # Polite 1-second delay\n",
    "\n",
    "ds_elsevier.to_csv(ELSEVIER_ABSTRACTS, index=False)\n",
    "print(\"Summary of ELSEVIER abstract search:\")\n",
    "print(f\"Total missing abstracts at start: {counters['total_missing']}\")\n",
    "print(f\"Abstracts found via ELSEVIER: {counters['elsevier_found']}\")\n",
    "print(f\"Failed attempts: {counters['failed']}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a56b780b",
   "metadata": {},
   "source": [
    "## **Task 2:**\n",
    "\n",
    "Use the EUtilities tool to search for articles whose content is not relevant to this task. Size of the dataset should be the same of relevant documents."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "2c625aab",
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_articles_pmids_for_title(title, count):\n",
    "    \n",
    "    params = {\n",
    "        \"db\": \"pubmed\",\n",
    "        \"term\": f\"{title}[tiab]\",\n",
    "        \"retmode\": \"json\",\n",
    "        \"retmax\": count\n",
    "    }\n",
    "\n",
    "    try:\n",
    "        response = requests.get(ESEARCH_URL, params=params)\n",
    "        response.raise_for_status()\n",
    "        data = response.json()\n",
    "\n",
    "        if 'esearchresult' in data and data['esearchresult']['count'] != '0':\n",
    "            print(f\"Found {data['esearchresult']['count']} articles.\")\n",
    "            return data['esearchresult']['idlist']\n",
    "        else:\n",
    "            print(f\"Found {data['esearchresult']['count']} irrelevant articles.\")\n",
    "            return []\n",
    "\n",
    "    except requests.exceptions.RequestException as e:\n",
    "        print(f\"Error during request for irrelevant articles: {e}\")\n",
    "        return []\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "84a0f689",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found 2529778 articles.\n",
      "[1/1203] Fetching abstract for PMID: 41204822\n",
      "> Fetched abstract!!\n",
      "[2/1203] Fetching abstract for PMID: 41204804\n",
      "> Fetched abstract!!\n",
      "[3/1203] Fetching abstract for PMID: 41204791\n",
      "> Fetched abstract!!\n",
      "[4/1203] Fetching abstract for PMID: 41204774\n",
      "> Fetched abstract!!\n",
      "[5/1203] Fetching abstract for PMID: 41204772\n",
      "> Fetched abstract!!\n",
      "[6/1203] Fetching abstract for PMID: 41204766\n",
      "> Fetched abstract!!\n",
      "[7/1203] Fetching abstract for PMID: 41204749\n",
      "> Fetched abstract!!\n",
      "[8/1203] Fetching abstract for PMID: 41204725\n",
      "> Fetched abstract!!\n",
      "[9/1203] Fetching abstract for PMID: 41204722\n",
      "> Fetched abstract!!\n",
      "[10/1203] Fetching abstract for PMID: 41204717\n",
      "> Fetched abstract!!\n",
      "[11/1203] Fetching abstract for PMID: 41204712\n",
      "> Fetched abstract!!\n",
      "[12/1203] Fetching abstract for PMID: 41204710\n",
      "> Fetched abstract!!\n",
      "[13/1203] Fetching abstract for PMID: 41204697\n",
      "> Fetched abstract!!\n",
      "[14/1203] Fetching abstract for PMID: 41204684\n",
      "> Fetched abstract!!\n",
      "[15/1203] Fetching abstract for PMID: 41204682\n",
      "> Fetched abstract!!\n",
      "[16/1203] Fetching abstract for PMID: 41204681\n",
      "> Fetched abstract!!\n",
      "[17/1203] Fetching abstract for PMID: 41204677\n",
      "> Fetched abstract!!\n",
      "[18/1203] Fetching abstract for PMID: 41204675\n",
      "> Fetched abstract!!\n",
      "[19/1203] Fetching abstract for PMID: 41204672\n",
      "> Fetched abstract!!\n",
      "[20/1203] Fetching abstract for PMID: 41204580\n",
      "> Fetched abstract!!\n",
      "[21/1203] Fetching abstract for PMID: 41204566\n",
      "> Fetched abstract!!\n",
      "[22/1203] Fetching abstract for PMID: 41204558\n",
      "> Fetched abstract!!\n",
      "[23/1203] Fetching abstract for PMID: 41204557\n",
      "> Fetched abstract!!\n",
      "[24/1203] Fetching abstract for PMID: 41204550\n",
      "> Fetched abstract!!\n",
      "[25/1203] Fetching abstract for PMID: 41204543\n",
      "> Fetched abstract!!\n",
      "[26/1203] Fetching abstract for PMID: 41204540\n",
      "> Fetched abstract!!\n",
      "[27/1203] Fetching abstract for PMID: 41204525\n",
      "> Fetched abstract!!\n",
      "[28/1203] Fetching abstract for PMID: 41204522\n",
      "> Fetched abstract!!\n",
      "[29/1203] Fetching abstract for PMID: 41204520\n",
      "> Fetched abstract!!\n",
      "[30/1203] Fetching abstract for PMID: 41204513\n",
      "> Fetched abstract!!\n",
      "[31/1203] Fetching abstract for PMID: 41204507\n",
      "> Fetched abstract!!\n",
      "[32/1203] Fetching abstract for PMID: 41204503\n",
      "> Fetched abstract!!\n",
      "[33/1203] Fetching abstract for PMID: 41204490\n",
      "> Fetched abstract!!\n",
      "[34/1203] Fetching abstract for PMID: 41204487\n",
      "> Fetched abstract!!\n",
      "[35/1203] Fetching abstract for PMID: 41204484\n",
      "> Fetched abstract!!\n",
      "[36/1203] Fetching abstract for PMID: 41204478\n",
      "> Fetched abstract!!\n",
      "[37/1203] Fetching abstract for PMID: 41204475\n",
      "> Fetched abstract!!\n",
      "[38/1203] Fetching abstract for PMID: 41204473\n",
      "> Fetched abstract!!\n",
      "[39/1203] Fetching abstract for PMID: 41204468\n",
      "> Fetched abstract!!\n",
      "[40/1203] Fetching abstract for PMID: 41204467\n",
      "> Fetched abstract!!\n",
      "[41/1203] Fetching abstract for PMID: 41204466\n",
      "> Fetched abstract!!\n",
      "[42/1203] Fetching abstract for PMID: 41204462\n",
      "> Fetched abstract!!\n",
      "[43/1203] Fetching abstract for PMID: 41204446\n",
      "> Fetched abstract!!\n",
      "[44/1203] Fetching abstract for PMID: 41204424\n",
      "> Fetched abstract!!\n",
      "[45/1203] Fetching abstract for PMID: 41204416\n",
      "> Fetched abstract!!\n",
      "[46/1203] Fetching abstract for PMID: 41204401\n",
      "> Fetched abstract!!\n",
      "[47/1203] Fetching abstract for PMID: 41204391\n",
      "> Fetched abstract!!\n",
      "[48/1203] Fetching abstract for PMID: 41204384\n",
      "> Fetched abstract!!\n",
      "[49/1203] Fetching abstract for PMID: 41204349\n",
      "> Fetched abstract!!\n",
      "[50/1203] Fetching abstract for PMID: 41204345\n",
      "> Fetched abstract!!\n",
      "[51/1203] Fetching abstract for PMID: 41204330\n",
      "> Fetched abstract!!\n",
      "[52/1203] Fetching abstract for PMID: 41204325\n",
      "> Fetched abstract!!\n",
      "[53/1203] Fetching abstract for PMID: 41204322\n",
      "> Fetched abstract!!\n",
      "[54/1203] Fetching abstract for PMID: 41204315\n",
      "> Fetched abstract!!\n",
      "[55/1203] Fetching abstract for PMID: 41204308\n",
      "> Fetched abstract!!\n",
      "[56/1203] Fetching abstract for PMID: 41204305\n",
      "> Fetched abstract!!\n",
      "[57/1203] Fetching abstract for PMID: 41204294\n",
      "> Fetched abstract!!\n",
      "[58/1203] Fetching abstract for PMID: 41204284\n",
      "> Fetched abstract!!\n",
      "[59/1203] Fetching abstract for PMID: 41204256\n",
      "> Fetched abstract!!\n",
      "[60/1203] Fetching abstract for PMID: 41204224\n",
      "> Fetched abstract!!\n",
      "[61/1203] Fetching abstract for PMID: 41204206\n",
      "> Fetched abstract!!\n",
      "[62/1203] Fetching abstract for PMID: 41204195\n",
      "> Fetched abstract!!\n",
      "[63/1203] Fetching abstract for PMID: 41204193\n",
      "> Fetched abstract!!\n",
      "[64/1203] Fetching abstract for PMID: 41204180\n",
      "> Fetched abstract!!\n",
      "[65/1203] Fetching abstract for PMID: 41204178\n",
      "> Fetched abstract!!\n",
      "[66/1203] Fetching abstract for PMID: 41204154\n",
      "> Fetched abstract!!\n",
      "[67/1203] Fetching abstract for PMID: 41204125\n",
      "> Fetched abstract!!\n",
      "[68/1203] Fetching abstract for PMID: 41204120\n",
      "> Fetched abstract!!\n",
      "[69/1203] Fetching abstract for PMID: 41204115\n",
      "> Fetched abstract!!\n",
      "[70/1203] Fetching abstract for PMID: 41204108\n",
      "> Fetched abstract!!\n",
      "[71/1203] Fetching abstract for PMID: 41204095\n",
      "> Fetched abstract!!\n",
      "[72/1203] Fetching abstract for PMID: 41204069\n",
      "> Fetched abstract!!\n",
      "[73/1203] Fetching abstract for PMID: 41204051\n",
      "> Fetched abstract!!\n",
      "[74/1203] Fetching abstract for PMID: 41204047\n",
      "> Fetched abstract!!\n",
      "[75/1203] Fetching abstract for PMID: 41204038\n",
      "> Fetched abstract!!\n",
      "[76/1203] Fetching abstract for PMID: 41204036\n",
      "> Fetched abstract!!\n",
      "[77/1203] Fetching abstract for PMID: 41204028\n",
      "> Fetched abstract!!\n",
      "[78/1203] Fetching abstract for PMID: 41204014\n",
      "> Fetched abstract!!\n",
      "[79/1203] Fetching abstract for PMID: 41204013\n",
      "> Fetched abstract!!\n",
      "[80/1203] Fetching abstract for PMID: 41204010\n",
      "> Fetched abstract!!\n",
      "[81/1203] Fetching abstract for PMID: 41204005\n",
      "> Fetched abstract!!\n",
      "[82/1203] Fetching abstract for PMID: 41204004\n",
      "> Fetched abstract!!\n",
      "[83/1203] Fetching abstract for PMID: 41203997\n",
      "> Fetched abstract!!\n",
      "[84/1203] Fetching abstract for PMID: 41203996\n",
      "> Fetched abstract!!\n",
      "[85/1203] Fetching abstract for PMID: 41203982\n",
      "> Fetched abstract!!\n",
      "[86/1203] Fetching abstract for PMID: 41203980\n",
      "> Fetched abstract!!\n",
      "[87/1203] Fetching abstract for PMID: 41203958\n",
      "> Fetched abstract!!\n",
      "[88/1203] Fetching abstract for PMID: 41203951\n",
      "> Fetched abstract!!\n",
      "[89/1203] Fetching abstract for PMID: 41203950\n",
      "> Fetched abstract!!\n",
      "[90/1203] Fetching abstract for PMID: 41203947\n",
      "> Fetched abstract!!\n",
      "[91/1203] Fetching abstract for PMID: 41203944\n",
      "> Fetched abstract!!\n",
      "[92/1203] Fetching abstract for PMID: 41203941\n",
      "> Fetched abstract!!\n",
      "[93/1203] Fetching abstract for PMID: 41203923\n",
      "> Fetched abstract!!\n",
      "[94/1203] Fetching abstract for PMID: 41203914\n",
      "> Fetched abstract!!\n",
      "[95/1203] Fetching abstract for PMID: 41203913\n",
      "> Fetched abstract!!\n",
      "[96/1203] Fetching abstract for PMID: 41203904\n",
      "> Fetched abstract!!\n",
      "[97/1203] Fetching abstract for PMID: 41203902\n",
      "> Fetched abstract!!\n",
      "[98/1203] Fetching abstract for PMID: 41203878\n",
      "> Fetched abstract!!\n",
      "[99/1203] Fetching abstract for PMID: 41203875\n",
      "> Fetched abstract!!\n",
      "[100/1203] Fetching abstract for PMID: 41203869\n",
      "> Fetched abstract!!\n",
      "[101/1203] Fetching abstract for PMID: 41203868\n",
      "> Fetched abstract!!\n",
      "[102/1203] Fetching abstract for PMID: 41203853\n",
      "> Fetched abstract!!\n",
      "[103/1203] Fetching abstract for PMID: 41203847\n",
      "> Fetched abstract!!\n",
      "[104/1203] Fetching abstract for PMID: 41203834\n",
      "> Fetched abstract!!\n",
      "[105/1203] Fetching abstract for PMID: 41203832\n",
      "> Fetched abstract!!\n",
      "[106/1203] Fetching abstract for PMID: 41203822\n",
      "> Fetched abstract!!\n",
      "[107/1203] Fetching abstract for PMID: 41203813\n",
      "> Fetched abstract!!\n",
      "[108/1203] Fetching abstract for PMID: 41203799\n",
      "> Fetched abstract!!\n",
      "[109/1203] Fetching abstract for PMID: 41203797\n",
      "> Fetched abstract!!\n",
      "[110/1203] Fetching abstract for PMID: 41203786\n",
      "> Fetched abstract!!\n",
      "[111/1203] Fetching abstract for PMID: 41203783\n",
      "> Fetched abstract!!\n",
      "[112/1203] Fetching abstract for PMID: 41203771\n",
      "> Fetched abstract!!\n",
      "[113/1203] Fetching abstract for PMID: 41203756\n",
      "> Fetched abstract!!\n",
      "[114/1203] Fetching abstract for PMID: 41203724\n",
      "> Fetched abstract!!\n",
      "[115/1203] Fetching abstract for PMID: 41203703\n",
      "> Fetched abstract!!\n",
      "[116/1203] Fetching abstract for PMID: 41203700\n",
      "> Fetched abstract!!\n",
      "[117/1203] Fetching abstract for PMID: 41203698\n",
      "> Fetched abstract!!\n",
      "[118/1203] Fetching abstract for PMID: 41203676\n",
      "> Fetched abstract!!\n",
      "[119/1203] Fetching abstract for PMID: 41203648\n",
      "> Fetched abstract!!\n",
      "[120/1203] Fetching abstract for PMID: 41203645\n",
      "> Fetched abstract!!\n",
      "[121/1203] Fetching abstract for PMID: 41203628\n",
      "> Fetched abstract!!\n",
      "[122/1203] Fetching abstract for PMID: 41203626\n",
      "> Fetched abstract!!\n",
      "[123/1203] Fetching abstract for PMID: 41203621\n",
      "> Fetched abstract!!\n",
      "[124/1203] Fetching abstract for PMID: 41203616\n",
      "> Fetched abstract!!\n",
      "[125/1203] Fetching abstract for PMID: 41203614\n",
      "> Fetched abstract!!\n",
      "[126/1203] Fetching abstract for PMID: 41203604\n",
      "> Fetched abstract!!\n",
      "[127/1203] Fetching abstract for PMID: 41203598\n",
      "> Fetched abstract!!\n",
      "[128/1203] Fetching abstract for PMID: 41203588\n",
      "> Fetched abstract!!\n",
      "[129/1203] Fetching abstract for PMID: 41203587\n",
      "> Fetched abstract!!\n",
      "[130/1203] Fetching abstract for PMID: 41203583\n",
      "> Fetched abstract!!\n",
      "[131/1203] Fetching abstract for PMID: 41203581\n",
      "> Fetched abstract!!\n",
      "[132/1203] Fetching abstract for PMID: 41203578\n",
      "> Fetched abstract!!\n",
      "[133/1203] Fetching abstract for PMID: 41203575\n",
      "> Fetched abstract!!\n",
      "[134/1203] Fetching abstract for PMID: 41203569\n",
      "> Fetched abstract!!\n",
      "[135/1203] Fetching abstract for PMID: 41203545\n",
      "> Fetched abstract!!\n",
      "[136/1203] Fetching abstract for PMID: 41203543\n",
      "> Fetched abstract!!\n",
      "[137/1203] Fetching abstract for PMID: 41203528\n",
      "> Fetched abstract!!\n",
      "[138/1203] Fetching abstract for PMID: 41203517\n",
      "> Fetched abstract!!\n",
      "[139/1203] Fetching abstract for PMID: 41203516\n",
      "> Fetched abstract!!\n",
      "[140/1203] Fetching abstract for PMID: 41203506\n",
      "> Fetched abstract!!\n",
      "[141/1203] Fetching abstract for PMID: 41203497\n",
      "> Fetched abstract!!\n",
      "[142/1203] Fetching abstract for PMID: 41203468\n",
      "> Fetched abstract!!\n",
      "[143/1203] Fetching abstract for PMID: 41203462\n",
      "> Fetched abstract!!\n",
      "[144/1203] Fetching abstract for PMID: 41203458\n",
      "> Fetched abstract!!\n",
      "[145/1203] Fetching abstract for PMID: 41203456\n",
      "> Fetched abstract!!\n",
      "[146/1203] Fetching abstract for PMID: 41203455\n",
      "> Fetched abstract!!\n",
      "[147/1203] Fetching abstract for PMID: 41203448\n",
      "> Fetched abstract!!\n",
      "[148/1203] Fetching abstract for PMID: 41203438\n",
      "> Fetched abstract!!\n",
      "[149/1203] Fetching abstract for PMID: 41203414\n",
      "> Fetched abstract!!\n",
      "[150/1203] Fetching abstract for PMID: 41203405\n",
      "> Fetched abstract!!\n",
      "[151/1203] Fetching abstract for PMID: 41203382\n",
      "> Fetched abstract!!\n",
      "[152/1203] Fetching abstract for PMID: 41203376\n",
      "> Fetched abstract!!\n",
      "[153/1203] Fetching abstract for PMID: 41203372\n",
      "> Fetched abstract!!\n",
      "[154/1203] Fetching abstract for PMID: 41203371\n",
      "> Fetched abstract!!\n",
      "[155/1203] Fetching abstract for PMID: 41203370\n",
      "> Fetched abstract!!\n",
      "[156/1203] Fetching abstract for PMID: 41203369\n",
      "> Fetched abstract!!\n",
      "[157/1203] Fetching abstract for PMID: 41203368\n",
      "> Fetched abstract!!\n",
      "[158/1203] Fetching abstract for PMID: 41203359\n",
      "> Fetched abstract!!\n",
      "[159/1203] Fetching abstract for PMID: 41203350\n",
      "> Fetched abstract!!\n",
      "[160/1203] Fetching abstract for PMID: 41203347\n",
      "> Fetched abstract!!\n",
      "[161/1203] Fetching abstract for PMID: 41203284\n",
      "> Fetched abstract!!\n",
      "[162/1203] Fetching abstract for PMID: 41203263\n",
      "> Fetched abstract!!\n",
      "[163/1203] Fetching abstract for PMID: 41203258\n",
      "> Fetched abstract!!\n",
      "[164/1203] Fetching abstract for PMID: 41203254\n",
      "> Fetched abstract!!\n",
      "[165/1203] Fetching abstract for PMID: 41203247\n",
      "> Fetched abstract!!\n",
      "[166/1203] Fetching abstract for PMID: 41203242\n",
      "> Fetched abstract!!\n",
      "[167/1203] Fetching abstract for PMID: 41203216\n",
      "> Fetched abstract!!\n",
      "[168/1203] Fetching abstract for PMID: 41203206\n",
      "> Fetched abstract!!\n",
      "[169/1203] Fetching abstract for PMID: 41203195\n",
      "> Fetched abstract!!\n",
      "[170/1203] Fetching abstract for PMID: 41203185\n",
      "> Fetched abstract!!\n",
      "[171/1203] Fetching abstract for PMID: 41203183\n",
      "> Fetched abstract!!\n",
      "[172/1203] Fetching abstract for PMID: 41203182\n",
      "> Fetched abstract!!\n",
      "[173/1203] Fetching abstract for PMID: 41203181\n",
      "> Fetched abstract!!\n",
      "[174/1203] Fetching abstract for PMID: 41203170\n",
      "> Fetched abstract!!\n",
      "[175/1203] Fetching abstract for PMID: 41203167\n",
      "> Fetched abstract!!\n",
      "[176/1203] Fetching abstract for PMID: 41203163\n",
      "> Fetched abstract!!\n",
      "[177/1203] Fetching abstract for PMID: 41203160\n",
      "> Fetched abstract!!\n",
      "[178/1203] Fetching abstract for PMID: 41203148\n",
      "> Fetched abstract!!\n",
      "[179/1203] Fetching abstract for PMID: 41203116\n",
      "> Fetched abstract!!\n",
      "[180/1203] Fetching abstract for PMID: 41203099\n",
      "> Fetched abstract!!\n",
      "[181/1203] Fetching abstract for PMID: 41203098\n",
      "> Fetched abstract!!\n",
      "[182/1203] Fetching abstract for PMID: 41203074\n",
      "> Fetched abstract!!\n",
      "[183/1203] Fetching abstract for PMID: 41203070\n",
      "> Fetched abstract!!\n",
      "[184/1203] Fetching abstract for PMID: 41203047\n",
      "> Fetched abstract!!\n",
      "[185/1203] Fetching abstract for PMID: 41203044\n",
      "> Fetched abstract!!\n",
      "[186/1203] Fetching abstract for PMID: 41203043\n",
      "> Fetched abstract!!\n",
      "[187/1203] Fetching abstract for PMID: 41203034\n",
      "> Fetched abstract!!\n",
      "[188/1203] Fetching abstract for PMID: 41203026\n",
      "> Fetched abstract!!\n",
      "[189/1203] Fetching abstract for PMID: 41203018\n",
      "> Fetched abstract!!\n",
      "[190/1203] Fetching abstract for PMID: 41203014\n",
      "> Fetched abstract!!\n",
      "[191/1203] Fetching abstract for PMID: 41203001\n",
      "> Fetched abstract!!\n",
      "[192/1203] Fetching abstract for PMID: 41203000\n",
      "> Fetched abstract!!\n",
      "[193/1203] Fetching abstract for PMID: 41202991\n",
      "> Fetched abstract!!\n",
      "[194/1203] Fetching abstract for PMID: 41202965\n",
      "> Fetched abstract!!\n",
      "[195/1203] Fetching abstract for PMID: 41202964\n",
      "> Fetched abstract!!\n",
      "[196/1203] Fetching abstract for PMID: 41202951\n",
      "> Fetched abstract!!\n",
      "[197/1203] Fetching abstract for PMID: 41202948\n",
      "> Fetched abstract!!\n",
      "[198/1203] Fetching abstract for PMID: 41202944\n",
      "> Fetched abstract!!\n",
      "[199/1203] Fetching abstract for PMID: 41202917\n",
      "> Fetched abstract!!\n",
      "[200/1203] Fetching abstract for PMID: 41202893\n",
      "> Fetched abstract!!\n",
      "[201/1203] Fetching abstract for PMID: 41202885\n",
      "> Fetched abstract!!\n",
      "[202/1203] Fetching abstract for PMID: 41202884\n",
      "> Fetched abstract!!\n",
      "[203/1203] Fetching abstract for PMID: 41202880\n",
      "> Fetched abstract!!\n",
      "[204/1203] Fetching abstract for PMID: 41202862\n",
      "> Fetched abstract!!\n",
      "[205/1203] Fetching abstract for PMID: 41202852\n",
      "> Fetched abstract!!\n",
      "[206/1203] Fetching abstract for PMID: 41202840\n",
      "> Fetched abstract!!\n",
      "[207/1203] Fetching abstract for PMID: 41202839\n",
      "> Fetched abstract!!\n",
      "[208/1203] Fetching abstract for PMID: 41202824\n",
      "> Fetched abstract!!\n",
      "[209/1203] Fetching abstract for PMID: 41202810\n",
      "> Fetched abstract!!\n",
      "[210/1203] Fetching abstract for PMID: 41202809\n",
      "> Fetched abstract!!\n",
      "[211/1203] Fetching abstract for PMID: 41202808\n",
      "> Fetched abstract!!\n",
      "[212/1203] Fetching abstract for PMID: 41202804\n",
      "> Fetched abstract!!\n",
      "[213/1203] Fetching abstract for PMID: 41202783\n",
      "> Fetched abstract!!\n",
      "[214/1203] Fetching abstract for PMID: 41202779\n",
      "> Fetched abstract!!\n",
      "[215/1203] Fetching abstract for PMID: 41202710\n",
      "> Fetched abstract!!\n",
      "[216/1203] Fetching abstract for PMID: 41202701\n",
      "> Fetched abstract!!\n",
      "[217/1203] Fetching abstract for PMID: 41202700\n",
      "> Fetched abstract!!\n",
      "[218/1203] Fetching abstract for PMID: 41202699\n",
      "> Fetched abstract!!\n",
      "[219/1203] Fetching abstract for PMID: 41202697\n",
      "> Fetched abstract!!\n",
      "[220/1203] Fetching abstract for PMID: 41202695\n",
      "> Fetched abstract!!\n",
      "[221/1203] Fetching abstract for PMID: 41202694\n",
      "> Fetched abstract!!\n",
      "[222/1203] Fetching abstract for PMID: 41202669\n",
      "> Fetched abstract!!\n",
      "[223/1203] Fetching abstract for PMID: 41202650\n",
      "> Fetched abstract!!\n",
      "[224/1203] Fetching abstract for PMID: 41202649\n",
      "> Fetched abstract!!\n",
      "[225/1203] Fetching abstract for PMID: 41202616\n",
      "> Fetched abstract!!\n",
      "[226/1203] Fetching abstract for PMID: 41202545\n",
      "> Fetched abstract!!\n",
      "[227/1203] Fetching abstract for PMID: 41202523\n",
      "> Fetched abstract!!\n",
      "[228/1203] Fetching abstract for PMID: 41202516\n",
      "> Fetched abstract!!\n",
      "[229/1203] Fetching abstract for PMID: 41202514\n",
      "> Fetched abstract!!\n",
      "[230/1203] Fetching abstract for PMID: 41202504\n",
      "> Fetched abstract!!\n",
      "[231/1203] Fetching abstract for PMID: 41202503\n",
      "> Fetched abstract!!\n",
      "[232/1203] Fetching abstract for PMID: 41202502\n",
      "> Fetched abstract!!\n",
      "[233/1203] Fetching abstract for PMID: 41202501\n",
      "> Fetched abstract!!\n",
      "[234/1203] Fetching abstract for PMID: 41202500\n",
      "> Fetched abstract!!\n",
      "[235/1203] Fetching abstract for PMID: 41202458\n",
      "> Fetched abstract!!\n",
      "[236/1203] Fetching abstract for PMID: 41202457\n",
      "> Fetched abstract!!\n",
      "[237/1203] Fetching abstract for PMID: 41202433\n",
      "> Fetched abstract!!\n",
      "[238/1203] Fetching abstract for PMID: 41202429\n",
      "> Fetched abstract!!\n",
      "[239/1203] Fetching abstract for PMID: 41202422\n",
      "> Fetched abstract!!\n",
      "[240/1203] Fetching abstract for PMID: 41202420\n",
      "> Fetched abstract!!\n",
      "[241/1203] Fetching abstract for PMID: 41202415\n",
      "> Fetched abstract!!\n",
      "[242/1203] Fetching abstract for PMID: 41202411\n",
      "> Fetched abstract!!\n",
      "[243/1203] Fetching abstract for PMID: 41202407\n",
      "> Fetched abstract!!\n",
      "[244/1203] Fetching abstract for PMID: 41202406\n",
      "> Fetched abstract!!\n",
      "[245/1203] Fetching abstract for PMID: 41202397\n",
      "> Fetched abstract!!\n",
      "[246/1203] Fetching abstract for PMID: 41202390\n",
      "> Fetched abstract!!\n",
      "[247/1203] Fetching abstract for PMID: 41202387\n",
      "> Fetched abstract!!\n",
      "[248/1203] Fetching abstract for PMID: 41202379\n",
      "> Fetched abstract!!\n",
      "[249/1203] Fetching abstract for PMID: 41202375\n",
      "> Fetched abstract!!\n",
      "[250/1203] Fetching abstract for PMID: 41202374\n",
      "> Fetched abstract!!\n",
      "[251/1203] Fetching abstract for PMID: 41202352\n",
      "> Fetched abstract!!\n",
      "[252/1203] Fetching abstract for PMID: 41202333\n",
      "> Fetched abstract!!\n",
      "[253/1203] Fetching abstract for PMID: 41202326\n",
      "> Fetched abstract!!\n",
      "[254/1203] Fetching abstract for PMID: 41202323\n",
      "> Fetched abstract!!\n",
      "[255/1203] Fetching abstract for PMID: 41202321\n",
      "> Fetched abstract!!\n",
      "[256/1203] Fetching abstract for PMID: 41202320\n",
      "> Fetched abstract!!\n",
      "[257/1203] Fetching abstract for PMID: 41202319\n",
      "> Fetched abstract!!\n",
      "[258/1203] Fetching abstract for PMID: 41202318\n",
      "> Fetched abstract!!\n",
      "[259/1203] Fetching abstract for PMID: 41202316\n",
      "> Fetched abstract!!\n",
      "[260/1203] Fetching abstract for PMID: 41202315\n",
      "> Fetched abstract!!\n",
      "[261/1203] Fetching abstract for PMID: 41202314\n",
      "> Fetched abstract!!\n",
      "[262/1203] Fetching abstract for PMID: 41202311\n",
      "> Fetched abstract!!\n",
      "[263/1203] Fetching abstract for PMID: 41202298\n",
      "> Fetched abstract!!\n",
      "[264/1203] Fetching abstract for PMID: 41202211\n",
      "> Fetched abstract!!\n",
      "[265/1203] Fetching abstract for PMID: 41202209\n",
      "> Fetched abstract!!\n",
      "[266/1203] Fetching abstract for PMID: 41202204\n",
      "> Fetched abstract!!\n",
      "[267/1203] Fetching abstract for PMID: 41202201\n",
      "> Fetched abstract!!\n",
      "[268/1203] Fetching abstract for PMID: 41202200\n",
      "> Fetched abstract!!\n",
      "[269/1203] Fetching abstract for PMID: 41202197\n",
      "> Fetched abstract!!\n",
      "[270/1203] Fetching abstract for PMID: 41202192\n",
      "> Fetched abstract!!\n",
      "[271/1203] Fetching abstract for PMID: 41202162\n",
      "> Fetched abstract!!\n",
      "[272/1203] Fetching abstract for PMID: 41202161\n",
      "> Fetched abstract!!\n",
      "[273/1203] Fetching abstract for PMID: 41202131\n",
      "> Fetched abstract!!\n",
      "[274/1203] Fetching abstract for PMID: 41202091\n",
      "> Fetched abstract!!\n",
      "[275/1203] Fetching abstract for PMID: 41202074\n",
      "> Fetched abstract!!\n",
      "[276/1203] Fetching abstract for PMID: 41202063\n",
      "> Fetched abstract!!\n",
      "[277/1203] Fetching abstract for PMID: 41201994\n",
      "> Fetched abstract!!\n",
      "[278/1203] Fetching abstract for PMID: 41201985\n",
      "> Fetched abstract!!\n",
      "[279/1203] Fetching abstract for PMID: 41201982\n",
      "> Fetched abstract!!\n",
      "[280/1203] Fetching abstract for PMID: 41201981\n",
      "> Fetched abstract!!\n",
      "[281/1203] Fetching abstract for PMID: 41201977\n",
      "> Fetched abstract!!\n",
      "[282/1203] Fetching abstract for PMID: 41201972\n",
      "> Fetched abstract!!\n",
      "[283/1203] Fetching abstract for PMID: 41201971\n",
      "> Fetched abstract!!\n",
      "[284/1203] Fetching abstract for PMID: 41201959\n",
      "> Fetched abstract!!\n",
      "[285/1203] Fetching abstract for PMID: 41201958\n",
      "> Fetched abstract!!\n",
      "[286/1203] Fetching abstract for PMID: 41201956\n",
      "> Fetched abstract!!\n",
      "[287/1203] Fetching abstract for PMID: 41201955\n",
      "> Fetched abstract!!\n",
      "[288/1203] Fetching abstract for PMID: 41201943\n",
      "> Fetched abstract!!\n",
      "[289/1203] Fetching abstract for PMID: 41201933\n",
      "> Fetched abstract!!\n",
      "[290/1203] Fetching abstract for PMID: 41201920\n",
      "> Fetched abstract!!\n",
      "[291/1203] Fetching abstract for PMID: 41201834\n",
      "> Fetched abstract!!\n",
      "[292/1203] Fetching abstract for PMID: 41201825\n",
      "> Fetched abstract!!\n",
      "[293/1203] Fetching abstract for PMID: 41201801\n",
      "> Fetched abstract!!\n",
      "[294/1203] Fetching abstract for PMID: 41201753\n",
      "> Fetched abstract!!\n",
      "[295/1203] Fetching abstract for PMID: 41201743\n",
      "> Fetched abstract!!\n",
      "[296/1203] Fetching abstract for PMID: 41201742\n",
      "> Fetched abstract!!\n",
      "[297/1203] Fetching abstract for PMID: 41201738\n",
      "> Fetched abstract!!\n",
      "[298/1203] Fetching abstract for PMID: 41201732\n",
      "> Fetched abstract!!\n",
      "[299/1203] Fetching abstract for PMID: 41201730\n",
      "> Fetched abstract!!\n",
      "[300/1203] Fetching abstract for PMID: 41201728\n",
      "> Fetched abstract!!\n",
      "[301/1203] Fetching abstract for PMID: 41201724\n",
      "> Fetched abstract!!\n",
      "[302/1203] Fetching abstract for PMID: 41201718\n",
      "> Fetched abstract!!\n",
      "[303/1203] Fetching abstract for PMID: 41201717\n",
      "> Fetched abstract!!\n",
      "[304/1203] Fetching abstract for PMID: 41201704\n",
      "> Fetched abstract!!\n",
      "[305/1203] Fetching abstract for PMID: 41201701\n",
      "> Fetched abstract!!\n",
      "[306/1203] Fetching abstract for PMID: 41201690\n",
      "> Fetched abstract!!\n",
      "[307/1203] Fetching abstract for PMID: 41201680\n",
      "> Fetched abstract!!\n",
      "[308/1203] Fetching abstract for PMID: 41201679\n",
      "> Fetched abstract!!\n",
      "[309/1203] Fetching abstract for PMID: 41201676\n",
      "> Fetched abstract!!\n",
      "[310/1203] Fetching abstract for PMID: 41201667\n",
      "> Fetched abstract!!\n",
      "[311/1203] Fetching abstract for PMID: 41201663\n",
      "> Fetched abstract!!\n",
      "[312/1203] Fetching abstract for PMID: 41201646\n",
      "> Fetched abstract!!\n",
      "[313/1203] Fetching abstract for PMID: 41201642\n",
      "> Fetched abstract!!\n",
      "[314/1203] Fetching abstract for PMID: 41201640\n",
      "> Fetched abstract!!\n",
      "[315/1203] Fetching abstract for PMID: 41201634\n",
      "> Fetched abstract!!\n",
      "[316/1203] Fetching abstract for PMID: 41201629\n",
      "> Fetched abstract!!\n",
      "[317/1203] Fetching abstract for PMID: 41201628\n",
      "> Fetched abstract!!\n",
      "[318/1203] Fetching abstract for PMID: 41201621\n",
      "> Fetched abstract!!\n",
      "[319/1203] Fetching abstract for PMID: 41201601\n",
      "> Fetched abstract!!\n",
      "[320/1203] Fetching abstract for PMID: 41201599\n",
      "> Fetched abstract!!\n",
      "[321/1203] Fetching abstract for PMID: 41201598\n",
      "> Fetched abstract!!\n",
      "[322/1203] Fetching abstract for PMID: 41201595\n",
      "> Fetched abstract!!\n",
      "[323/1203] Fetching abstract for PMID: 41201582\n",
      "> Fetched abstract!!\n",
      "[324/1203] Fetching abstract for PMID: 41201568\n",
      "> Fetched abstract!!\n",
      "[325/1203] Fetching abstract for PMID: 41201567\n",
      "> Fetched abstract!!\n",
      "[326/1203] Fetching abstract for PMID: 41201561\n",
      "> Fetched abstract!!\n",
      "[327/1203] Fetching abstract for PMID: 41201557\n",
      "> Fetched abstract!!\n",
      "[328/1203] Fetching abstract for PMID: 41201555\n",
      "> Fetched abstract!!\n",
      "[329/1203] Fetching abstract for PMID: 41201553\n",
      "> Fetched abstract!!\n",
      "[330/1203] Fetching abstract for PMID: 41201532\n",
      "> Fetched abstract!!\n",
      "[331/1203] Fetching abstract for PMID: 41201528\n",
      "> Fetched abstract!!\n",
      "[332/1203] Fetching abstract for PMID: 41201523\n",
      "> Fetched abstract!!\n",
      "[333/1203] Fetching abstract for PMID: 41201522\n",
      "> Fetched abstract!!\n",
      "[334/1203] Fetching abstract for PMID: 41201511\n",
      "> Fetched abstract!!\n",
      "[335/1203] Fetching abstract for PMID: 41201494\n",
      "> Fetched abstract!!\n",
      "[336/1203] Fetching abstract for PMID: 41201488\n",
      "> Fetched abstract!!\n",
      "[337/1203] Fetching abstract for PMID: 41201478\n",
      "> Fetched abstract!!\n",
      "[338/1203] Fetching abstract for PMID: 41201474\n",
      "> Fetched abstract!!\n",
      "[339/1203] Fetching abstract for PMID: 41201473\n",
      "> Fetched abstract!!\n",
      "[340/1203] Fetching abstract for PMID: 41201464\n",
      "> Fetched abstract!!\n",
      "[341/1203] Fetching abstract for PMID: 41201462\n",
      "> Fetched abstract!!\n",
      "[342/1203] Fetching abstract for PMID: 41201455\n",
      "> Fetched abstract!!\n",
      "[343/1203] Fetching abstract for PMID: 41201451\n",
      "> Fetched abstract!!\n",
      "[344/1203] Fetching abstract for PMID: 41201442\n",
      "> Fetched abstract!!\n",
      "[345/1203] Fetching abstract for PMID: 41201396\n",
      "> Fetched abstract!!\n",
      "[346/1203] Fetching abstract for PMID: 41201395\n",
      "> Fetched abstract!!\n",
      "[347/1203] Fetching abstract for PMID: 41201349\n",
      "> Fetched abstract!!\n",
      "[348/1203] Fetching abstract for PMID: 41201322\n",
      "> Fetched abstract!!\n",
      "[349/1203] Fetching abstract for PMID: 41201296\n",
      "> Fetched abstract!!\n",
      "[350/1203] Fetching abstract for PMID: 41201291\n",
      "> Fetched abstract!!\n",
      "[351/1203] Fetching abstract for PMID: 41201262\n",
      "> Fetched abstract!!\n",
      "[352/1203] Fetching abstract for PMID: 41201259\n",
      "> Fetched abstract!!\n",
      "[353/1203] Fetching abstract for PMID: 41201246\n",
      "> Fetched abstract!!\n",
      "[354/1203] Fetching abstract for PMID: 41201242\n",
      "> Fetched abstract!!\n",
      "[355/1203] Fetching abstract for PMID: 41201236\n",
      "> Fetched abstract!!\n",
      "[356/1203] Fetching abstract for PMID: 41201222\n",
      "> Fetched abstract!!\n",
      "[357/1203] Fetching abstract for PMID: 41201209\n",
      "> Fetched abstract!!\n",
      "[358/1203] Fetching abstract for PMID: 41201200\n",
      "> Fetched abstract!!\n",
      "[359/1203] Fetching abstract for PMID: 41201196\n",
      "> Fetched abstract!!\n",
      "[360/1203] Fetching abstract for PMID: 41201183\n",
      "> Fetched abstract!!\n",
      "[361/1203] Fetching abstract for PMID: 41201177\n",
      "> Fetched abstract!!\n",
      "[362/1203] Fetching abstract for PMID: 41201175\n",
      "> Fetched abstract!!\n",
      "[363/1203] Fetching abstract for PMID: 41201164\n",
      "> Fetched abstract!!\n",
      "[364/1203] Fetching abstract for PMID: 41201153\n",
      "> Fetched abstract!!\n",
      "[365/1203] Fetching abstract for PMID: 41201142\n",
      "> Fetched abstract!!\n",
      "[366/1203] Fetching abstract for PMID: 41201121\n",
      "> Fetched abstract!!\n",
      "[367/1203] Fetching abstract for PMID: 41201114\n",
      "> Fetched abstract!!\n",
      "[368/1203] Fetching abstract for PMID: 41201112\n",
      "> Fetched abstract!!\n",
      "[369/1203] Fetching abstract for PMID: 41201102\n",
      "> Fetched abstract!!\n",
      "[370/1203] Fetching abstract for PMID: 41201099\n",
      "> Fetched abstract!!\n",
      "[371/1203] Fetching abstract for PMID: 41201091\n",
      "> Fetched abstract!!\n",
      "[372/1203] Fetching abstract for PMID: 41201090\n",
      "> Fetched abstract!!\n",
      "[373/1203] Fetching abstract for PMID: 41201075\n",
      "> Fetched abstract!!\n",
      "[374/1203] Fetching abstract for PMID: 41201070\n",
      "> Fetched abstract!!\n",
      "[375/1203] Fetching abstract for PMID: 41201063\n",
      "> Fetched abstract!!\n",
      "[376/1203] Fetching abstract for PMID: 41201056\n",
      "> Fetched abstract!!\n",
      "[377/1203] Fetching abstract for PMID: 41201054\n",
      "> Fetched abstract!!\n",
      "[378/1203] Fetching abstract for PMID: 41201053\n",
      "> Fetched abstract!!\n",
      "[379/1203] Fetching abstract for PMID: 41201047\n",
      "> Fetched abstract!!\n",
      "[380/1203] Fetching abstract for PMID: 41201046\n",
      "> Fetched abstract!!\n",
      "[381/1203] Fetching abstract for PMID: 41201045\n",
      "> Fetched abstract!!\n",
      "[382/1203] Fetching abstract for PMID: 41201043\n",
      "> Fetched abstract!!\n",
      "[383/1203] Fetching abstract for PMID: 41201040\n",
      "> Fetched abstract!!\n",
      "[384/1203] Fetching abstract for PMID: 41201028\n",
      "> Fetched abstract!!\n",
      "[385/1203] Fetching abstract for PMID: 41201026\n",
      "> Fetched abstract!!\n",
      "[386/1203] Fetching abstract for PMID: 41201023\n",
      "> Fetched abstract!!\n",
      "[387/1203] Fetching abstract for PMID: 41201019\n",
      "> Fetched abstract!!\n",
      "[388/1203] Fetching abstract for PMID: 41201018\n",
      "> Fetched abstract!!\n",
      "[389/1203] Fetching abstract for PMID: 41201017\n",
      "> Fetched abstract!!\n",
      "[390/1203] Fetching abstract for PMID: 41201015\n",
      "> Fetched abstract!!\n",
      "[391/1203] Fetching abstract for PMID: 41201012\n",
      "> Fetched abstract!!\n",
      "[392/1203] Fetching abstract for PMID: 41201008\n",
      "> Fetched abstract!!\n",
      "[393/1203] Fetching abstract for PMID: 41200996\n",
      "> Fetched abstract!!\n",
      "[394/1203] Fetching abstract for PMID: 41200974\n",
      "> Fetched abstract!!\n",
      "[395/1203] Fetching abstract for PMID: 41200966\n",
      "> Fetched abstract!!\n",
      "[396/1203] Fetching abstract for PMID: 41200965\n",
      "> Fetched abstract!!\n",
      "[397/1203] Fetching abstract for PMID: 41200958\n",
      "> Fetched abstract!!\n",
      "[398/1203] Fetching abstract for PMID: 41200944\n",
      "> Fetched abstract!!\n",
      "[399/1203] Fetching abstract for PMID: 41200928\n",
      "> Fetched abstract!!\n",
      "[400/1203] Fetching abstract for PMID: 41200918\n",
      "> Fetched abstract!!\n",
      "[401/1203] Fetching abstract for PMID: 41200914\n",
      "> Fetched abstract!!\n",
      "[402/1203] Fetching abstract for PMID: 41200911\n",
      "> Fetched abstract!!\n",
      "[403/1203] Fetching abstract for PMID: 41200902\n",
      "> Fetched abstract!!\n",
      "[404/1203] Fetching abstract for PMID: 41200891\n",
      "> Fetched abstract!!\n",
      "[405/1203] Fetching abstract for PMID: 41200882\n",
      "> Fetched abstract!!\n",
      "[406/1203] Fetching abstract for PMID: 41200879\n",
      "> Fetched abstract!!\n",
      "[407/1203] Fetching abstract for PMID: 41200876\n",
      "> Fetched abstract!!\n",
      "[408/1203] Fetching abstract for PMID: 41200835\n",
      "> Fetched abstract!!\n",
      "[409/1203] Fetching abstract for PMID: 41200828\n",
      "> Fetched abstract!!\n",
      "[410/1203] Fetching abstract for PMID: 41200810\n",
      "> Fetched abstract!!\n",
      "[411/1203] Fetching abstract for PMID: 41200801\n",
      "> Fetched abstract!!\n",
      "[412/1203] Fetching abstract for PMID: 41200787\n",
      "> Fetched abstract!!\n",
      "[413/1203] Fetching abstract for PMID: 41200780\n",
      "> Fetched abstract!!\n",
      "[414/1203] Fetching abstract for PMID: 41200768\n",
      "> Fetched abstract!!\n",
      "[415/1203] Fetching abstract for PMID: 41200756\n",
      "> Fetched abstract!!\n",
      "[416/1203] Fetching abstract for PMID: 41200741\n",
      "> Fetched abstract!!\n",
      "[417/1203] Fetching abstract for PMID: 41200733\n",
      "> Fetched abstract!!\n",
      "[418/1203] Fetching abstract for PMID: 41200722\n",
      "> Fetched abstract!!\n",
      "[419/1203] Fetching abstract for PMID: 41200720\n",
      "> Fetched abstract!!\n",
      "[420/1203] Fetching abstract for PMID: 41200700\n",
      "> Fetched abstract!!\n",
      "[421/1203] Fetching abstract for PMID: 41200699\n",
      "> Fetched abstract!!\n",
      "[422/1203] Fetching abstract for PMID: 41200697\n",
      "> Fetched abstract!!\n",
      "[423/1203] Fetching abstract for PMID: 41200668\n",
      "> Fetched abstract!!\n",
      "[424/1203] Fetching abstract for PMID: 41200664\n",
      "> Fetched abstract!!\n",
      "[425/1203] Fetching abstract for PMID: 41200660\n",
      "> Fetched abstract!!\n",
      "[426/1203] Fetching abstract for PMID: 41200650\n",
      "> Fetched abstract!!\n",
      "[427/1203] Fetching abstract for PMID: 41200646\n",
      "> Fetched abstract!!\n",
      "[428/1203] Fetching abstract for PMID: 41200643\n",
      "> Fetched abstract!!\n",
      "[429/1203] Fetching abstract for PMID: 41200642\n",
      "> Fetched abstract!!\n",
      "[430/1203] Fetching abstract for PMID: 41200633\n",
      "> Fetched abstract!!\n",
      "[431/1203] Fetching abstract for PMID: 41200621\n",
      "> Fetched abstract!!\n",
      "[432/1203] Fetching abstract for PMID: 41200618\n",
      "> Fetched abstract!!\n",
      "[433/1203] Fetching abstract for PMID: 41200608\n",
      "> Fetched abstract!!\n",
      "[434/1203] Fetching abstract for PMID: 41200605\n",
      "> Fetched abstract!!\n",
      "[435/1203] Fetching abstract for PMID: 41200588\n",
      "> Fetched abstract!!\n",
      "[436/1203] Fetching abstract for PMID: 41200583\n",
      "> Fetched abstract!!\n",
      "[437/1203] Fetching abstract for PMID: 41200578\n",
      "> Fetched abstract!!\n",
      "[438/1203] Fetching abstract for PMID: 41200572\n",
      "> Fetched abstract!!\n",
      "[439/1203] Fetching abstract for PMID: 41200564\n",
      "> Fetched abstract!!\n",
      "[440/1203] Fetching abstract for PMID: 41200464\n",
      "> Fetched abstract!!\n",
      "[441/1203] Fetching abstract for PMID: 41200460\n",
      "> Fetched abstract!!\n",
      "[442/1203] Fetching abstract for PMID: 41200408\n",
      "> Fetched abstract!!\n",
      "[443/1203] Fetching abstract for PMID: 41200384\n",
      "> Fetched abstract!!\n",
      "[444/1203] Fetching abstract for PMID: 41200369\n",
      "> Fetched abstract!!\n",
      "[445/1203] Fetching abstract for PMID: 41200368\n",
      "> Fetched abstract!!\n",
      "[446/1203] Fetching abstract for PMID: 41200360\n",
      "> Fetched abstract!!\n",
      "[447/1203] Fetching abstract for PMID: 41200338\n",
      "> Fetched abstract!!\n",
      "[448/1203] Fetching abstract for PMID: 41200311\n",
      "> Fetched abstract!!\n",
      "[449/1203] Fetching abstract for PMID: 41200301\n",
      "> Fetched abstract!!\n",
      "[450/1203] Fetching abstract for PMID: 41200281\n",
      "> Fetched abstract!!\n",
      "[451/1203] Fetching abstract for PMID: 41200280\n",
      "> Fetched abstract!!\n",
      "[452/1203] Fetching abstract for PMID: 41200279\n",
      "> Fetched abstract!!\n",
      "[453/1203] Fetching abstract for PMID: 41200217\n",
      "> Fetched abstract!!\n",
      "[454/1203] Fetching abstract for PMID: 41200213\n",
      "> Fetched abstract!!\n",
      "[455/1203] Fetching abstract for PMID: 41200204\n",
      "> Fetched abstract!!\n",
      "[456/1203] Fetching abstract for PMID: 41200202\n",
      "> Fetched abstract!!\n",
      "[457/1203] Fetching abstract for PMID: 41200195\n",
      "> Fetched abstract!!\n",
      "[458/1203] Fetching abstract for PMID: 41200193\n",
      "> Fetched abstract!!\n",
      "[459/1203] Fetching abstract for PMID: 41200187\n",
      "> Fetched abstract!!\n",
      "[460/1203] Fetching abstract for PMID: 41200180\n",
      "> Fetched abstract!!\n",
      "[461/1203] Fetching abstract for PMID: 41200178\n",
      "> Fetched abstract!!\n",
      "[462/1203] Fetching abstract for PMID: 41200174\n",
      "> Fetched abstract!!\n",
      "[463/1203] Fetching abstract for PMID: 41200173\n",
      "> Fetched abstract!!\n",
      "[464/1203] Fetching abstract for PMID: 41200171\n",
      "> Fetched abstract!!\n",
      "[465/1203] Fetching abstract for PMID: 41200166\n",
      "> Fetched abstract!!\n",
      "[466/1203] Fetching abstract for PMID: 41200165\n",
      "> Fetched abstract!!\n",
      "[467/1203] Fetching abstract for PMID: 41200162\n",
      "> Fetched abstract!!\n",
      "[468/1203] Fetching abstract for PMID: 41200147\n",
      "> Fetched abstract!!\n",
      "[469/1203] Fetching abstract for PMID: 41200141\n",
      "> Fetched abstract!!\n",
      "[470/1203] Fetching abstract for PMID: 41200138\n",
      "> Fetched abstract!!\n",
      "[471/1203] Fetching abstract for PMID: 41200137\n",
      "> Fetched abstract!!\n",
      "[472/1203] Fetching abstract for PMID: 41200093\n",
      "> Fetched abstract!!\n",
      "[473/1203] Fetching abstract for PMID: 41200067\n",
      "> Fetched abstract!!\n",
      "[474/1203] Fetching abstract for PMID: 41200066\n",
      "> Fetched abstract!!\n",
      "[475/1203] Fetching abstract for PMID: 41200065\n",
      "> Fetched abstract!!\n",
      "[476/1203] Fetching abstract for PMID: 41200064\n",
      "> Fetched abstract!!\n",
      "[477/1203] Fetching abstract for PMID: 41200063\n",
      "> Fetched abstract!!\n",
      "[478/1203] Fetching abstract for PMID: 41200062\n",
      "> Fetched abstract!!\n",
      "[479/1203] Fetching abstract for PMID: 41200004\n",
      "> Fetched abstract!!\n",
      "[480/1203] Fetching abstract for PMID: 41199987\n",
      "> Fetched abstract!!\n",
      "[481/1203] Fetching abstract for PMID: 41199985\n",
      "> Fetched abstract!!\n",
      "[482/1203] Fetching abstract for PMID: 41199984\n",
      "> Fetched abstract!!\n",
      "[483/1203] Fetching abstract for PMID: 41199980\n",
      "> Fetched abstract!!\n",
      "[484/1203] Fetching abstract for PMID: 41199979\n",
      "> Fetched abstract!!\n",
      "[485/1203] Fetching abstract for PMID: 41199978\n",
      "> Fetched abstract!!\n",
      "[486/1203] Fetching abstract for PMID: 41199974\n",
      "> Fetched abstract!!\n",
      "[487/1203] Fetching abstract for PMID: 41199971\n",
      "> Fetched abstract!!\n",
      "[488/1203] Fetching abstract for PMID: 41199967\n",
      "> Fetched abstract!!\n",
      "[489/1203] Fetching abstract for PMID: 41199965\n",
      "> Fetched abstract!!\n",
      "[490/1203] Fetching abstract for PMID: 41199964\n",
      "> Fetched abstract!!\n",
      "[491/1203] Fetching abstract for PMID: 41199963\n",
      "> Fetched abstract!!\n",
      "[492/1203] Fetching abstract for PMID: 41199959\n",
      "> Fetched abstract!!\n",
      "[493/1203] Fetching abstract for PMID: 41199958\n",
      "> Fetched abstract!!\n",
      "[494/1203] Fetching abstract for PMID: 41199957\n",
      "> Fetched abstract!!\n",
      "[495/1203] Fetching abstract for PMID: 41199950\n",
      "> Fetched abstract!!\n",
      "[496/1203] Fetching abstract for PMID: 41199904\n",
      "> Fetched abstract!!\n",
      "[497/1203] Fetching abstract for PMID: 41199894\n",
      "> Fetched abstract!!\n",
      "[498/1203] Fetching abstract for PMID: 41199892\n",
      "> Fetched abstract!!\n",
      "[499/1203] Fetching abstract for PMID: 41199882\n",
      "> Fetched abstract!!\n",
      "[500/1203] Fetching abstract for PMID: 41199854\n",
      "> Fetched abstract!!\n",
      "[501/1203] Fetching abstract for PMID: 41199853\n",
      "> Fetched abstract!!\n",
      "[502/1203] Fetching abstract for PMID: 41199852\n",
      "> Fetched abstract!!\n",
      "[503/1203] Fetching abstract for PMID: 41199849\n",
      "> Fetched abstract!!\n",
      "[504/1203] Fetching abstract for PMID: 41199848\n",
      "> Fetched abstract!!\n",
      "[505/1203] Fetching abstract for PMID: 41199847\n",
      "> Fetched abstract!!\n",
      "[506/1203] Fetching abstract for PMID: 41199846\n",
      "> Fetched abstract!!\n",
      "[507/1203] Fetching abstract for PMID: 41199845\n",
      "> Fetched abstract!!\n",
      "[508/1203] Fetching abstract for PMID: 41199843\n",
      "> Fetched abstract!!\n",
      "[509/1203] Fetching abstract for PMID: 41199842\n",
      "> Fetched abstract!!\n",
      "[510/1203] Fetching abstract for PMID: 41199840\n",
      "> Fetched abstract!!\n",
      "[511/1203] Fetching abstract for PMID: 41199839\n",
      "> Fetched abstract!!\n",
      "[512/1203] Fetching abstract for PMID: 41199838\n",
      "> Fetched abstract!!\n",
      "[513/1203] Fetching abstract for PMID: 41199836\n",
      "> Fetched abstract!!\n",
      "[514/1203] Fetching abstract for PMID: 41199834\n",
      "> Fetched abstract!!\n",
      "[515/1203] Fetching abstract for PMID: 41199833\n",
      "> Fetched abstract!!\n",
      "[516/1203] Fetching abstract for PMID: 41199832\n",
      "> Fetched abstract!!\n",
      "[517/1203] Fetching abstract for PMID: 41199831\n",
      "> Fetched abstract!!\n",
      "[518/1203] Fetching abstract for PMID: 41199830\n",
      "> Fetched abstract!!\n",
      "[519/1203] Fetching abstract for PMID: 41199826\n",
      "> Fetched abstract!!\n",
      "[520/1203] Fetching abstract for PMID: 41199824\n",
      "> Fetched abstract!!\n",
      "[521/1203] Fetching abstract for PMID: 41199821\n",
      "> Fetched abstract!!\n",
      "[522/1203] Fetching abstract for PMID: 41199819\n",
      "> Fetched abstract!!\n",
      "[523/1203] Fetching abstract for PMID: 41199784\n",
      "> Fetched abstract!!\n",
      "[524/1203] Fetching abstract for PMID: 41199766\n",
      "> Fetched abstract!!\n",
      "[525/1203] Fetching abstract for PMID: 41199747\n",
      "> Fetched abstract!!\n",
      "[526/1203] Fetching abstract for PMID: 41199745\n",
      "> Fetched abstract!!\n",
      "[527/1203] Fetching abstract for PMID: 41199740\n",
      "> Fetched abstract!!\n",
      "[528/1203] Fetching abstract for PMID: 41199735\n",
      "> Fetched abstract!!\n",
      "[529/1203] Fetching abstract for PMID: 41199687\n",
      "> Fetched abstract!!\n",
      "[530/1203] Fetching abstract for PMID: 41199686\n",
      "> Fetched abstract!!\n",
      "[531/1203] Fetching abstract for PMID: 41199675\n",
      "> Fetched abstract!!\n",
      "[532/1203] Fetching abstract for PMID: 41199671\n",
      "> Fetched abstract!!\n",
      "[533/1203] Fetching abstract for PMID: 41199668\n",
      "> Fetched abstract!!\n",
      "[534/1203] Fetching abstract for PMID: 41199659\n",
      "> Fetched abstract!!\n",
      "[535/1203] Fetching abstract for PMID: 41199656\n",
      "> Fetched abstract!!\n",
      "[536/1203] Fetching abstract for PMID: 41199648\n",
      "> Fetched abstract!!\n",
      "[537/1203] Fetching abstract for PMID: 41199633\n",
      "> Fetched abstract!!\n",
      "[538/1203] Fetching abstract for PMID: 41199631\n",
      "> Fetched abstract!!\n",
      "[539/1203] Fetching abstract for PMID: 41199607\n",
      "> Fetched abstract!!\n",
      "[540/1203] Fetching abstract for PMID: 41199590\n",
      "> Fetched abstract!!\n",
      "[541/1203] Fetching abstract for PMID: 41199577\n",
      "> Fetched abstract!!\n",
      "[542/1203] Fetching abstract for PMID: 41199544\n",
      "> Fetched abstract!!\n",
      "[543/1203] Fetching abstract for PMID: 41199536\n",
      "> Fetched abstract!!\n",
      "[544/1203] Fetching abstract for PMID: 41199530\n",
      "> Fetched abstract!!\n",
      "[545/1203] Fetching abstract for PMID: 41199529\n",
      "> Fetched abstract!!\n",
      "[546/1203] Fetching abstract for PMID: 41199527\n",
      "> Fetched abstract!!\n",
      "[547/1203] Fetching abstract for PMID: 41199501\n",
      "> Fetched abstract!!\n",
      "[548/1203] Fetching abstract for PMID: 41199491\n",
      "> Fetched abstract!!\n",
      "[549/1203] Fetching abstract for PMID: 41199483\n",
      "> Fetched abstract!!\n",
      "[550/1203] Fetching abstract for PMID: 41199480\n",
      "> Fetched abstract!!\n",
      "[551/1203] Fetching abstract for PMID: 41199479\n",
      "> Fetched abstract!!\n",
      "[552/1203] Fetching abstract for PMID: 41199468\n",
      "> Fetched abstract!!\n",
      "[553/1203] Fetching abstract for PMID: 41199451\n",
      "> Fetched abstract!!\n",
      "[554/1203] Fetching abstract for PMID: 41199407\n",
      "> Fetched abstract!!\n",
      "[555/1203] Fetching abstract for PMID: 41199374\n",
      "> Fetched abstract!!\n",
      "[556/1203] Fetching abstract for PMID: 41199356\n",
      "> Fetched abstract!!\n",
      "[557/1203] Fetching abstract for PMID: 41199339\n",
      "> Fetched abstract!!\n",
      "[558/1203] Fetching abstract for PMID: 41199321\n",
      "> Fetched abstract!!\n",
      "[559/1203] Fetching abstract for PMID: 41199310\n",
      "> Fetched abstract!!\n",
      "[560/1203] Fetching abstract for PMID: 41199302\n",
      "> Fetched abstract!!\n",
      "[561/1203] Fetching abstract for PMID: 41199301\n",
      "> Fetched abstract!!\n",
      "[562/1203] Fetching abstract for PMID: 41199300\n",
      "> Fetched abstract!!\n",
      "[563/1203] Fetching abstract for PMID: 41199297\n",
      "> Fetched abstract!!\n",
      "[564/1203] Fetching abstract for PMID: 41199296\n",
      "> Fetched abstract!!\n",
      "[565/1203] Fetching abstract for PMID: 41199286\n",
      "> Fetched abstract!!\n",
      "[566/1203] Fetching abstract for PMID: 41199284\n",
      "> Fetched abstract!!\n",
      "[567/1203] Fetching abstract for PMID: 41199275\n",
      "> Fetched abstract!!\n",
      "[568/1203] Fetching abstract for PMID: 41199271\n",
      "> Fetched abstract!!\n",
      "[569/1203] Fetching abstract for PMID: 41199268\n",
      "> Fetched abstract!!\n",
      "[570/1203] Fetching abstract for PMID: 41199267\n",
      "> Fetched abstract!!\n",
      "[571/1203] Fetching abstract for PMID: 41199255\n",
      "> Fetched abstract!!\n",
      "[572/1203] Fetching abstract for PMID: 41199231\n",
      "> Fetched abstract!!\n",
      "[573/1203] Fetching abstract for PMID: 41199217\n",
      "> Fetched abstract!!\n",
      "[574/1203] Fetching abstract for PMID: 41199193\n",
      "> Fetched abstract!!\n",
      "[575/1203] Fetching abstract for PMID: 41199148\n",
      "> Fetched abstract!!\n",
      "[576/1203] Fetching abstract for PMID: 41199135\n",
      "> Fetched abstract!!\n",
      "[577/1203] Fetching abstract for PMID: 41199113\n",
      "> Fetched abstract!!\n",
      "[578/1203] Fetching abstract for PMID: 41199110\n",
      "> Fetched abstract!!\n",
      "[579/1203] Fetching abstract for PMID: 41199107\n",
      "> Fetched abstract!!\n",
      "[580/1203] Fetching abstract for PMID: 41199097\n",
      "> Fetched abstract!!\n",
      "[581/1203] Fetching abstract for PMID: 41199088\n",
      "> Fetched abstract!!\n",
      "[582/1203] Fetching abstract for PMID: 41199084\n",
      "> Fetched abstract!!\n",
      "[583/1203] Fetching abstract for PMID: 41199077\n",
      "> Fetched abstract!!\n",
      "[584/1203] Fetching abstract for PMID: 41199076\n",
      "> Fetched abstract!!\n",
      "[585/1203] Fetching abstract for PMID: 41199053\n",
      "> Fetched abstract!!\n",
      "[586/1203] Fetching abstract for PMID: 41199023\n",
      "> Fetched abstract!!\n",
      "[587/1203] Fetching abstract for PMID: 41199020\n",
      "> Fetched abstract!!\n",
      "[588/1203] Fetching abstract for PMID: 41199012\n",
      "> Fetched abstract!!\n",
      "[589/1203] Fetching abstract for PMID: 41199008\n",
      "> Fetched abstract!!\n",
      "[590/1203] Fetching abstract for PMID: 41198953\n",
      "> Fetched abstract!!\n",
      "[591/1203] Fetching abstract for PMID: 41198947\n",
      "> Fetched abstract!!\n",
      "[592/1203] Fetching abstract for PMID: 41198931\n",
      "> Fetched abstract!!\n",
      "[593/1203] Fetching abstract for PMID: 41198929\n",
      "> Fetched abstract!!\n",
      "[594/1203] Fetching abstract for PMID: 41198926\n",
      "> Fetched abstract!!\n",
      "[595/1203] Fetching abstract for PMID: 41198925\n",
      "> Fetched abstract!!\n",
      "[596/1203] Fetching abstract for PMID: 41198923\n",
      "> Fetched abstract!!\n",
      "[597/1203] Fetching abstract for PMID: 41198919\n",
      "> Fetched abstract!!\n",
      "[598/1203] Fetching abstract for PMID: 41198914\n",
      "> Fetched abstract!!\n",
      "[599/1203] Fetching abstract for PMID: 41198911\n",
      "> Fetched abstract!!\n",
      "[600/1203] Fetching abstract for PMID: 41198892\n",
      "> Fetched abstract!!\n",
      "[601/1203] Fetching abstract for PMID: 41198884\n",
      "> Fetched abstract!!\n",
      "[602/1203] Fetching abstract for PMID: 41198876\n",
      "> Fetched abstract!!\n",
      "[603/1203] Fetching abstract for PMID: 41198869\n",
      "> Fetched abstract!!\n",
      "[604/1203] Fetching abstract for PMID: 41198868\n",
      "> Fetched abstract!!\n",
      "[605/1203] Fetching abstract for PMID: 41198866\n",
      "> Fetched abstract!!\n",
      "[606/1203] Fetching abstract for PMID: 41198839\n",
      "> Fetched abstract!!\n",
      "[607/1203] Fetching abstract for PMID: 41198835\n",
      "> Fetched abstract!!\n",
      "[608/1203] Fetching abstract for PMID: 41198827\n",
      "> Fetched abstract!!\n",
      "[609/1203] Fetching abstract for PMID: 41198817\n",
      "> Fetched abstract!!\n",
      "[610/1203] Fetching abstract for PMID: 41198811\n",
      "> Fetched abstract!!\n",
      "[611/1203] Fetching abstract for PMID: 41198802\n",
      "> Fetched abstract!!\n",
      "[612/1203] Fetching abstract for PMID: 41198795\n",
      "> Fetched abstract!!\n",
      "[613/1203] Fetching abstract for PMID: 41198762\n",
      "> Fetched abstract!!\n",
      "[614/1203] Fetching abstract for PMID: 41198760\n",
      "> Fetched abstract!!\n",
      "[615/1203] Fetching abstract for PMID: 41198751\n",
      "> Fetched abstract!!\n",
      "[616/1203] Fetching abstract for PMID: 41198749\n",
      "> Fetched abstract!!\n",
      "[617/1203] Fetching abstract for PMID: 41198737\n",
      "> Fetched abstract!!\n",
      "[618/1203] Fetching abstract for PMID: 41198736\n",
      "> Fetched abstract!!\n",
      "[619/1203] Fetching abstract for PMID: 41198720\n",
      "> Fetched abstract!!\n",
      "[620/1203] Fetching abstract for PMID: 41198712\n",
      "> Fetched abstract!!\n",
      "[621/1203] Fetching abstract for PMID: 41198707\n",
      "> Fetched abstract!!\n",
      "[622/1203] Fetching abstract for PMID: 41198706\n",
      "> Fetched abstract!!\n",
      "[623/1203] Fetching abstract for PMID: 41198691\n",
      "> Fetched abstract!!\n",
      "[624/1203] Fetching abstract for PMID: 41198678\n",
      "> Fetched abstract!!\n",
      "[625/1203] Fetching abstract for PMID: 41198671\n",
      "> Fetched abstract!!\n",
      "[626/1203] Fetching abstract for PMID: 41198664\n",
      "> Fetched abstract!!\n",
      "[627/1203] Fetching abstract for PMID: 41198662\n",
      "> Fetched abstract!!\n",
      "[628/1203] Fetching abstract for PMID: 41198652\n",
      "> Fetched abstract!!\n",
      "[629/1203] Fetching abstract for PMID: 41198650\n",
      "> Fetched abstract!!\n",
      "[630/1203] Fetching abstract for PMID: 41198649\n",
      "> Fetched abstract!!\n",
      "[631/1203] Fetching abstract for PMID: 41198619\n",
      "> Fetched abstract!!\n",
      "[632/1203] Fetching abstract for PMID: 41198614\n",
      "> Fetched abstract!!\n",
      "[633/1203] Fetching abstract for PMID: 41198600\n",
      "> Fetched abstract!!\n",
      "[634/1203] Fetching abstract for PMID: 41198588\n",
      "> Fetched abstract!!\n",
      "[635/1203] Fetching abstract for PMID: 41198583\n",
      "> Fetched abstract!!\n",
      "[636/1203] Fetching abstract for PMID: 41198575\n",
      "> Fetched abstract!!\n",
      "[637/1203] Fetching abstract for PMID: 41198561\n",
      "> Fetched abstract!!\n",
      "[638/1203] Fetching abstract for PMID: 41198558\n",
      "> Fetched abstract!!\n",
      "[639/1203] Fetching abstract for PMID: 41198554\n",
      "> Fetched abstract!!\n",
      "[640/1203] Fetching abstract for PMID: 41198550\n",
      "> Fetched abstract!!\n",
      "[641/1203] Fetching abstract for PMID: 41198549\n",
      "> Fetched abstract!!\n",
      "[642/1203] Fetching abstract for PMID: 41198547\n",
      "> Fetched abstract!!\n",
      "[643/1203] Fetching abstract for PMID: 41198543\n",
      "> Fetched abstract!!\n",
      "[644/1203] Fetching abstract for PMID: 41198542\n",
      "> Fetched abstract!!\n",
      "[645/1203] Fetching abstract for PMID: 41198537\n",
      "> Fetched abstract!!\n",
      "[646/1203] Fetching abstract for PMID: 41198535\n",
      "> Fetched abstract!!\n",
      "[647/1203] Fetching abstract for PMID: 41198526\n",
      "> Fetched abstract!!\n",
      "[648/1203] Fetching abstract for PMID: 41198520\n",
      "> Fetched abstract!!\n",
      "[649/1203] Fetching abstract for PMID: 41198507\n",
      "> Fetched abstract!!\n",
      "[650/1203] Fetching abstract for PMID: 41198504\n",
      "> Fetched abstract!!\n",
      "[651/1203] Fetching abstract for PMID: 41198503\n",
      "> Fetched abstract!!\n",
      "[652/1203] Fetching abstract for PMID: 41198501\n",
      "> Fetched abstract!!\n",
      "[653/1203] Fetching abstract for PMID: 41198498\n",
      "> Fetched abstract!!\n",
      "[654/1203] Fetching abstract for PMID: 41198477\n",
      "> Fetched abstract!!\n",
      "[655/1203] Fetching abstract for PMID: 41198466\n",
      "> Fetched abstract!!\n",
      "[656/1203] Fetching abstract for PMID: 41198465\n",
      "> Fetched abstract!!\n",
      "[657/1203] Fetching abstract for PMID: 41198456\n",
      "> Fetched abstract!!\n",
      "[658/1203] Fetching abstract for PMID: 41198454\n",
      "> Fetched abstract!!\n",
      "[659/1203] Fetching abstract for PMID: 41198445\n",
      "> Fetched abstract!!\n",
      "[660/1203] Fetching abstract for PMID: 41198426\n",
      "> Fetched abstract!!\n",
      "[661/1203] Fetching abstract for PMID: 41198414\n",
      "> Fetched abstract!!\n",
      "[662/1203] Fetching abstract for PMID: 41198390\n",
      "> Fetched abstract!!\n",
      "[663/1203] Fetching abstract for PMID: 41198381\n",
      "> Fetched abstract!!\n",
      "[664/1203] Fetching abstract for PMID: 41198380\n",
      "> Fetched abstract!!\n",
      "[665/1203] Fetching abstract for PMID: 41198366\n",
      "> Fetched abstract!!\n",
      "[666/1203] Fetching abstract for PMID: 41198362\n",
      "> Fetched abstract!!\n",
      "[667/1203] Fetching abstract for PMID: 41198341\n",
      "> Fetched abstract!!\n",
      "[668/1203] Fetching abstract for PMID: 41198335\n",
      "> Fetched abstract!!\n",
      "[669/1203] Fetching abstract for PMID: 41198329\n",
      "> Fetched abstract!!\n",
      "[670/1203] Fetching abstract for PMID: 41198286\n",
      "> Fetched abstract!!\n",
      "[671/1203] Fetching abstract for PMID: 41198268\n",
      "> Fetched abstract!!\n",
      "[672/1203] Fetching abstract for PMID: 41198260\n",
      "> Fetched abstract!!\n",
      "[673/1203] Fetching abstract for PMID: 41198240\n",
      "> Fetched abstract!!\n",
      "[674/1203] Fetching abstract for PMID: 41198239\n",
      "> Fetched abstract!!\n",
      "[675/1203] Fetching abstract for PMID: 41198237\n",
      "> Fetched abstract!!\n",
      "[676/1203] Fetching abstract for PMID: 41198236\n",
      "> Fetched abstract!!\n",
      "[677/1203] Fetching abstract for PMID: 41198235\n",
      "> ERROR fetching abstract for PMID '41198235': 500 Server Error: Internal Server Error for url: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pubmed&id=41198235&retmode=text&rettype=abstract\n",
      "[678/1203] Fetching abstract for PMID: 41198234\n",
      "> Fetched abstract!!\n",
      "[679/1203] Fetching abstract for PMID: 41198210\n",
      "> Fetched abstract!!\n",
      "[680/1203] Fetching abstract for PMID: 41198208\n",
      "> Fetched abstract!!\n",
      "[681/1203] Fetching abstract for PMID: 41198201\n",
      "> Fetched abstract!!\n",
      "[682/1203] Fetching abstract for PMID: 41198197\n",
      "> Fetched abstract!!\n",
      "[683/1203] Fetching abstract for PMID: 41198192\n",
      "> Fetched abstract!!\n",
      "[684/1203] Fetching abstract for PMID: 41198191\n",
      "> Fetched abstract!!\n",
      "[685/1203] Fetching abstract for PMID: 41198175\n",
      "> Fetched abstract!!\n",
      "[686/1203] Fetching abstract for PMID: 41198162\n",
      "> Fetched abstract!!\n",
      "[687/1203] Fetching abstract for PMID: 41198160\n",
      "> Fetched abstract!!\n",
      "[688/1203] Fetching abstract for PMID: 41198159\n",
      "> Fetched abstract!!\n",
      "[689/1203] Fetching abstract for PMID: 41198154\n",
      "> Fetched abstract!!\n",
      "[690/1203] Fetching abstract for PMID: 41198147\n",
      "> Fetched abstract!!\n",
      "[691/1203] Fetching abstract for PMID: 41198144\n",
      "> Fetched abstract!!\n",
      "[692/1203] Fetching abstract for PMID: 41198094\n",
      "> Fetched abstract!!\n",
      "[693/1203] Fetching abstract for PMID: 41198065\n",
      "> Fetched abstract!!\n",
      "[694/1203] Fetching abstract for PMID: 41198046\n",
      "> Fetched abstract!!\n",
      "[695/1203] Fetching abstract for PMID: 41198041\n",
      "> Fetched abstract!!\n",
      "[696/1203] Fetching abstract for PMID: 41198016\n",
      "> Fetched abstract!!\n",
      "[697/1203] Fetching abstract for PMID: 41198001\n",
      "> Fetched abstract!!\n",
      "[698/1203] Fetching abstract for PMID: 41197974\n",
      "> Fetched abstract!!\n",
      "[699/1203] Fetching abstract for PMID: 41197973\n",
      "> Fetched abstract!!\n",
      "[700/1203] Fetching abstract for PMID: 41197971\n",
      "> Fetched abstract!!\n",
      "[701/1203] Fetching abstract for PMID: 41197969\n",
      "> Fetched abstract!!\n",
      "[702/1203] Fetching abstract for PMID: 41197933\n",
      "> Fetched abstract!!\n",
      "[703/1203] Fetching abstract for PMID: 41197929\n",
      "> Fetched abstract!!\n",
      "[704/1203] Fetching abstract for PMID: 41197925\n",
      "> Fetched abstract!!\n",
      "[705/1203] Fetching abstract for PMID: 41197905\n",
      "> Fetched abstract!!\n",
      "[706/1203] Fetching abstract for PMID: 41197902\n",
      "> Fetched abstract!!\n",
      "[707/1203] Fetching abstract for PMID: 41197890\n",
      "> Fetched abstract!!\n",
      "[708/1203] Fetching abstract for PMID: 41197887\n",
      "> Fetched abstract!!\n",
      "[709/1203] Fetching abstract for PMID: 41197885\n",
      "> Fetched abstract!!\n",
      "[710/1203] Fetching abstract for PMID: 41197884\n",
      "> Fetched abstract!!\n",
      "[711/1203] Fetching abstract for PMID: 41197832\n",
      "> Fetched abstract!!\n",
      "[712/1203] Fetching abstract for PMID: 41197829\n",
      "> Fetched abstract!!\n",
      "[713/1203] Fetching abstract for PMID: 41197798\n",
      "> Fetched abstract!!\n",
      "[714/1203] Fetching abstract for PMID: 41197788\n",
      "> Fetched abstract!!\n",
      "[715/1203] Fetching abstract for PMID: 41197773\n",
      "> Fetched abstract!!\n",
      "[716/1203] Fetching abstract for PMID: 41197772\n",
      "> Fetched abstract!!\n",
      "[717/1203] Fetching abstract for PMID: 41197768\n",
      "> Fetched abstract!!\n",
      "[718/1203] Fetching abstract for PMID: 41197758\n",
      "> Fetched abstract!!\n",
      "[719/1203] Fetching abstract for PMID: 41197755\n",
      "> Fetched abstract!!\n",
      "[720/1203] Fetching abstract for PMID: 41197754\n",
      "> Fetched abstract!!\n",
      "[721/1203] Fetching abstract for PMID: 41197753\n",
      "> Fetched abstract!!\n",
      "[722/1203] Fetching abstract for PMID: 41197750\n",
      "> Fetched abstract!!\n",
      "[723/1203] Fetching abstract for PMID: 41197749\n",
      "> Fetched abstract!!\n",
      "[724/1203] Fetching abstract for PMID: 41197740\n",
      "> Fetched abstract!!\n",
      "[725/1203] Fetching abstract for PMID: 41197735\n",
      "> Fetched abstract!!\n",
      "[726/1203] Fetching abstract for PMID: 41197720\n",
      "> Fetched abstract!!\n",
      "[727/1203] Fetching abstract for PMID: 41197707\n",
      "> Fetched abstract!!\n",
      "[728/1203] Fetching abstract for PMID: 41197706\n",
      "> Fetched abstract!!\n",
      "[729/1203] Fetching abstract for PMID: 41197699\n",
      "> Fetched abstract!!\n",
      "[730/1203] Fetching abstract for PMID: 41197687\n",
      "> Fetched abstract!!\n",
      "[731/1203] Fetching abstract for PMID: 41197639\n",
      "> Fetched abstract!!\n",
      "[732/1203] Fetching abstract for PMID: 41197618\n",
      "> Fetched abstract!!\n",
      "[733/1203] Fetching abstract for PMID: 41197616\n",
      "> Fetched abstract!!\n",
      "[734/1203] Fetching abstract for PMID: 41197587\n",
      "> Fetched abstract!!\n",
      "[735/1203] Fetching abstract for PMID: 41197584\n",
      "> Fetched abstract!!\n",
      "[736/1203] Fetching abstract for PMID: 41197580\n",
      "> Fetched abstract!!\n",
      "[737/1203] Fetching abstract for PMID: 41197564\n",
      "> Fetched abstract!!\n",
      "[738/1203] Fetching abstract for PMID: 41197539\n",
      "> Fetched abstract!!\n",
      "[739/1203] Fetching abstract for PMID: 41197528\n",
      "> Fetched abstract!!\n",
      "[740/1203] Fetching abstract for PMID: 41197527\n",
      "> Fetched abstract!!\n",
      "[741/1203] Fetching abstract for PMID: 41197523\n",
      "> Fetched abstract!!\n",
      "[742/1203] Fetching abstract for PMID: 41197522\n",
      "> Fetched abstract!!\n",
      "[743/1203] Fetching abstract for PMID: 41197474\n",
      "> Fetched abstract!!\n",
      "[744/1203] Fetching abstract for PMID: 41197473\n",
      "> Fetched abstract!!\n",
      "[745/1203] Fetching abstract for PMID: 41197471\n",
      "> Fetched abstract!!\n",
      "[746/1203] Fetching abstract for PMID: 41197470\n",
      "> Fetched abstract!!\n",
      "[747/1203] Fetching abstract for PMID: 41197469\n",
      "> Fetched abstract!!\n",
      "[748/1203] Fetching abstract for PMID: 41197451\n",
      "> Fetched abstract!!\n",
      "[749/1203] Fetching abstract for PMID: 41197449\n",
      "> Fetched abstract!!\n",
      "[750/1203] Fetching abstract for PMID: 41197437\n",
      "> Fetched abstract!!\n",
      "[751/1203] Fetching abstract for PMID: 41197435\n",
      "> Fetched abstract!!\n",
      "[752/1203] Fetching abstract for PMID: 41197434\n",
      "> Fetched abstract!!\n",
      "[753/1203] Fetching abstract for PMID: 41197433\n",
      "> Fetched abstract!!\n",
      "[754/1203] Fetching abstract for PMID: 41197432\n",
      "> Fetched abstract!!\n",
      "[755/1203] Fetching abstract for PMID: 41197423\n",
      "> Fetched abstract!!\n",
      "[756/1203] Fetching abstract for PMID: 41197411\n",
      "> Fetched abstract!!\n",
      "[757/1203] Fetching abstract for PMID: 41197409\n",
      "> Fetched abstract!!\n",
      "[758/1203] Fetching abstract for PMID: 41197407\n",
      "> Fetched abstract!!\n",
      "[759/1203] Fetching abstract for PMID: 41197406\n",
      "> Fetched abstract!!\n",
      "[760/1203] Fetching abstract for PMID: 41197404\n",
      "> Fetched abstract!!\n",
      "[761/1203] Fetching abstract for PMID: 41197401\n",
      "> Fetched abstract!!\n",
      "[762/1203] Fetching abstract for PMID: 41197387\n",
      "> Fetched abstract!!\n",
      "[763/1203] Fetching abstract for PMID: 41197386\n",
      "> Fetched abstract!!\n",
      "[764/1203] Fetching abstract for PMID: 41197383\n",
      "> Fetched abstract!!\n",
      "[765/1203] Fetching abstract for PMID: 41197382\n",
      "> Fetched abstract!!\n",
      "[766/1203] Fetching abstract for PMID: 41197381\n",
      "> Fetched abstract!!\n",
      "[767/1203] Fetching abstract for PMID: 41197375\n",
      "> Fetched abstract!!\n",
      "[768/1203] Fetching abstract for PMID: 41197344\n",
      "> Fetched abstract!!\n",
      "[769/1203] Fetching abstract for PMID: 41197335\n",
      "> Fetched abstract!!\n",
      "[770/1203] Fetching abstract for PMID: 41197301\n",
      "> Fetched abstract!!\n",
      "[771/1203] Fetching abstract for PMID: 41197289\n",
      "> Fetched abstract!!\n",
      "[772/1203] Fetching abstract for PMID: 41197273\n",
      "> Fetched abstract!!\n",
      "[773/1203] Fetching abstract for PMID: 41197251\n",
      "> Fetched abstract!!\n",
      "[774/1203] Fetching abstract for PMID: 41197250\n",
      "> Fetched abstract!!\n",
      "[775/1203] Fetching abstract for PMID: 41197230\n",
      "> Fetched abstract!!\n",
      "[776/1203] Fetching abstract for PMID: 41197209\n",
      "> Fetched abstract!!\n",
      "[777/1203] Fetching abstract for PMID: 41197206\n",
      "> Fetched abstract!!\n",
      "[778/1203] Fetching abstract for PMID: 41197204\n",
      "> Fetched abstract!!\n",
      "[779/1203] Fetching abstract for PMID: 41197203\n",
      "> Fetched abstract!!\n",
      "[780/1203] Fetching abstract for PMID: 41197201\n",
      "> Fetched abstract!!\n",
      "[781/1203] Fetching abstract for PMID: 41197191\n",
      "> Fetched abstract!!\n",
      "[782/1203] Fetching abstract for PMID: 41197185\n",
      "> Fetched abstract!!\n",
      "[783/1203] Fetching abstract for PMID: 41197182\n",
      "> Fetched abstract!!\n",
      "[784/1203] Fetching abstract for PMID: 41197143\n",
      "> Fetched abstract!!\n",
      "[785/1203] Fetching abstract for PMID: 41197127\n",
      "> Fetched abstract!!\n",
      "[786/1203] Fetching abstract for PMID: 41197113\n",
      "> Fetched abstract!!\n",
      "[787/1203] Fetching abstract for PMID: 41197110\n",
      "> Fetched abstract!!\n",
      "[788/1203] Fetching abstract for PMID: 41197089\n",
      "> Fetched abstract!!\n",
      "[789/1203] Fetching abstract for PMID: 41197086\n",
      "> Fetched abstract!!\n",
      "[790/1203] Fetching abstract for PMID: 41197082\n",
      "> Fetched abstract!!\n",
      "[791/1203] Fetching abstract for PMID: 41197081\n",
      "> Fetched abstract!!\n",
      "[792/1203] Fetching abstract for PMID: 41197079\n",
      "> Fetched abstract!!\n",
      "[793/1203] Fetching abstract for PMID: 41197067\n",
      "> Fetched abstract!!\n",
      "[794/1203] Fetching abstract for PMID: 41197056\n",
      "> Fetched abstract!!\n",
      "[795/1203] Fetching abstract for PMID: 41197032\n",
      "> Fetched abstract!!\n",
      "[796/1203] Fetching abstract for PMID: 41197028\n",
      "> Fetched abstract!!\n",
      "[797/1203] Fetching abstract for PMID: 41196952\n",
      "> Fetched abstract!!\n",
      "[798/1203] Fetching abstract for PMID: 41196951\n",
      "> Fetched abstract!!\n",
      "[799/1203] Fetching abstract for PMID: 41196948\n",
      "> Fetched abstract!!\n",
      "[800/1203] Fetching abstract for PMID: 41196941\n",
      "> Fetched abstract!!\n",
      "[801/1203] Fetching abstract for PMID: 41196940\n",
      "> Fetched abstract!!\n",
      "[802/1203] Fetching abstract for PMID: 41196900\n",
      "> Fetched abstract!!\n",
      "[803/1203] Fetching abstract for PMID: 41196876\n",
      "> Fetched abstract!!\n",
      "[804/1203] Fetching abstract for PMID: 41196839\n",
      "> Fetched abstract!!\n",
      "[805/1203] Fetching abstract for PMID: 41196829\n",
      "> Fetched abstract!!\n",
      "[806/1203] Fetching abstract for PMID: 41196813\n",
      "> Fetched abstract!!\n",
      "[807/1203] Fetching abstract for PMID: 41196808\n",
      "> Fetched abstract!!\n",
      "[808/1203] Fetching abstract for PMID: 41196806\n",
      "> Fetched abstract!!\n",
      "[809/1203] Fetching abstract for PMID: 41196727\n",
      "> Fetched abstract!!\n",
      "[810/1203] Fetching abstract for PMID: 41196722\n",
      "> Fetched abstract!!\n",
      "[811/1203] Fetching abstract for PMID: 41196717\n",
      "> Fetched abstract!!\n",
      "[812/1203] Fetching abstract for PMID: 41196694\n",
      "> Fetched abstract!!\n",
      "[813/1203] Fetching abstract for PMID: 41196684\n",
      "> Fetched abstract!!\n",
      "[814/1203] Fetching abstract for PMID: 41196681\n",
      "> Fetched abstract!!\n",
      "[815/1203] Fetching abstract for PMID: 41196680\n",
      "> Fetched abstract!!\n",
      "[816/1203] Fetching abstract for PMID: 41196678\n",
      "> Fetched abstract!!\n",
      "[817/1203] Fetching abstract for PMID: 41196676\n",
      "> Fetched abstract!!\n",
      "[818/1203] Fetching abstract for PMID: 41196671\n",
      "> Fetched abstract!!\n",
      "[819/1203] Fetching abstract for PMID: 41196658\n",
      "> Fetched abstract!!\n",
      "[820/1203] Fetching abstract for PMID: 41196646\n",
      "> Fetched abstract!!\n",
      "[821/1203] Fetching abstract for PMID: 41196643\n",
      "> Fetched abstract!!\n",
      "[822/1203] Fetching abstract for PMID: 41196637\n",
      "> Fetched abstract!!\n",
      "[823/1203] Fetching abstract for PMID: 41196636\n",
      "> Fetched abstract!!\n",
      "[824/1203] Fetching abstract for PMID: 41196635\n",
      "> Fetched abstract!!\n",
      "[825/1203] Fetching abstract for PMID: 41196631\n",
      "> Fetched abstract!!\n",
      "[826/1203] Fetching abstract for PMID: 41196615\n",
      "> Fetched abstract!!\n",
      "[827/1203] Fetching abstract for PMID: 41196612\n",
      "> Fetched abstract!!\n",
      "[828/1203] Fetching abstract for PMID: 41196599\n",
      "> Fetched abstract!!\n",
      "[829/1203] Fetching abstract for PMID: 41196597\n",
      "> Fetched abstract!!\n",
      "[830/1203] Fetching abstract for PMID: 41196576\n",
      "> Fetched abstract!!\n",
      "[831/1203] Fetching abstract for PMID: 41196575\n",
      "> Fetched abstract!!\n",
      "[832/1203] Fetching abstract for PMID: 41196565\n",
      "> Fetched abstract!!\n",
      "[833/1203] Fetching abstract for PMID: 41196538\n",
      "> Fetched abstract!!\n",
      "[834/1203] Fetching abstract for PMID: 41196536\n",
      "> Fetched abstract!!\n",
      "[835/1203] Fetching abstract for PMID: 41196535\n",
      "> Fetched abstract!!\n",
      "[836/1203] Fetching abstract for PMID: 41196534\n",
      "> Fetched abstract!!\n",
      "[837/1203] Fetching abstract for PMID: 41196526\n",
      "> Fetched abstract!!\n",
      "[838/1203] Fetching abstract for PMID: 41196523\n",
      "> Fetched abstract!!\n",
      "[839/1203] Fetching abstract for PMID: 41196519\n",
      "> Fetched abstract!!\n",
      "[840/1203] Fetching abstract for PMID: 41196510\n",
      "> Fetched abstract!!\n",
      "[841/1203] Fetching abstract for PMID: 41196508\n",
      "> Fetched abstract!!\n",
      "[842/1203] Fetching abstract for PMID: 41196507\n",
      "> Fetched abstract!!\n",
      "[843/1203] Fetching abstract for PMID: 41196498\n",
      "> Fetched abstract!!\n",
      "[844/1203] Fetching abstract for PMID: 41196493\n",
      "> Fetched abstract!!\n",
      "[845/1203] Fetching abstract for PMID: 41196489\n",
      "> Fetched abstract!!\n",
      "[846/1203] Fetching abstract for PMID: 41196469\n",
      "> Fetched abstract!!\n",
      "[847/1203] Fetching abstract for PMID: 41196468\n",
      "> Fetched abstract!!\n",
      "[848/1203] Fetching abstract for PMID: 41196460\n",
      "> Fetched abstract!!\n",
      "[849/1203] Fetching abstract for PMID: 41196459\n",
      "> Fetched abstract!!\n",
      "[850/1203] Fetching abstract for PMID: 41196455\n",
      "> Fetched abstract!!\n",
      "[851/1203] Fetching abstract for PMID: 41196451\n",
      "> Fetched abstract!!\n",
      "[852/1203] Fetching abstract for PMID: 41196435\n",
      "> Fetched abstract!!\n",
      "[853/1203] Fetching abstract for PMID: 41196428\n",
      "> Fetched abstract!!\n",
      "[854/1203] Fetching abstract for PMID: 41196424\n",
      "> Fetched abstract!!\n",
      "[855/1203] Fetching abstract for PMID: 41196414\n",
      "> Fetched abstract!!\n",
      "[856/1203] Fetching abstract for PMID: 41196404\n",
      "> Fetched abstract!!\n",
      "[857/1203] Fetching abstract for PMID: 41196390\n",
      "> Fetched abstract!!\n",
      "[858/1203] Fetching abstract for PMID: 41196378\n",
      "> Fetched abstract!!\n",
      "[859/1203] Fetching abstract for PMID: 41196374\n",
      "> Fetched abstract!!\n",
      "[860/1203] Fetching abstract for PMID: 41196365\n",
      "> Fetched abstract!!\n",
      "[861/1203] Fetching abstract for PMID: 41196363\n",
      "> Fetched abstract!!\n",
      "[862/1203] Fetching abstract for PMID: 41196356\n",
      "> Fetched abstract!!\n",
      "[863/1203] Fetching abstract for PMID: 41196351\n",
      "> Fetched abstract!!\n",
      "[864/1203] Fetching abstract for PMID: 41196342\n",
      "> Fetched abstract!!\n",
      "[865/1203] Fetching abstract for PMID: 41196341\n",
      "> Fetched abstract!!\n",
      "[866/1203] Fetching abstract for PMID: 41196329\n",
      "> Fetched abstract!!\n",
      "[867/1203] Fetching abstract for PMID: 41196321\n",
      "> Fetched abstract!!\n",
      "[868/1203] Fetching abstract for PMID: 41196306\n",
      "> Fetched abstract!!\n",
      "[869/1203] Fetching abstract for PMID: 41196286\n",
      "> Fetched abstract!!\n",
      "[870/1203] Fetching abstract for PMID: 41196284\n",
      "> Fetched abstract!!\n",
      "[871/1203] Fetching abstract for PMID: 41196234\n",
      "> Fetched abstract!!\n",
      "[872/1203] Fetching abstract for PMID: 41196139\n",
      "> Fetched abstract!!\n",
      "[873/1203] Fetching abstract for PMID: 41196138\n",
      "> Fetched abstract!!\n",
      "[874/1203] Fetching abstract for PMID: 41196112\n",
      "> Fetched abstract!!\n",
      "[875/1203] Fetching abstract for PMID: 41196104\n",
      "> Fetched abstract!!\n",
      "[876/1203] Fetching abstract for PMID: 41196089\n",
      "> Fetched abstract!!\n",
      "[877/1203] Fetching abstract for PMID: 41196086\n",
      "> Fetched abstract!!\n",
      "[878/1203] Fetching abstract for PMID: 41196084\n",
      "> Fetched abstract!!\n",
      "[879/1203] Fetching abstract for PMID: 41196083\n",
      "> Fetched abstract!!\n",
      "[880/1203] Fetching abstract for PMID: 41196081\n",
      "> Fetched abstract!!\n",
      "[881/1203] Fetching abstract for PMID: 41196080\n",
      "> Fetched abstract!!\n",
      "[882/1203] Fetching abstract for PMID: 41196079\n",
      "> Fetched abstract!!\n",
      "[883/1203] Fetching abstract for PMID: 41196078\n",
      "> Fetched abstract!!\n",
      "[884/1203] Fetching abstract for PMID: 41196077\n",
      "> Fetched abstract!!\n",
      "[885/1203] Fetching abstract for PMID: 41196057\n",
      "> Fetched abstract!!\n",
      "[886/1203] Fetching abstract for PMID: 41196054\n",
      "> Fetched abstract!!\n",
      "[887/1203] Fetching abstract for PMID: 41196052\n",
      "> Fetched abstract!!\n",
      "[888/1203] Fetching abstract for PMID: 41196049\n",
      "> Fetched abstract!!\n",
      "[889/1203] Fetching abstract for PMID: 41196039\n",
      "> Fetched abstract!!\n",
      "[890/1203] Fetching abstract for PMID: 41196029\n",
      "> Fetched abstract!!\n",
      "[891/1203] Fetching abstract for PMID: 41196022\n",
      "> Fetched abstract!!\n",
      "[892/1203] Fetching abstract for PMID: 41196020\n",
      "> Fetched abstract!!\n",
      "[893/1203] Fetching abstract for PMID: 41196016\n",
      "> Fetched abstract!!\n",
      "[894/1203] Fetching abstract for PMID: 41196015\n",
      "> Fetched abstract!!\n",
      "[895/1203] Fetching abstract for PMID: 41196005\n",
      "> Fetched abstract!!\n",
      "[896/1203] Fetching abstract for PMID: 41196004\n",
      "> Fetched abstract!!\n",
      "[897/1203] Fetching abstract for PMID: 41196003\n",
      "> Fetched abstract!!\n",
      "[898/1203] Fetching abstract for PMID: 41196001\n",
      "> Fetched abstract!!\n",
      "[899/1203] Fetching abstract for PMID: 41195991\n",
      "> Fetched abstract!!\n",
      "[900/1203] Fetching abstract for PMID: 41195987\n",
      "> Fetched abstract!!\n",
      "[901/1203] Fetching abstract for PMID: 41195968\n",
      "> Fetched abstract!!\n",
      "[902/1203] Fetching abstract for PMID: 41195964\n",
      "> Fetched abstract!!\n",
      "[903/1203] Fetching abstract for PMID: 41195962\n",
      "> Fetched abstract!!\n",
      "[904/1203] Fetching abstract for PMID: 41195957\n",
      "> Fetched abstract!!\n",
      "[905/1203] Fetching abstract for PMID: 41195952\n",
      "> Fetched abstract!!\n",
      "[906/1203] Fetching abstract for PMID: 41195911\n",
      "> Fetched abstract!!\n",
      "[907/1203] Fetching abstract for PMID: 41195893\n",
      "> Fetched abstract!!\n",
      "[908/1203] Fetching abstract for PMID: 41195890\n",
      "> Fetched abstract!!\n",
      "[909/1203] Fetching abstract for PMID: 41195885\n",
      "> Fetched abstract!!\n",
      "[910/1203] Fetching abstract for PMID: 41195871\n",
      "> Fetched abstract!!\n",
      "[911/1203] Fetching abstract for PMID: 41195848\n",
      "> Fetched abstract!!\n",
      "[912/1203] Fetching abstract for PMID: 41195835\n",
      "> Fetched abstract!!\n",
      "[913/1203] Fetching abstract for PMID: 41195833\n",
      "> Fetched abstract!!\n",
      "[914/1203] Fetching abstract for PMID: 41195832\n",
      "> Fetched abstract!!\n",
      "[915/1203] Fetching abstract for PMID: 41195830\n",
      "> Fetched abstract!!\n",
      "[916/1203] Fetching abstract for PMID: 41195790\n",
      "> Fetched abstract!!\n",
      "[917/1203] Fetching abstract for PMID: 41195756\n",
      "> Fetched abstract!!\n",
      "[918/1203] Fetching abstract for PMID: 41195742\n",
      "> Fetched abstract!!\n",
      "[919/1203] Fetching abstract for PMID: 41195699\n",
      "> Fetched abstract!!\n",
      "[920/1203] Fetching abstract for PMID: 41195695\n",
      "> Fetched abstract!!\n",
      "[921/1203] Fetching abstract for PMID: 41195686\n",
      "> Fetched abstract!!\n",
      "[922/1203] Fetching abstract for PMID: 41195681\n",
      "> Fetched abstract!!\n",
      "[923/1203] Fetching abstract for PMID: 41195663\n",
      "> Fetched abstract!!\n",
      "[924/1203] Fetching abstract for PMID: 41195660\n",
      "> Fetched abstract!!\n",
      "[925/1203] Fetching abstract for PMID: 41195653\n",
      "> Fetched abstract!!\n",
      "[926/1203] Fetching abstract for PMID: 41195642\n",
      "> Fetched abstract!!\n",
      "[927/1203] Fetching abstract for PMID: 41195639\n",
      "> Fetched abstract!!\n",
      "[928/1203] Fetching abstract for PMID: 41195636\n",
      "> Fetched abstract!!\n",
      "[929/1203] Fetching abstract for PMID: 41195635\n",
      "> Fetched abstract!!\n",
      "[930/1203] Fetching abstract for PMID: 41195619\n",
      "> Fetched abstract!!\n",
      "[931/1203] Fetching abstract for PMID: 41195615\n",
      "> Fetched abstract!!\n",
      "[932/1203] Fetching abstract for PMID: 41195614\n",
      "> Fetched abstract!!\n",
      "[933/1203] Fetching abstract for PMID: 41195613\n",
      "> Fetched abstract!!\n",
      "[934/1203] Fetching abstract for PMID: 41195595\n",
      "> Fetched abstract!!\n",
      "[935/1203] Fetching abstract for PMID: 41195591\n",
      "> Fetched abstract!!\n",
      "[936/1203] Fetching abstract for PMID: 41195584\n",
      "> Fetched abstract!!\n",
      "[937/1203] Fetching abstract for PMID: 41195560\n",
      "> Fetched abstract!!\n",
      "[938/1203] Fetching abstract for PMID: 41195557\n",
      "> Fetched abstract!!\n",
      "[939/1203] Fetching abstract for PMID: 41195551\n",
      "> Fetched abstract!!\n",
      "[940/1203] Fetching abstract for PMID: 41195538\n",
      "> Fetched abstract!!\n",
      "[941/1203] Fetching abstract for PMID: 41195534\n",
      "> Fetched abstract!!\n",
      "[942/1203] Fetching abstract for PMID: 41195524\n",
      "> Fetched abstract!!\n",
      "[943/1203] Fetching abstract for PMID: 41195487\n",
      "> Fetched abstract!!\n",
      "[944/1203] Fetching abstract for PMID: 41195486\n",
      "> Fetched abstract!!\n",
      "[945/1203] Fetching abstract for PMID: 41195479\n",
      "> Fetched abstract!!\n",
      "[946/1203] Fetching abstract for PMID: 41195472\n",
      "> Fetched abstract!!\n",
      "[947/1203] Fetching abstract for PMID: 41195461\n",
      "> Fetched abstract!!\n",
      "[948/1203] Fetching abstract for PMID: 41195443\n",
      "> Fetched abstract!!\n",
      "[949/1203] Fetching abstract for PMID: 41195440\n",
      "> Fetched abstract!!\n",
      "[950/1203] Fetching abstract for PMID: 41195436\n",
      "> Fetched abstract!!\n",
      "[951/1203] Fetching abstract for PMID: 41195424\n",
      "> Fetched abstract!!\n",
      "[952/1203] Fetching abstract for PMID: 41195420\n",
      "> Fetched abstract!!\n",
      "[953/1203] Fetching abstract for PMID: 41195406\n",
      "> Fetched abstract!!\n",
      "[954/1203] Fetching abstract for PMID: 41195396\n",
      "> Fetched abstract!!\n",
      "[955/1203] Fetching abstract for PMID: 41195370\n",
      "> Fetched abstract!!\n",
      "[956/1203] Fetching abstract for PMID: 41195364\n",
      "> Fetched abstract!!\n",
      "[957/1203] Fetching abstract for PMID: 41195359\n",
      "> Fetched abstract!!\n",
      "[958/1203] Fetching abstract for PMID: 41195336\n",
      "> Fetched abstract!!\n",
      "[959/1203] Fetching abstract for PMID: 41195326\n",
      "> Fetched abstract!!\n",
      "[960/1203] Fetching abstract for PMID: 41195323\n",
      "> Fetched abstract!!\n",
      "[961/1203] Fetching abstract for PMID: 41195294\n",
      "> Fetched abstract!!\n",
      "[962/1203] Fetching abstract for PMID: 41195279\n",
      "> Fetched abstract!!\n",
      "[963/1203] Fetching abstract for PMID: 41195278\n",
      "> Fetched abstract!!\n",
      "[964/1203] Fetching abstract for PMID: 41195277\n",
      "> Fetched abstract!!\n",
      "[965/1203] Fetching abstract for PMID: 41195276\n",
      "> Fetched abstract!!\n",
      "[966/1203] Fetching abstract for PMID: 41195275\n",
      "> Fetched abstract!!\n",
      "[967/1203] Fetching abstract for PMID: 41195273\n",
      "> Fetched abstract!!\n",
      "[968/1203] Fetching abstract for PMID: 41195272\n",
      "> Fetched abstract!!\n",
      "[969/1203] Fetching abstract for PMID: 41195271\n",
      "> Fetched abstract!!\n",
      "[970/1203] Fetching abstract for PMID: 41195270\n",
      "> Fetched abstract!!\n",
      "[971/1203] Fetching abstract for PMID: 41195268\n",
      "> Fetched abstract!!\n",
      "[972/1203] Fetching abstract for PMID: 41195267\n",
      "> Fetched abstract!!\n",
      "[973/1203] Fetching abstract for PMID: 41195265\n",
      "> Fetched abstract!!\n",
      "[974/1203] Fetching abstract for PMID: 41195264\n",
      "> Fetched abstract!!\n",
      "[975/1203] Fetching abstract for PMID: 41195263\n",
      "> Fetched abstract!!\n",
      "[976/1203] Fetching abstract for PMID: 41195259\n",
      "> Fetched abstract!!\n",
      "[977/1203] Fetching abstract for PMID: 41195234\n",
      "> Fetched abstract!!\n",
      "[978/1203] Fetching abstract for PMID: 41195221\n",
      "> Fetched abstract!!\n",
      "[979/1203] Fetching abstract for PMID: 41195187\n",
      "> Fetched abstract!!\n",
      "[980/1203] Fetching abstract for PMID: 41195169\n",
      "> Fetched abstract!!\n",
      "[981/1203] Fetching abstract for PMID: 41195107\n",
      "> Fetched abstract!!\n",
      "[982/1203] Fetching abstract for PMID: 41195106\n",
      "> Fetched abstract!!\n",
      "[983/1203] Fetching abstract for PMID: 41195104\n",
      "> Fetched abstract!!\n",
      "[984/1203] Fetching abstract for PMID: 41195102\n",
      "> Fetched abstract!!\n",
      "[985/1203] Fetching abstract for PMID: 41195101\n",
      "> Fetched abstract!!\n",
      "[986/1203] Fetching abstract for PMID: 41195099\n",
      "> Fetched abstract!!\n",
      "[987/1203] Fetching abstract for PMID: 41195097\n",
      "> Fetched abstract!!\n",
      "[988/1203] Fetching abstract for PMID: 41195096\n",
      "> Fetched abstract!!\n",
      "[989/1203] Fetching abstract for PMID: 41195095\n",
      "> Fetched abstract!!\n",
      "[990/1203] Fetching abstract for PMID: 41195093\n",
      "> Fetched abstract!!\n",
      "[991/1203] Fetching abstract for PMID: 41195046\n",
      "> Fetched abstract!!\n",
      "[992/1203] Fetching abstract for PMID: 41195031\n",
      "> Fetched abstract!!\n",
      "[993/1203] Fetching abstract for PMID: 41195024\n",
      "> Fetched abstract!!\n",
      "[994/1203] Fetching abstract for PMID: 41195022\n",
      "> Fetched abstract!!\n",
      "[995/1203] Fetching abstract for PMID: 41195007\n",
      "> Fetched abstract!!\n",
      "[996/1203] Fetching abstract for PMID: 41195006\n",
      "> Fetched abstract!!\n",
      "[997/1203] Fetching abstract for PMID: 41194999\n",
      "> Fetched abstract!!\n",
      "[998/1203] Fetching abstract for PMID: 41194995\n",
      "> Fetched abstract!!\n",
      "[999/1203] Fetching abstract for PMID: 41194992\n",
      "> Fetched abstract!!\n",
      "[1000/1203] Fetching abstract for PMID: 41194987\n",
      "> Fetched abstract!!\n",
      "[1001/1203] Fetching abstract for PMID: 41194971\n",
      "> Fetched abstract!!\n",
      "[1002/1203] Fetching abstract for PMID: 41194934\n",
      "> Fetched abstract!!\n",
      "[1003/1203] Fetching abstract for PMID: 41194933\n",
      "> Fetched abstract!!\n",
      "[1004/1203] Fetching abstract for PMID: 41194932\n",
      "> Fetched abstract!!\n",
      "[1005/1203] Fetching abstract for PMID: 41194931\n",
      "> Fetched abstract!!\n",
      "[1006/1203] Fetching abstract for PMID: 41194930\n",
      "> Fetched abstract!!\n",
      "[1007/1203] Fetching abstract for PMID: 41194925\n",
      "> Fetched abstract!!\n",
      "[1008/1203] Fetching abstract for PMID: 41194922\n",
      "> Fetched abstract!!\n",
      "[1009/1203] Fetching abstract for PMID: 41194917\n",
      "> Fetched abstract!!\n",
      "[1010/1203] Fetching abstract for PMID: 41194886\n",
      "> Fetched abstract!!\n",
      "[1011/1203] Fetching abstract for PMID: 41194885\n",
      "> Fetched abstract!!\n",
      "[1012/1203] Fetching abstract for PMID: 41194879\n",
      "> Fetched abstract!!\n",
      "[1013/1203] Fetching abstract for PMID: 41194870\n",
      "> Fetched abstract!!\n",
      "[1014/1203] Fetching abstract for PMID: 41194863\n",
      "> Fetched abstract!!\n",
      "[1015/1203] Fetching abstract for PMID: 41194856\n",
      "> Fetched abstract!!\n",
      "[1016/1203] Fetching abstract for PMID: 41194833\n",
      "> Fetched abstract!!\n",
      "[1017/1203] Fetching abstract for PMID: 41194828\n",
      "> Fetched abstract!!\n",
      "[1018/1203] Fetching abstract for PMID: 41194824\n",
      "> Fetched abstract!!\n",
      "[1019/1203] Fetching abstract for PMID: 41194819\n",
      "> Fetched abstract!!\n",
      "[1020/1203] Fetching abstract for PMID: 41194812\n",
      "> Fetched abstract!!\n",
      "[1021/1203] Fetching abstract for PMID: 41194809\n",
      "> Fetched abstract!!\n",
      "[1022/1203] Fetching abstract for PMID: 41194787\n",
      "> Fetched abstract!!\n",
      "[1023/1203] Fetching abstract for PMID: 41194782\n",
      "> Fetched abstract!!\n",
      "[1024/1203] Fetching abstract for PMID: 41194763\n",
      "> Fetched abstract!!\n",
      "[1025/1203] Fetching abstract for PMID: 41194762\n",
      "> Fetched abstract!!\n",
      "[1026/1203] Fetching abstract for PMID: 41194758\n",
      "> Fetched abstract!!\n",
      "[1027/1203] Fetching abstract for PMID: 41194705\n",
      "> Fetched abstract!!\n",
      "[1028/1203] Fetching abstract for PMID: 41194702\n",
      "> Fetched abstract!!\n",
      "[1029/1203] Fetching abstract for PMID: 41194701\n",
      "> Fetched abstract!!\n",
      "[1030/1203] Fetching abstract for PMID: 41194698\n",
      "> Fetched abstract!!\n",
      "[1031/1203] Fetching abstract for PMID: 41194695\n",
      "> Fetched abstract!!\n",
      "[1032/1203] Fetching abstract for PMID: 41194693\n",
      "> Fetched abstract!!\n",
      "[1033/1203] Fetching abstract for PMID: 41194688\n",
      "> Fetched abstract!!\n",
      "[1034/1203] Fetching abstract for PMID: 41194666\n",
      "> Fetched abstract!!\n",
      "[1035/1203] Fetching abstract for PMID: 41194664\n",
      "> Fetched abstract!!\n",
      "[1036/1203] Fetching abstract for PMID: 41194660\n",
      "> Fetched abstract!!\n",
      "[1037/1203] Fetching abstract for PMID: 41194605\n",
      "> Fetched abstract!!\n",
      "[1038/1203] Fetching abstract for PMID: 41194602\n",
      "> Fetched abstract!!\n",
      "[1039/1203] Fetching abstract for PMID: 41194597\n",
      "> Fetched abstract!!\n",
      "[1040/1203] Fetching abstract for PMID: 41194592\n",
      "> Fetched abstract!!\n",
      "[1041/1203] Fetching abstract for PMID: 41194587\n",
      "> Fetched abstract!!\n",
      "[1042/1203] Fetching abstract for PMID: 41194585\n",
      "> Fetched abstract!!\n",
      "[1043/1203] Fetching abstract for PMID: 41194583\n",
      "> Fetched abstract!!\n",
      "[1044/1203] Fetching abstract for PMID: 41194556\n",
      "> Fetched abstract!!\n",
      "[1045/1203] Fetching abstract for PMID: 41194524\n",
      "> Fetched abstract!!\n",
      "[1046/1203] Fetching abstract for PMID: 41194508\n",
      "> Fetched abstract!!\n",
      "[1047/1203] Fetching abstract for PMID: 41194506\n",
      "> Fetched abstract!!\n",
      "[1048/1203] Fetching abstract for PMID: 41194504\n",
      "> Fetched abstract!!\n",
      "[1049/1203] Fetching abstract for PMID: 41194500\n",
      "> Fetched abstract!!\n",
      "[1050/1203] Fetching abstract for PMID: 41194455\n",
      "> Fetched abstract!!\n",
      "[1051/1203] Fetching abstract for PMID: 41194451\n",
      "> Fetched abstract!!\n",
      "[1052/1203] Fetching abstract for PMID: 41194439\n",
      "> Fetched abstract!!\n",
      "[1053/1203] Fetching abstract for PMID: 41194437\n",
      "> Fetched abstract!!\n",
      "[1054/1203] Fetching abstract for PMID: 41194413\n",
      "> Fetched abstract!!\n",
      "[1055/1203] Fetching abstract for PMID: 41194371\n",
      "> Fetched abstract!!\n",
      "[1056/1203] Fetching abstract for PMID: 41194370\n",
      "> Fetched abstract!!\n",
      "[1057/1203] Fetching abstract for PMID: 41194367\n",
      "> Fetched abstract!!\n",
      "[1058/1203] Fetching abstract for PMID: 41194366\n",
      "> Fetched abstract!!\n",
      "[1059/1203] Fetching abstract for PMID: 41194320\n",
      "> Fetched abstract!!\n",
      "[1060/1203] Fetching abstract for PMID: 41194308\n",
      "> Fetched abstract!!\n",
      "[1061/1203] Fetching abstract for PMID: 41194290\n",
      "> Fetched abstract!!\n",
      "[1062/1203] Fetching abstract for PMID: 41194282\n",
      "> Fetched abstract!!\n",
      "[1063/1203] Fetching abstract for PMID: 41194281\n",
      "> Fetched abstract!!\n",
      "[1064/1203] Fetching abstract for PMID: 41194272\n",
      "> Fetched abstract!!\n",
      "[1065/1203] Fetching abstract for PMID: 41194266\n",
      "> Fetched abstract!!\n",
      "[1066/1203] Fetching abstract for PMID: 41194259\n",
      "> Fetched abstract!!\n",
      "[1067/1203] Fetching abstract for PMID: 41194255\n",
      "> Fetched abstract!!\n",
      "[1068/1203] Fetching abstract for PMID: 41194243\n",
      "> Fetched abstract!!\n",
      "[1069/1203] Fetching abstract for PMID: 41194239\n",
      "> Fetched abstract!!\n",
      "[1070/1203] Fetching abstract for PMID: 41194237\n",
      "> Fetched abstract!!\n",
      "[1071/1203] Fetching abstract for PMID: 41194226\n",
      "> Fetched abstract!!\n",
      "[1072/1203] Fetching abstract for PMID: 41194225\n",
      "> Fetched abstract!!\n",
      "[1073/1203] Fetching abstract for PMID: 41194215\n",
      "> Fetched abstract!!\n",
      "[1074/1203] Fetching abstract for PMID: 41194205\n",
      "> Fetched abstract!!\n",
      "[1075/1203] Fetching abstract for PMID: 41194198\n",
      "> Fetched abstract!!\n",
      "[1076/1203] Fetching abstract for PMID: 41194193\n",
      "> Fetched abstract!!\n",
      "[1077/1203] Fetching abstract for PMID: 41194181\n",
      "> Fetched abstract!!\n",
      "[1078/1203] Fetching abstract for PMID: 41194180\n",
      "> Fetched abstract!!\n",
      "[1079/1203] Fetching abstract for PMID: 41194170\n",
      "> Fetched abstract!!\n",
      "[1080/1203] Fetching abstract for PMID: 41194150\n",
      "> Fetched abstract!!\n",
      "[1081/1203] Fetching abstract for PMID: 41194145\n",
      "> Fetched abstract!!\n",
      "[1082/1203] Fetching abstract for PMID: 41194119\n",
      "> Fetched abstract!!\n",
      "[1083/1203] Fetching abstract for PMID: 41194113\n",
      "> Fetched abstract!!\n",
      "[1084/1203] Fetching abstract for PMID: 41194109\n",
      "> Fetched abstract!!\n",
      "[1085/1203] Fetching abstract for PMID: 41194105\n",
      "> Fetched abstract!!\n",
      "[1086/1203] Fetching abstract for PMID: 41194096\n",
      "> Fetched abstract!!\n",
      "[1087/1203] Fetching abstract for PMID: 41194091\n",
      "> Fetched abstract!!\n",
      "[1088/1203] Fetching abstract for PMID: 41194086\n",
      "> Fetched abstract!!\n",
      "[1089/1203] Fetching abstract for PMID: 41194083\n",
      "> Fetched abstract!!\n",
      "[1090/1203] Fetching abstract for PMID: 41194072\n",
      "> Fetched abstract!!\n",
      "[1091/1203] Fetching abstract for PMID: 41194047\n",
      "> Fetched abstract!!\n",
      "[1092/1203] Fetching abstract for PMID: 41194034\n",
      "> Fetched abstract!!\n",
      "[1093/1203] Fetching abstract for PMID: 41194025\n",
      "> Fetched abstract!!\n",
      "[1094/1203] Fetching abstract for PMID: 41194017\n",
      "> Fetched abstract!!\n",
      "[1095/1203] Fetching abstract for PMID: 41194015\n",
      "> Fetched abstract!!\n",
      "[1096/1203] Fetching abstract for PMID: 41194013\n",
      "> Fetched abstract!!\n",
      "[1097/1203] Fetching abstract for PMID: 41193995\n",
      "> Fetched abstract!!\n",
      "[1098/1203] Fetching abstract for PMID: 41193990\n",
      "> Fetched abstract!!\n",
      "[1099/1203] Fetching abstract for PMID: 41193989\n",
      "> Fetched abstract!!\n",
      "[1100/1203] Fetching abstract for PMID: 41193981\n",
      "> Fetched abstract!!\n",
      "[1101/1203] Fetching abstract for PMID: 41193978\n",
      "> Fetched abstract!!\n",
      "[1102/1203] Fetching abstract for PMID: 41193953\n",
      "> Fetched abstract!!\n",
      "[1103/1203] Fetching abstract for PMID: 41193924\n",
      "> Fetched abstract!!\n",
      "[1104/1203] Fetching abstract for PMID: 41193918\n",
      "> Fetched abstract!!\n",
      "[1105/1203] Fetching abstract for PMID: 41193917\n",
      "> Fetched abstract!!\n",
      "[1106/1203] Fetching abstract for PMID: 41193908\n",
      "> Fetched abstract!!\n",
      "[1107/1203] Fetching abstract for PMID: 41193907\n",
      "> Fetched abstract!!\n",
      "[1108/1203] Fetching abstract for PMID: 41193904\n",
      "> Fetched abstract!!\n",
      "[1109/1203] Fetching abstract for PMID: 41193888\n",
      "> Fetched abstract!!\n",
      "[1110/1203] Fetching abstract for PMID: 41193885\n",
      "> Fetched abstract!!\n",
      "[1111/1203] Fetching abstract for PMID: 41193870\n",
      "> Fetched abstract!!\n",
      "[1112/1203] Fetching abstract for PMID: 41193855\n",
      "> Fetched abstract!!\n",
      "[1113/1203] Fetching abstract for PMID: 41193854\n",
      "> Fetched abstract!!\n",
      "[1114/1203] Fetching abstract for PMID: 41193853\n",
      "> Fetched abstract!!\n",
      "[1115/1203] Fetching abstract for PMID: 41193835\n",
      "> Fetched abstract!!\n",
      "[1116/1203] Fetching abstract for PMID: 41193833\n",
      "> Fetched abstract!!\n",
      "[1117/1203] Fetching abstract for PMID: 41193825\n",
      "> Fetched abstract!!\n",
      "[1118/1203] Fetching abstract for PMID: 41193823\n",
      "> Fetched abstract!!\n",
      "[1119/1203] Fetching abstract for PMID: 41193821\n",
      "> Fetched abstract!!\n",
      "[1120/1203] Fetching abstract for PMID: 41193820\n",
      "> Fetched abstract!!\n",
      "[1121/1203] Fetching abstract for PMID: 41193807\n",
      "> Fetched abstract!!\n",
      "[1122/1203] Fetching abstract for PMID: 41193800\n",
      "> Fetched abstract!!\n",
      "[1123/1203] Fetching abstract for PMID: 41193799\n",
      "> Fetched abstract!!\n",
      "[1124/1203] Fetching abstract for PMID: 41193789\n",
      "> Fetched abstract!!\n",
      "[1125/1203] Fetching abstract for PMID: 41193783\n",
      "> Fetched abstract!!\n",
      "[1126/1203] Fetching abstract for PMID: 41193778\n",
      "> Fetched abstract!!\n",
      "[1127/1203] Fetching abstract for PMID: 41193742\n",
      "> Fetched abstract!!\n",
      "[1128/1203] Fetching abstract for PMID: 41193733\n",
      "> Fetched abstract!!\n",
      "[1129/1203] Fetching abstract for PMID: 41193722\n",
      "> Fetched abstract!!\n",
      "[1130/1203] Fetching abstract for PMID: 41193715\n",
      "> Fetched abstract!!\n",
      "[1131/1203] Fetching abstract for PMID: 41193709\n",
      "> Fetched abstract!!\n",
      "[1132/1203] Fetching abstract for PMID: 41193692\n",
      "> Fetched abstract!!\n",
      "[1133/1203] Fetching abstract for PMID: 41193668\n",
      "> Fetched abstract!!\n",
      "[1134/1203] Fetching abstract for PMID: 41193664\n",
      "> Fetched abstract!!\n",
      "[1135/1203] Fetching abstract for PMID: 41193657\n",
      "> Fetched abstract!!\n",
      "[1136/1203] Fetching abstract for PMID: 41193656\n",
      "> Fetched abstract!!\n",
      "[1137/1203] Fetching abstract for PMID: 41193643\n",
      "> Fetched abstract!!\n",
      "[1138/1203] Fetching abstract for PMID: 41193641\n",
      "> Fetched abstract!!\n",
      "[1139/1203] Fetching abstract for PMID: 41193623\n",
      "> Fetched abstract!!\n",
      "[1140/1203] Fetching abstract for PMID: 41193619\n",
      "> Fetched abstract!!\n",
      "[1141/1203] Fetching abstract for PMID: 41193576\n",
      "> Fetched abstract!!\n",
      "[1142/1203] Fetching abstract for PMID: 41193574\n",
      "> Fetched abstract!!\n",
      "[1143/1203] Fetching abstract for PMID: 41193573\n",
      "> Fetched abstract!!\n",
      "[1144/1203] Fetching abstract for PMID: 41193563\n",
      "> Fetched abstract!!\n",
      "[1145/1203] Fetching abstract for PMID: 41193536\n",
      "> Fetched abstract!!\n",
      "[1146/1203] Fetching abstract for PMID: 41193520\n",
      "> Fetched abstract!!\n",
      "[1147/1203] Fetching abstract for PMID: 41193507\n",
      "> Fetched abstract!!\n",
      "[1148/1203] Fetching abstract for PMID: 41193491\n",
      "> Fetched abstract!!\n",
      "[1149/1203] Fetching abstract for PMID: 41193477\n",
      "> Fetched abstract!!\n",
      "[1150/1203] Fetching abstract for PMID: 41193475\n",
      "> Fetched abstract!!\n",
      "[1151/1203] Fetching abstract for PMID: 41193452\n",
      "> Fetched abstract!!\n",
      "[1152/1203] Fetching abstract for PMID: 41193449\n",
      "> Fetched abstract!!\n",
      "[1153/1203] Fetching abstract for PMID: 41193448\n",
      "> Fetched abstract!!\n",
      "[1154/1203] Fetching abstract for PMID: 41193447\n",
      "> Fetched abstract!!\n",
      "[1155/1203] Fetching abstract for PMID: 41193446\n",
      "> Fetched abstract!!\n",
      "[1156/1203] Fetching abstract for PMID: 41193441\n",
      "> Fetched abstract!!\n",
      "[1157/1203] Fetching abstract for PMID: 41193432\n",
      "> Fetched abstract!!\n",
      "[1158/1203] Fetching abstract for PMID: 41193428\n",
      "> Fetched abstract!!\n",
      "[1159/1203] Fetching abstract for PMID: 41193425\n",
      "> Fetched abstract!!\n",
      "[1160/1203] Fetching abstract for PMID: 41193417\n",
      "> Fetched abstract!!\n",
      "[1161/1203] Fetching abstract for PMID: 41193407\n",
      "> Fetched abstract!!\n",
      "[1162/1203] Fetching abstract for PMID: 41193405\n",
      "> Fetched abstract!!\n",
      "[1163/1203] Fetching abstract for PMID: 41193397\n",
      "> Fetched abstract!!\n",
      "[1164/1203] Fetching abstract for PMID: 41193395\n",
      "> Fetched abstract!!\n",
      "[1165/1203] Fetching abstract for PMID: 41193393\n",
      "> Fetched abstract!!\n",
      "[1166/1203] Fetching abstract for PMID: 41193373\n",
      "> Fetched abstract!!\n",
      "[1167/1203] Fetching abstract for PMID: 41193372\n",
      "> Fetched abstract!!\n",
      "[1168/1203] Fetching abstract for PMID: 41193370\n",
      "> Fetched abstract!!\n",
      "[1169/1203] Fetching abstract for PMID: 41193369\n",
      "> Fetched abstract!!\n",
      "[1170/1203] Fetching abstract for PMID: 41193357\n",
      "> Fetched abstract!!\n",
      "[1171/1203] Fetching abstract for PMID: 41193346\n",
      "> Fetched abstract!!\n",
      "[1172/1203] Fetching abstract for PMID: 41193345\n",
      "> Fetched abstract!!\n",
      "[1173/1203] Fetching abstract for PMID: 41193333\n",
      "> Fetched abstract!!\n",
      "[1174/1203] Fetching abstract for PMID: 41193320\n",
      "> Fetched abstract!!\n",
      "[1175/1203] Fetching abstract for PMID: 41193292\n",
      "> Fetched abstract!!\n",
      "[1176/1203] Fetching abstract for PMID: 41193273\n",
      "> Fetched abstract!!\n",
      "[1177/1203] Fetching abstract for PMID: 41193272\n",
      "> Fetched abstract!!\n",
      "[1178/1203] Fetching abstract for PMID: 41193271\n",
      "> Fetched abstract!!\n",
      "[1179/1203] Fetching abstract for PMID: 41193263\n",
      "> Fetched abstract!!\n",
      "[1180/1203] Fetching abstract for PMID: 41193246\n",
      "> Fetched abstract!!\n",
      "[1181/1203] Fetching abstract for PMID: 41193244\n",
      "> Fetched abstract!!\n",
      "[1182/1203] Fetching abstract for PMID: 41193236\n",
      "> Fetched abstract!!\n",
      "[1183/1203] Fetching abstract for PMID: 41193221\n",
      "> Fetched abstract!!\n",
      "[1184/1203] Fetching abstract for PMID: 41193212\n",
      "> Fetched abstract!!\n",
      "[1185/1203] Fetching abstract for PMID: 41193187\n",
      "> Fetched abstract!!\n",
      "[1186/1203] Fetching abstract for PMID: 41193184\n",
      "> Fetched abstract!!\n",
      "[1187/1203] Fetching abstract for PMID: 41193183\n",
      "> Fetched abstract!!\n",
      "[1188/1203] Fetching abstract for PMID: 41193182\n",
      "> Fetched abstract!!\n",
      "[1189/1203] Fetching abstract for PMID: 41193181\n",
      "> Fetched abstract!!\n",
      "[1190/1203] Fetching abstract for PMID: 41193175\n",
      "> Fetched abstract!!\n",
      "[1191/1203] Fetching abstract for PMID: 41193117\n",
      "> Fetched abstract!!\n",
      "[1192/1203] Fetching abstract for PMID: 41193053\n",
      "> Fetched abstract!!\n",
      "[1193/1203] Fetching abstract for PMID: 41193052\n",
      "> Fetched abstract!!\n",
      "[1194/1203] Fetching abstract for PMID: 41193050\n",
      "> Fetched abstract!!\n",
      "[1195/1203] Fetching abstract for PMID: 41193049\n",
      "> Fetched abstract!!\n",
      "[1196/1203] Fetching abstract for PMID: 41193048\n",
      "> Fetched abstract!!\n",
      "[1197/1203] Fetching abstract for PMID: 41193045\n",
      "> Fetched abstract!!\n",
      "[1198/1203] Fetching abstract for PMID: 41193041\n",
      "> Fetched abstract!!\n",
      "[1199/1203] Fetching abstract for PMID: 41193040\n",
      "> Fetched abstract!!\n",
      "[1200/1203] Fetching abstract for PMID: 41193032\n",
      "> Fetched abstract!!\n",
      "[1201/1203] Fetching abstract for PMID: 41193016\n",
      "> Fetched abstract!!\n",
      "[1202/1203] Fetching abstract for PMID: 41192978\n",
      "> Fetched abstract!!\n",
      "[1203/1203] Fetching abstract for PMID: 41192973\n",
      "> Fetched abstract!!\n"
     ]
    }
   ],
   "source": [
    "final_df = pd.read_csv('publications_abstract_pubmed_elsevier.csv')\n",
    "num_with_abstracts = final_df['abstract'].notnull().sum()\n",
    "\n",
    "irrelevant_pmids_list = get_articles_pmids_for_title(\"cancer\", int(num_with_abstracts))\n",
    "irrelevant_abstracts = []\n",
    "for i, pmid in enumerate(irrelevant_pmids_list):\n",
    "\n",
    "    article_info = {\n",
    "        'pmid': pmid,\n",
    "        'abstract': None\n",
    "    }\n",
    "\n",
    "    print(f\"[{i + 1}/{len(irrelevant_pmids_list)}] Fetching abstract for PMID: {pmid}\")\n",
    "    article_info['abstract'] = fetch_abstract_by_pmid(pmid)\n",
    "    irrelevant_abstracts.append(article_info)\n",
    "\n",
    "# Save irrelevant abstracts to a new dataset\n",
    "irrelevant_df = pd.DataFrame(irrelevant_abstracts)\n",
    "irrelevant_df.to_csv(IRRELEVANT_PUBLICATIONS, index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "904e0778",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "pmid",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "abstract",
         "rawType": "object",
         "type": "unknown"
        }
       ],
       "ref": "69a29b32-7614-44bd-b14f-7c352e2408d1",
       "rows": [
        [
         "0",
         "41204822",
         "1. Cell Cycle. 2025 Nov 8:1-23. doi: 10.1080/15384101.2025.2583289. Online ahead\nof  print.\n\nExosomes in hepatocellular carcinoma: involvement in reprogramming the tumor \nmicroenvironment for immune evasion, metastasis, angiogenesis, and drug \nresistance.\n\nGhasemian-Irani M(1)(2), Babaei S(1)(2), Kazemi T(1)(2).\n\nAuthor information:\n(1)Department of Immunology, Faculty of Medicine, Tabriz University of Medical \nSciences, Tabriz, Iran.\n(2)Immunology Research Center, Tabriz University of Medical Sciences, Tabriz, \nIran.\n\nHepatocellular carcinoma (HCC) is a highly aggressive liver cancer, and its \nprogression is significantly influenced by the tumor microenvironment (TME). \nTumor-derived exosomes (TEXs), an important component of the TME, significantly \ninfluence tumor growth by regulating immune responses, facilitating metastasis, \nand enhancing resistance to therapy. These extracellular vesicles (EVs) \ntransport bioactive substances, such as proteins, lipids, and nucleic acids that \npromote interaction between cells in the TME. Recent research indicates that \nHCC-derived exosomes can inhibit immune cell activity, specifically in T cells, \nthus creating an immunosuppressive TME that facilitates tumor immune escape. \nThey also augment metastatic capability by restructuring the extracellular \nmatrix and promoting angiogenesis. Moreover, HCC-derived exosomes have been \nassociated with developing resistance to drug therapy by transferring molecules \nsuch as apoptotic signals and oncogenic microRNAs, circRNAs and lncRNA. \nUnderstanding how HCC-derived exosomes affect immune modulation, metastasis, and \ndrug resistance could yield innovative therapeutic targets to enhance therapy \noutcomes. This review focuses on recent research on the diverse functions of \nTEXs in HCC progression.\n\nDOI: 10.1080/15384101.2025.2583289\nPMID: 41204822"
        ],
        [
         "1",
         "41204804",
         "1. Int J Environ Health Res. 2025 Nov 8:1-12. doi: 10.1080/09603123.2025.2586625.\n Online ahead of print.\n\nGlycyrrhiza glabra L. suppress the proliferation of non-small cell lung cancer \nby inducing necrosis rather than apoptosis despite increasing bax level.\n\nYumrutaş Ö(1), Yumrutaş P(2), Martinez JL(3), Pérez GA(4), Korkmaz M(5), Escobar \nJ(6), Taşdemir D(7), Rios M(8), Parlar A(9).\n\nAuthor information:\n(1)Department of Medical Biology, Faculty of Medicine, University of Adiyaman, \nAdiyaman, Turkey.\n(2)Department of Respiratory Disease and Cancer Biology, Faculty of Medicine, \nGaziantep University, Gaziantep, Turkey.\n(3)Departamento de Ingeniería Metalurgica, Facultad de Ingenieria, University of \nSantiago de Chile, Santiago ,Chile.\n(4)Departamento de Química, Facultad de Ciencias Básicas, Universidad Del \nAtlántico, Barranquilla, Colombia.\n(5)Department of Medical Biology, Faculty of Medicine, Gaziantep Islam Science \nand Technology University, Gaziantep, Turkey.\n(6)Laboratorio de Química Biológica, Facultad de Ciencias, Pontificia \nUniversidad Católica de Valparaíso, Valparaíso, Chile.\n(7)Department of Medical Biochemistry, Faculty of Medicine, Gaziantep \nUniversity, Gaziantep, Turkey.\n(8)Departamento de Biología, Facultad de Química Y Biología, Universidad de \nSantiago de Chile, Santiago, Chile.\n(9)Department of Medical Pharmacology, Faculty of Medicine, Adiyaman University, \nAdiyaman, Turkey.\n\nLung cancer remains the leading cause of cancer-related mortality worldwide, \nhighlighting the urgent need for the development of novel therapeutic agents. \nGlycyrrhiza glabra, a medicinally significant plant, is known for its rich \ncomposition of diverse phytochemicals. However, its role in the induction of \napoptosis in lung cancer cells has not been well elucidated. The present study \naimed to evaluate the apoptosis-associated cytotoxic and antiproliferative \neffects of Glycyrrhiza glabra methanol extract (GGME) on non-small cell lung \ncancer (A549) cells. The viability of GGME-treated A549 cells was assessed using \nan MTT assay, which revealed a significant antiproliferative effect at 200 µg/ml \n(p < 0.05). Morphological changes were observed via phase-contrast inverted \nmicroscopy. To elucidate the mode of cell death, Annexin V/PI staining was \nperformed, and the mRNA levels of the pro-apoptotic Bax and anti-apoptotic Bcl-2 \ngenes were quantified by real-time PCR. Interestingly, while Bax expression \nincreased 4-fold (p < 0.01) compared to the control, Bcl-2 levels remained \nunchanged. Despite this pro-apoptotic shift in gene expression, the cells \npredominantly underwent necrotic cell death rather than apoptosis. Furthermore, \nLC-MS/MS analysis identified vanillic acid, fumaric acid, syringic acid, and \nthymoquinone as the major compounds in GGME. In conclusion, GGME exerts a \nantiproliferative effect primarily through necrosis rather than apoptosis.\n\nDOI: 10.1080/09603123.2025.2586625\nPMID: 41204804"
        ],
        [
         "2",
         "41204791",
         "1. Cancer Med. 2025 Nov;14(21):e71372. doi: 10.1002/cam4.71372.\n\nA Novel Nomogram to Predict Pathological Complete Response in Breast Cancer \nPatients and Identify Candidates Who Might Omit Surgery: A Large Cohort Study.\n\nYe K(1), Liao X(2), Yang W(1), Weng J(1), Dai Y(3), Chen X(4), Liu Y(5), Du \nK(3).\n\nAuthor information:\n(1)Department of Pathology, Zhangzhou Affiliated Hospital of Fujian Medical \nUniversity, Zhangzhou, China.\n(2)Department of Pathology, Sapporo Medical University, Sapporo, Japan.\n(3)Department of Radiation Oncology, Fujian Medical University Xiamen Humanity \nHospital, Xiamen, China.\n(4)Department of Pathology, Fujian Medical University Xiamen Humanity Hospital, \nXiamen, China.\n(5)Department of Pathology, Zhangzhou Traditional Chinese Medical Hospital, \nZhangzhou, China.\n\nPURPOSE: To establish and validate a nomogram for predicting pathological \ncomplete response (pCR) after neoadjuvant therapy (NAT) in breast cancer (BC) \npatients, aiming to identify subgroups potentially suitable for non-surgical \nmanagement.\nMETHODS: Between 2010 and 2015, 4402 BC patients (3037 surgery, 1365 \nnon-surgery) were extracted from the Surveillance, Epidemiology, and End Results \n(SEER) database, with external validation in 339 BC patients from our hospital. \nLogistic regression identified pCR predictors and a nomogram model was \nconstructed. Propensity score matching (PSM) was applied to minimize the effect \nof the imbalance of the prognostic factors between the surgery group and the \nnon-surgery group.\nRESULTS: Univariable and multivariable analyses revealed that age, marital \nstatus, T stage, N stage, differentiation grade, hormone receptor (HR) status, \nhuman epidermal growth factor receptor 2 (HER2) status and chemotherapy were \nsignificant predictors of pCR in the surgery group (all p < 0.05). The area \nunder the receiver operating characteristic curve (AUC) for predicting pCR was \n0.756 (95% CI: 0.736-0.776), 0.717 (95% CI: 0.681-0.754), and 0.744 (95% CI: \n0.687-0.800) in the training, internal validation, and external validation sets, \nrespectively. Non-surgery patients were stratified into high (> 375), medium \n(162.5-375), and low (< 162.5) nomogram score groups, with 5-year OS rates of \n70.7%, 49.2%, and 29.2% (p < 0.05). After PSM, 358 pairs of patients were \nincluded to compare 5-year OS between the surgery group and the non-surgery \ngroup, and the results showed that the 5-year OS of patients stratified by \nhigh-score group, medium-score group and low-score group between the surgery \ngroup and the non-surgery group were 85.4% vs. 72.8%, 78.3% vs. 59.8% and 79.3% \nvs. 20.0% (all p < 0.05).\nCONCLUSION: We successfully established a nomogram for pCR in BC patients. Based \non this predictive model, patients with higher scores may represent potential \ncandidates for non-surgical therapeutic approaches, warranting further \ninvestigation in future studies.\n\n© 2025 The Author(s). Cancer Medicine published by John Wiley & Sons Ltd.\n\nDOI: 10.1002/cam4.71372\nPMID: 41204791 [Indexed for MEDLINE]"
        ],
        [
         "3",
         "41204774",
         "1. IUBMB Life. 2025 Nov;77(11):e70075. doi: 10.1002/iub.70075.\n\n4-Methylumbelliferone Sensitizes Breast Cancer Spheroids to Chemotherapeutic \nTreatment With Epirubicin: New Perspectives for Pharmacological Repositioning.\n\nVitale DL(1)(2), Icardi A(1)(2), Rosales P(1)(2), Morán C(1)(2), Sevic I(1)(2), \nAlaniz LD(1)(2).\n\nAuthor information:\n(1)Laboratorio de Microambiente Tumoral - Centro de Investigaciones Básicas y \nAplicadas (CIBA) - Universidad Nacional del Noroeste de la Provincia de Buenos \nAires, Junín, Argentina.\n(2)Centro de Investigaciones y Transferencia del Noroeste de la Provincia de \nBuenos Aires (CITNOBA) - UNNOBA-UNSAdA-CONICET, Junín, Argentina.\n\nChemoresistance remains a significant hurdle in breast cancer treatment. To \naddress this, we explored the therapeutic potential of combining epirubicin, an \nanthracycline commonly associated with the development of resistance in cancer \ncells and with known cytotoxic effects, with 4-methylumbelliferone, a \nplant-derived coumarin. Our results demonstrated a coactive enhancement of \nepirubicin efficacy when this combination was applied to 3D breast cancer \nmodels, irrespective of molecular subtype. Mechanistically, \n4-methylumbelliferone inhibits synthesis of hyaluronan, a key component of the \ntumor microenvironment that promotes tumor growth and metastasis. By disrupting \nhyaluronan production, this compound facilitated drug penetration into tumor \nspheroids and reduced the expression of drug efflux pumps, thereby increasing \nintracellular drug accumulation. Consequently, the combined treatment led to a \nsignificant reduction in cell viability and promotion of cell death. Our \nfindings suggest that targeting hyaluronan metabolism in conjunction with \nconventional chemotherapy offers a promising strategy to overcome drug \nresistance in breast cancer. This drug repositioning approach not only improves \ntreatment efficacy but also highlights the potential of targeting the tumor \nmicroenvironment to enhance cancer therapy. Further studies are warranted to \nelucidate the underlying mechanisms and to evaluate the clinical translation of \nthis combination therapy.\n\n© 2025 International Union of Biochemistry and Molecular Biology.\n\nDOI: 10.1002/iub.70075\nPMID: 41204774 [Indexed for MEDLINE]"
        ],
        [
         "4",
         "41204772",
         "1. Am J Med Genet A. 2025 Nov 8:e64305. doi: 10.1002/ajmg.a.64305. Online ahead\nof  print.\n\nMalignant Phyllodes Tumor of the Breast in a Young Adult With Neurofibromatosis \nType 1.\n\nOkumura A(1), Fujii K(2), Kurahashi H(1), Nakano S(2).\n\nAuthor information:\n(1)Department of Pediatrics, Aichi Medical University, Nagakute, Japan.\n(2)Department of Surgery, Division of Breast and Endocrine Surgery, Aichi \nMedical University, Nagakute, Japan.\n\nNeurofibromatosis 1 (NF1) is caused by pathogenic variants of the NF1 gene and \nincreases the risk of tumor development. Phyllodes tumors are rare \nfibroepithelial neoplasms of the breast, for which the malignant forms exhibit \nhigh recurrence and metastasis rates. Herein, we report the case of a \n19-year-old female with NF1 who developed a malignant phyllodes tumor of the \nbreast. She noticed a breast mass 3 years before referral, which was diagnosed \nas a fibroadenoma. However, rapid tumor growth was observed 3 months prior to \nher hospital visit. Magnetic resonance imaging of the breast revealed a 9-cm \ntumor, and lumpectomy confirmed the malignancy. Recurrence necessitated a total \nmastectomy and pectoralis major resection. One year after surgery, metastases \nwere detected in the left axillary lymph nodes and femoral head, requiring \nsurgical intervention and denosumab treatment. No relapses were observed after \n18 months. Comprehensive genomic profiling revealed a germline NF1 nonsense \nvariant with loss of heterozygosity (LOH) in tumor cells. Additional somatic \nvariants of TP53 and SMARCB1 have been identified. The TP53 variant is listed in \nthe Catalogue of Somatic Variants in Cancer as a breast cancer-associated \nvariant, whereas a splice site variant in SMARCB1 and LOH suggests a loss of \nSMARCB1 function. This case highlights the increased risk of malignancy in NF1 \npatients and underscores the need for comprehensive genomic profiling.\n\n© 2025 Wiley Periodicals LLC.\n\nDOI: 10.1002/ajmg.a.64305\nPMID: 41204772"
        ],
        [
         "5",
         "41204766",
         "1. Cancer Med. 2025 Nov;14(21):e71366. doi: 10.1002/cam4.71366.\n\nCervical Adenocarcinoma: What's Special About the Long-Term Reproductive and \nOncological Outcomes of Fertility-Sparing Radical Trachelectomy in It?\n\nGan J(1), Cao D(1), Zhou H(1), Yu M(1), Wang T(1), Zhang Y(1), Cheng N(1), Peng \nP(1), Yang J(1), Huang H(1), Shen K(1).\n\nAuthor information:\n(1)National Clinical Research Center for Obstetric & Gynecologic Diseases, \nDepartment of Obstetrics and Gynecology, Peking Union Medical College Hospital, \nChinese Academy of Medical Sciences & Peking Union Medical College, Peking Union \nMedical College Hospital (Dongdan Campus), Beijing, China.\n\nOBJECTIVE: To present reproductive and oncological outcomes of radical \ntrachelectomy (RT) in patients with cervical adenocarcinomas (AC).\nMETHODS: This retrospective study included 51 patients with cervical AC who \nunderwent RT at Peking Union Medical Hospital from January 1, 2005 to June 1, \n2023.\nRESULTS: Five patients (9.8%) experienced cervical stenosis following RT, which \nlikely occurred in cases of abdominal RT (50%) and virginal prophylactic \ncerclage (33.33%) and those without copper T intrauterine devices during RT \n(20%). In total, 30 patients (58.82%) attempted to conceive, and 11 (36.67%) \nsucceeded. Five patients (45.45%) achieved pregnancy with fertility assistance. \nThe mean surgery-pregnancy interval was 27 months (range, 17-118). Two preterm \nand two full-term births were achieved. With a median follow-up of 50 months \n(range, 7-238), seven patients (13.73%) experienced recurrence and three (5.88%) \ndied. Six of seven patients relapsed beyond the residual cervix. The cancer \nrecurrence rate (CRR) was 5.88% for patients with pre-cervical conization and \n17.65% for those with biopsy (p = 0.250); 11.63% had human \npapillomavirus-associated (HPVA) disease and 25% had non-HPVA \n(NHPVA) (p = 0.313). The cancer death rate (CDR) was 4.65% with HPVA and 12.50% \nwith NHPVA (p = 0.386); 13.63% had the endogenous type and 0 had the exogenous \ntype (p = 0.04). Chemotherapy in patients with risk factors resulted in better \nCRR and CDR than in those without (5.88% vs. 17.65%, 0% vs. 8.82%). The \ncumulative 5-year recurrence-free survival (RFS) and overall survival rates were \n82.03% and 94.39%, respectively.\nCONCLUSION: RT in patients with AC led to an acceptable pregnancy rate but a \nhigher CRR and lower 5-year RFS. Careful patient selection for RT, combined with \nadjuvant chemotherapy when indicated, is crucial to optimize the balance between \nreproductive and oncological outcomes in AC.\n\n© 2025 The Author(s). Cancer Medicine published by John Wiley & Sons Ltd.\n\nDOI: 10.1002/cam4.71366\nPMID: 41204766 [Indexed for MEDLINE]"
        ],
        [
         "6",
         "41204749",
         "1. Adv Sci (Weinh). 2025 Nov 7:e10307. doi: 10.1002/advs.202510307. Online ahead\nof  print.\n\nPredicting MammaPrint Recurrence Risk from Breast Cancer Pathological Images \nUsing a Weakly Supervised Transformer.\n\nYan C(1), Li L(2)(3)(4), Qian X(3)(4)(5), Ou Y(2)(3)(4), Huang Z(2)(3)(4), Ruan \nZ(1), Xiang W(1), Liu H(2)(3)(4), Liu J(6).\n\nAuthor information:\n(1)Centre for Bioinformatics and Intelligent Medicine, College of Computer \nScience, Nankai University, Tianjin, 300350, China.\n(2)The Second Surgical Department of Breast Cancer, Tianjin Medical University \nCancer Institute & Hospital, National Clinical Research Center for Cancer, \nTianjin, 300060, China.\n(3)Tianjin's Clinical Research Center for Cancer, Tianjin, 300060, China.\n(4)Key Laboratory of Breast Cancer Prevention and Therapy, Tianjin Medical \nUniversity, Ministry of Education, Tianjin, 300060, China.\n(5)Department of Breast Pathology and Lab, Tianjin Medical University Cancer \nInstitute & Hospital, National Clinical Research Center for Cancer, Tianjin, \n300060, China.\n(6)State Key Laboratory of Medicinal Chemical Biology, College of Computer \nScience, Nankai University, Tianjin, 300350, China.\n\nRecurrence related to poor prognosis is a leading cause of mortality in patients \nwith breast cancer (BC). The MammaPrint (MP) genomic assay is designed to \nstratify recurrence risk and evaluate chemotherapy benefits for early-stage \nHR+/HER2- BC patients. However, MP fails to reveal spatial tumor morphology and \nis limited by high costs. In this study, a BC MP cohort is established and CPMP \nis developed, a weakly supervised agent-attention transformer model, to predict \nMP recurrence risk from annotation-free BC histopathological slides. CPMP \nachieves an AUROC of 0.824 ± 0.03 in predicting MP risk groups. CPMP is further \nleveraged for spatial and morphological analyses to explore histological \npatterns associated with MP risk groups. The model reveals tumor spatial \nlocalization at the whole-slide level and highlights distinct intercellular \ninteraction patterns of MP groups. It also characterizes the diversity in tumor \nmorphology and uncovers MP high-specific, low-specific, and colocalized \nmorphological phenotypes that differ in quantitative cellular composition. \nPrognostic evaluation in the external cohort exhibits significant stratification \nof distant metastasis risk (HR: 3.14, p-value = 0.0014), underscoring the \nprognostic power of CPMP. These findings demonstrate the capability of CPMP in \nMP risk prediction, offering a flexible supplement to genomic risk assessment in \nearly-stage BC.\n\n© 2025 The Author(s). Advanced Science published by Wiley‐VCH GmbH.\n\nDOI: 10.1002/advs.202510307\nPMID: 41204749"
        ],
        [
         "7",
         "41204725",
         "1. J Adolesc Young Adult Oncol. 2025 Nov 6. doi: 10.1177/21565333251394601.\nOnline  ahead of print.\n\nWorking in This Field and Living in This Field: The Importance of Cancer \nSurvivors in the Professional Setting.\n\nTopalian A(1), Baldwin E(2), Panagiotou V(3).\n\nAuthor information:\n(1)Department of Family and Community Medicine, Division of Cancer Survivorship \nand Supportive Services, University of Cincinnati College of Medicine, \nCincinnati, Ohio, USA.\n(2)Blood Cancer United (Formerly The Leukemia & Lymphoma Society), Washington, \nDistrict of Columbia, USA.\n(3)Cancer Nation (Formerly the National Coalition for Cancer Survivorship), \nSilver Spring, Maryland, USA.\n\nDOI: 10.1177/21565333251394601\nPMID: 41204725"
        ],
        [
         "8",
         "41204722",
         "1. Nucleic Acid Ther. 2025 Nov 5. doi: 10.1177/21593337251387423. Online ahead of\n print.\n\nImmunogenicity Risk Assessment for Nucleic Acid Therapeutics: A Comprehensive \nEvaluation for ASO, siRNA, and Nonvaccine mRNA/LNP Therapies by the IQ \nConsortium.\n\nGrudzinska-Goebel J(1), Braun M(1), Chen LZ(2), Gupta S(3), Lohmann S(4), Morais \nP(5), Niu T(6), Tarcsa E(7), Tourdot S(8), Youssef AS(9), Jawa V(10).\n\nAuthor information:\n(1)Preclinical Development, Pharmaceuticals R&D, Bayer AG, Berlin, Germany.\n(2)Boehringer Ingelheim Pharmaceuticals, Ridgefield, Connecticut, USA.\n(3)Abbvie, Irvine, California, USA.\n(4)Roche Pharma Research and Early Development (pRED), Roche Innovation Center \nMunich, Roche Diagnostics GmbH, Penzberg, Germany.\n(5)Preclinical Development, Pharmaceuticals R&D, Bayer AG, Wuppertal, Germany.\n(6)Quantitative Clinical Pharmacology, Sarepta Therapeutics, Cambridge, \nMassachusetts, USA.\n(7)Abbvie Bioresearch Center, Worcester, Massachusetts, USA.\n(8)Pfizer, Pharmacokinetics, Dynamics and Metabolism, Andover, Massachusetts, \nUSA.\n(9)Clinical Pharmacology, Modeling and Simulation, GSK, Collegeville, \nPennsylvania, USA.\n(10)Translational Medicine, Bristol Myers Squibb, Lawrenceville, New Jersey, \nUSA.\n\nThe emergence of nucleic acid (NA) therapeutics, including antisense \noligonucleotides (ASOs), small interfering RNAs (siRNAs), which are usually \ndelivered directly, and messenger RNAs (mRNAs), which are typically encapsulated \nin lipid nanoparticles (LNPs), marks a transformative era in precision medicine. \nWhile these therapies offer precise approaches for gene regulation or \nexpression, they can trigger unwanted innate and/or adaptive immune responses \nthat can either have no significant impact or adversely affect treatment \nefficacy and/or patient safety. Consequently, therapies where an adaptive immune \nresponse is desired, such mRNA/LNP-based vaccines against infectious diseases or \ncancer are out of scope of this article. In the present work, the Innovation and \nQuality Consortium Nucleic Acids Immunogenicity Working Group examines how the \nvarious components of NA-based therapies might contribute to their immunogenic \npotential and describes risk mitigation strategies through product design \nadaptations during early development stages. In addition, a comprehensive \nimmunogenicity risk assessment framework is described, allowing to effectively \ndefine a tailored clinical testing strategy for different NA modalities with \nvarying immunogenicity (IG) consequences. A streamlined monitoring strategy is \nrecommended when minimal impact is expected, whereas extensive testing is \nsuggested when safety concerns arise. Overall, these recommendations ensure that \nsafe and effective NA-based therapies reach patients with an appropriate \nassessment of the IG potential.\n\nDOI: 10.1177/21593337251387423\nPMID: 41204722"
        ],
        [
         "9",
         "41204717",
         "1. Acta Biochim Biophys Sin (Shanghai). 2025 Nov 7. doi: 10.3724/abbs.2025204. \nOnline ahead of print.\n\nUnveiling the potential role of ACSL1 in colorectal cancer proliferation: a \nnovel therapeutic avenue.\n\nLiu J(1), Yao X(1), Chen J(1), Gao Y(1)(2), Gong L(1), Li J(1), Cao D(1)(2), \nZhao C(2)(3), Gao G(1)(2)(4).\n\nAuthor information:\n(1)Department of Biliary-Pancreatic Surgery, Renji Hospital, Shanghai Jiao Tong \nUniversity School of Medicine, Shanghai 200127, China.\n(2)State Key Laboratory of Systems Medicine for Cancer, Shanghai Cancer \nInstitute, Renji Hospital, Shanghai Jiao Tong University School of Medicine, \nShanghai 200127, China.\n(3)Shanghai Key Laboratory for Cancer Systems Regulation and Clinical \nTranslation, Shanghai Jiading District Central Hospital, Shanghai 201899, China.\n(4)Zhangjiang Center for Translational Medicine, Shanghai Biotecan \nPharmaceuticals Co. Ltd., Shanghai 201204, China.\n\nAberrant expression of acyl-CoA synthetase long-chain family member 1 (ACSL1) \noccurs in multiple cancer types and is closely linked to patient prognosis. \nNevertheless, its precise mechanistic role in colorectal cancer (CRC) remains \npoorly understood. In this study, we assess ACSL1 expression in CRC and \nelucidate the molecular mechanisms through which ACSL1 regulates tumor \nproliferation and migration. Our results show that ACSL1 is significantly \nupregulated in CRC and is associated with poor patient survival. Knockdown of \nACSL1 suppresses CRC cell proliferation both in vitro and in vivo. Furthermore, \nACSL1 silencing downregulates the JAK2-STAT3 signaling axis. A strong \ncorrelation also exists between ACSL1 expression and epithelial-mesenchymal \ntransition (EMT). Collectively, these findings indicate that ACSL1 is highly \nexpressed in CRC tissues and is correlated with poor prognosis. Importantly, our \nstudy provides the first evidence that the ACSL1-JAK2-STAT3 pathway facilitates \nCRC cell proliferation and migration, highlighting its potential as a \ntherapeutic target.\n\nDOI: 10.3724/abbs.2025204\nPMID: 41204717"
        ],
        [
         "10",
         "41204712",
         "1. Aging Male. 2025 Dec 11;28(1):2578633. doi: 10.1080/13685538.2025.2578633.\nEpub  2025 Nov 7.\n\nBiomarker-driven mechanisms and therapeutic targets for prostate diseases \nthrough mendelian randomization and molecular docking.\n\nLu L(1), Shen J(2), Chen Z(2), Zhang J(1), Gong X(2), Pan H(1), Luo X(1), Meng \nS(1), Zheng Y(1)(3).\n\nAuthor information:\n(1)Department of Surgery, the Fourth Affiliated Hospital of School of Medicine, \nand International School of Medicine, International Institutes of Medicine, \nZhejiang University, Yiwu, China.\n(2)Department of Urology, the Fourth Affiliated Hospital of School of Medicine, \nand International School of Medicine, International Institutes of Medicine, \nZhejiang University, Yiwu, China.\n(3)Department of Thyroid Surgery, 2nd Affiliated Hospital, Zhejiang University \nSchool of Medicine, Hangzhou, China.\n\nBACKGROUND: Prostate diseases, comprising prostatitis, benign prostatic \nhyperplasia (BPH), and prostate cancer (PCA), represent substantial public \nhealth burdens. This study aimed to clarify biomarker-disease relationships and \nidentify potential therapeutic targets for prostate disorders using a Mendelian \nRandomization (MR) framework.\nMETHODS: We conducted two-sample MR analyses using genome-wide association data. \nCausal effects were estimated using complementary MR methods. To identify \ntherapeutic targets, MR findings were integrated with DrugBank, protein-protein \ninteraction networks, bulk RNA sequencing, and molecular docking.\nRESULTS: Six biomarkers showed causal effects on prostate diseases: URK on \nprostatitis; GGT and TBIL on BPH; and UCR, PHOS, and BUN on PCA. Functional \nintegration highlighted MPO and TUBB as drug targets for BPH, while docking \nimplicated MET and ATP8B1 in BPH and GATA3 and ENPP3 in PCA. Drug repurposing \nsuggested dexamethasone for BPH and colchicine and metformin for PCA.\nCONCLUSION: This study provides genetic evidence linking biomarkers to prostate \ndiseases and identifies novel druggable targets and repurposing opportunities, \noffering mechanistic insights and prospects for precision therapy.\n\nPlain Language Summary: This article applies two-sample Mendelian randomization \nto establish causal relationships between circulating biomarkers and prostate \ndiseases.Identified six key biomarkers with significant causal effects on \nprostatitis, BPH, and prostate cancer.Integrated DrugBank, protein–protein \ninteraction networks, and bulk RNA-seq to reveal novel drug target \ngenes.Molecular docking and drug prediction suggested dexamethasone as a \ncandidate for BPH, and colchicine and metformin as potential therapies for \nprostate cancer.Provides new insights into biomarker-driven mechanisms and \ntherapeutic opportunities for prostate disorders.\n\nDOI: 10.1080/13685538.2025.2578633\nPMID: 41204712 [Indexed for MEDLINE]"
        ],
        [
         "11",
         "41204710",
         "1. Nutr Cancer. 2025 Nov 7:1-11. doi: 10.1080/01635581.2025.2582236. Online ahead\n of print.\n\nEvaluation of the Relationship Between Sarcopenia, Nutritional Status, and \nFrailty in Older Patients with Cancer.\n\nDoğan Akagündüz D(1), Şahin H(2), Akagündüz B(3)(4).\n\nAuthor information:\n(1)Department of Nutrition and Dietetics, Erzincan Binali Yildirim University \nMengücek Gazi Training and Research Hospital, Erzincan, Turkey.\n(2)Department of Nutrition and Dietetics, Faculty of Health Science, Binali \nYildirim University, Erzincan, Türkiye.\n(3)Department of Medical Oncology, Erzincan Binali Yildirim University Medical \nSchool, Erzincan, Turkey.\n(4)King Hamad University Hospital, Bahrain Oncology Center, Muharraq, Bahrain.\n\nSarcopenia, malnutrition, and frailty frequently coexist in older adults with \ncancer, adversely affecting treatment tolerance, functional independence, and \nsurvival. While comprehensive geriatric assessment (CGA) is the gold standard \nfor frailty evaluation, its resource-intensive nature limits routine use, and \nsimple tools are needed. This cross-sectional study included 291 patients aged \n≥70 years with histologically confirmed cancer, evaluated at a oncology \noutpatient clinic between January and November 2024. Sarcopenia was defined \naccording to EWGSOP2 criteria: handgrip strength (HGS) and skeletal muscle index \n(SMI) by bioelectrical impedance analysis (BIA). Nutritional status was assessed \nwith the Mini Nutritional Assessment-Short Form (MNA-SF), and frailty was \ndefined as G8 ≤ 14. Additional measures included SARC-F, Mini-Mental State \nExamination (MMSE), and Geriatric Depression Scale. The median age was 74 years, \nand 53% were male. Possible sarcopenia was identified in 39.5%, confirmed \nsarcopenia in 33.7%, severe sarcopenia in 26.8%. Nutritional risk was observed \nin 66.7%, and frailty in 44.7%. Frailty correlated strongly with nutritional \nstatus and moderately with muscle strength and mass. In multivariable analysis, \nlow HGS, low SMI, nutritional risk, and depression independently predicted \nfrailty. ROC analysis showed, G8 with HGS and nutritional assessment improved \npredictive accuracy. These findings highlight that simple measures enhance \nfrailty detection and may support personalized care.\n\nDOI: 10.1080/01635581.2025.2582236\nPMID: 41204710"
        ],
        [
         "12",
         "41204697",
         "1. J Appl Toxicol. 2025 Nov 7. doi: 10.1002/jat.4981. Online ahead of print.\n\nToxicological Perspectives on RNA m1A Methylation: Biological Processes and \nPathological Consequences.\n\nFu J(1), Huang T(1), Li G(1), Zhang X(1), Tan L(1), Zhu C(1), Zhang W(2), Zhang \nW(1).\n\nAuthor information:\n(1)Department of Public Health and Preventive Medicine, School of Medicine, \nJinan University, Guangzhou, Guangdong, China.\n(2)Key Laboratory of Crop Genetic Improvement of Guangdong Province, Crops \nResearch Institute, Guangdong Academy of Agricultural Sciences, Guangzhou, \nChina.\n\nRNA N1-methyladenosine (m1A) methylation is dynamically regulated by \nmethyltransferases (TRMT6/61/61B/10C), demethylases (ALKBH1/3), and binding \nproteins (YTHDF1/2/3), which collectively fine-tune gene expression through \nsite-specific modifications. Exogenous environmental factors such as persistent \norganic pollutants, heavy metals, and radiation can trigger cellular oxidative \nstress and stimulate the secretion of reactive oxygen species and \nsenescence-associated secretory phenotypes. The concurrent accumulation of these \ndamaging agents, along with dysregulation of intracellular conditions including \ntemperature, pH, and divalent metal ion concentrations under pathological \nstates, disrupts m1A methylation and alters the expression of associated genes, \nultimately leading to adverse cellular outcomes. In addition, we searched a \nlarge number of literature and found that m1A methylation exhibits elevated \nlevels in neurodegeneration, ischemia-reperfusion injury, and hepatocellular and \nbladder cancer, whereas decreased levels are observed in Alzheimer's disease and \nmyocardial infarction. Correspondingly, methyltransferases and binding proteins \ninvolved in m1A modification are generally upregulated across multiple disease \ncontexts. We propose that m1A methylation serves as a molecular sensor for \nenvironmental-cell interactions, dynamically regulating gene expression through \nregulators and exerting bidirectional control in disease processes. Given its \nregulatory versatility, m1A modification holds promise for applications in \ntoxicological risk assessment, precision medicine, and the development of \ntargeted therapies against exogenous factor-induced pathologies.\n\n© 2025 John Wiley & Sons Ltd.\n\nDOI: 10.1002/jat.4981\nPMID: 41204697"
        ],
        [
         "13",
         "41204684",
         "1. Br J Pharmacol. 2025 Nov 7. doi: 10.1111/bph.70220. Online ahead of print.\n\nCircPTK2 enhances non-small cell lung cancer proliferation by inhibiting \ncellular senescence through stabilizing NFYA and elevating FOXM1.\n\nLi Q(1)(2)(3), Zhao K(4), Shen Y(1)(2), Ying Q(2), Chen B(5), Zhang J(4), Wang \nB(1)(2)(3), Dong S(1)(2)(3), Shi X(1)(2)(3).\n\nAuthor information:\n(1)Department of Respiratory Medicine, Huzhou Central Hospital, Fifth School of \nClinical Medicine of Zhejiang Chinese Medical University, Huzhou, China.\n(2)Department of Respiratory Medicine, Huzhou Central Hospital, Affiliated \nCentral Hospital Huzhou University, Huzhou, China.\n(3)Huzhou Key Laboratory of Precision Diagnosis and Treatment in Respiratory \nDiseases, Huzhou Central Hospital, Huzhou, China.\n(4)Department of Oncology, Huaian Hospital of Huaian City, Huaian, China.\n(5)Department of Respiratory Medicine, Huzhou Hospital, Zhejiang University \nSchool of Medicine, Huzhou, China.\n\nBACKGROUND AND PURPOSE: Cellular senescence, a stress-induced cell cycle arrest \nstate, plays a dual role in cancer. Circular RNAs (circRNAs), endogenous \nnoncoding RNAs, regulate physiological processes and are linked to diseases \nincluding cancer. This study investigates the role and mechanism of circPTK2 in \nnon-small cell lung cancer (NSCLC).\nEXPERIMENTAL APPROACH: Lentivirus-mediated knockdown or overexpression of \ncircPTK2 were used to assess effects on proliferation and senescence. circPTK2 \nexpression in NSCLC tissues was examined using tissue microarrays and in situ \nhybridization. RNA-seq identified circPTK2-regulated signalling pathways and \ndownstream targets. Transcriptional regulation of FOXM1 by NFYA was investigated \nusing chromatin immunoprecipitation (ChIP) and dual-luciferase reporter assays. \nCo-immunoprecipitation (Co-IP) was conducted to assess the interaction between \nNFYA and MDM2, as well as NFYA ubiquitination.\nKEY RESULTS: circPTK2 was highly expressed in NSCLC tissues and was associated \nwith poor prognosis. Knockdown of circPTK2 significantly inhibited cancer cell \ngrowth and enhanced cellular senescence. RNA-seq analysis identified FOXM1 as a \ndownstream target of circPTK2. Database predictions suggested that NFYA could \ntranscriptionally regulate FOXM1. Knockdown of circPTK2 induced NFYA \ndegradation. circPTK2 was found to bind to the E3 ubiquitin ligase MDM2 in the \ncytoplasm, thereby preventing MDM2 from interacting with the transcription \nfactor NFYA. This interaction blocked MDM2-mediated ubiquitylation and \ndegradation of NFYA. Additionally, NFYA bound to the FOXM1 gene promoter, \nultimately leading to the up-regulation of FOXM1 and suppression of cellular \nsenescence.\nCONCLUSIONS AND IMPLICATIONS: Elevated circPTK2 expression correlates with \npoorer survival, representing a potential therapeutic target in NSCLC.\n\n© 2025 British Pharmacological Society.\n\nDOI: 10.1111/bph.70220\nPMID: 41204684"
        ],
        [
         "14",
         "41204682",
         "1. ACS Appl Bio Mater. 2025 Nov 7. doi: 10.1021/acsabm.5c01310. Online ahead of \nprint.\n\nInduction of Carbodiimide Coupling of Alendronate to Chitosan with Samarium \nEntrapment and NaDyF(4)@NaYF(4):Nd(3+) in PVA Double-Layered Hydrogel for \nPhotothermal Therapy and Bone Remodeling.\n\nMurugan D(1), Kumar A(2), Dhayalan A(2), Kannan S(1).\n\nAuthor information:\n(1)Centre for Nanoscience and Technology, Pondicherry University, Puducherry 605 \n014, India.\n(2)Department of Biotechnology, Pondicherry University, Puducherry 605 014, \nIndia.\n\nThe combination of challenges associated with osteosarcoma treatment emerges \nfrom the inadequacy of conventional cancer therapies to heal the extensive bone \ndefects generated following tumor excision. On the other hand, the \nbone-regeneration approach lacks in mitigating the risk of tumor recurrence. \nThis therapeutic gap demands the necessity of a multifunctional approach that \nsimultaneously inhibits cancer recurrence and promotes the process of bone \nregeneration. The extensively employed traditional photosensitizers display \nlimited anticancer activity and lack regenerative properties. These constraints \nare circumvented through the development of a bilayer Janus hydrogel that \nintegrates NaDyF4@NaYF4:Nd3+ nanophosphors within a poly(vinyl alcohol) (PVA) \nlayer, while a chitosan layer encompasses functionalized alendronate and \nsamarium. This multifunctional hydrogel, developed through a freeze-thaw \nprocess, exhibited strong photothermal activity and also demonstrated efficacy \nin eradicating MG63 osteosarcoma cells in vitro. Further, the typical \nextracellular matrix resemblance characteristic of the hydrogel demonstrated \nbetter cell proliferation and osteogenic differentiation of MG63 cells and \npromoted vascularization in the CAM model. Moreover, the presence of lanthanides \nin the hydrogel facilitates multimodal imaging characteristics of magnetic \nresonance imaging (MRI), computed tomography (CT), and near-infrared (NIR) \nluminescence. Thus, the synergistic effect of encompassing localized cancer \ntherapy and efficient bone restoration offers a promising strategy to treat bone \ntumors.\n\nDOI: 10.1021/acsabm.5c01310\nPMID: 41204682"
        ],
        [
         "15",
         "41204681",
         "1. Hematol Oncol. 2025 Nov;43(6):e70151. doi: 10.1002/hon.70151.\n\nFatigue, Neuropathy and Psychological Distress and Their Association With \nHealth-Related Quality of Life in Survivors of Diffuse Large B-Cell Lymphoma: A \nProspective Cohort Study and Comparison With a Normative Population.\n\nNeppelenbroek SIM(1), Ekels A(1)(2), Kersten MJ(3), Issa DE(4), Thielen N(5), \nvan de Poll-Franse LV(1)(2), Oerlemans S(2).\n\nAuthor information:\n(1)Department of Psychosocial Research, Division of Psychosocial Research & \nEpidemiology, The Netherlands Cancer Institute, Amsterdam, the Netherlands.\n(2)Department of Research & Development, Netherlands Comprehensive Cancer \nOrganisation (IKNL), Utrecht, the Netherlands.\n(3)Department of Hematology, Amsterdam University Medical Centers, Location \nUniversity of Amsterdam, Cancer Center Amsterdam, Amsterdam, the Netherlands.\n(4)Department of Hematology, Jeroen Bosch Ziekenhuis, Den Bosch, the \nNetherlands.\n(5)Department of Internal Medicine, Diakonessenhuis, Utrecht, the Netherlands.\n\nWe investigated the course of fatigue, neuropathy, psychological distress and \ntheir impact on HRQoL among diffuse large B-Cell Lymphoma (DLBCL) survivors up \nto 13 years after diagnosis, and compared it to an age- and sex-matched \npopulation. DLBCL survivors diagnosed between 2004 and 2011, who survived \n≥ 1 year were selected from the Netherlands Cancer Registry. Survivors completed \nannual questionnaires 2009-2019, including EORTC QLQ-C30, EORTC CLL-17 and the \nHospital Anxiety and Depression Scale. Linear mixed models were used to assess \nsymptom trends and HRQoL associations over time. 302 survivors (response rate \n84%) completed a median of three questionnaires. Fatigue improved slightly over \ntime but neuropathy and psychological distress remained unchanged; persistent \nsymptoms were reported by 25%, 28%, and 23% of survivors, respectively. \nHaving two or more comorbidities was associated with higher symptom levels \n(βfatigue = 7.8, p-value < 0.001; βneuropathy = 6.7, p-value = 0.003; \nβpsychological distress = 2.2, p-value < 0.001). Having received seven to eight \ncycles of (R-)CHOP14 was associated with higher neuropathy levels (β = 14.5, \np < 0.001) and non-(R-)CHOP treatments were associated to high fatigue levels \n(β = 14.0, p-value = 0.02) and psychological distress (β = 4.3, p-value = 0.01). \nHigher symptom levels were associated with poorer HRQoL. One in four DLBCL \nsurvivors reported persistent fatigue, neuropathy or psychological distress, \nnegatively impacting their HRQoL. Routine symptom monitoring is essential to \nidentify needs and guide supportive care referrals.\n\n© 2025 John Wiley & Sons Ltd.\n\nDOI: 10.1002/hon.70151\nPMID: 41204681 [Indexed for MEDLINE]"
        ],
        [
         "16",
         "41204677",
         "1. Br J Clin Pharmacol. 2025 Nov 7. doi: 10.1002/bcp.70339. Online ahead of\nprint.\n\nComment on: 'Association of concomitant proton pump inhibitor use with worse \nsurvival outcomes in patients with advanced gastric cancer receiving \nramucirumab'-The composition of the 'non-PPI user' group as a confounding \nfactor.\n\nMao G(1), Liu M(1), Wang H(1).\n\nAuthor information:\n(1)Department of Pharmacy, Danyang People's Hospital and Danyang Hospital \nAffiliated to Nantong University, Zhenjiang, Jiangsu Province, China.\n\nDOI: 10.1002/bcp.70339\nPMID: 41204677"
        ],
        [
         "17",
         "41204675",
         "1. Stem Cells Dev. 2025 Nov 7. doi: 10.1177/15473287251393202. Online ahead of \nprint.\n\nPOSTN+ Cancer-Associated Fibroblasts Promote Gastric Cancer Invasion by \nActivating AKT Signaling.\n\nXu X(1), You T(1), Sun Z(1), Bai C(1), Han Q(2), Zhao RC(2).\n\nAuthor information:\n(1)Department of Medical Oncology, Peking Union Medical College Hospital, \nChinese Academy of Medical Sciences & Peking Union Medical College, Beijing, \nChina.\n(2)School of Basic Medicine, Institute of Basic Medical Sciences, Chinese \nAcademy of Medical Sciences, Peking Union Medical College, Beijing, China.\n\nMetastasis is the primary cause of death in advanced/recurrent cancer patients. \nCancer metastatic capability depends not only on cancer cells but also on the \ncancer microenvironment, particularly cancer-associated fibroblasts (CAFs), a \nhighly heterogeneous population. Our prior work identified a POSTN-secreting \nCAFs subpopulation linked to gastric cancer (GC) invasion and poor survival. The \nCancer Genome Atlas analysis in GC showed POSTN association with \nepithelial-mesenchymal transition and extracellular matrix degradation pathways. \nIn vitro, GC exosomes induced adipose-derived mesenchymal stem cells (MSCs) into \nPOSTN-expressing CAFs. Lentiviral POSTN overexpression in CAFs enhanced GC cell \nmigration/invasion, while knockdown had the opposite effect. These results were \nvalidated in a nude mouse GC model. As POSTN is an integrin ligand, \nPOSTN-positive CAFs (POSTN+ CAFs) activated integrin downstream AKT signaling. \nAKT inhibition significantly diminished the pro-migratory/invasive effect of \nPOSTN-overexpressing CAFs. In summary, POSTN+ CAFs promote GC invasion via AKT \npathway activation.\n\nDOI: 10.1177/15473287251393202\nPMID: 41204675"
        ],
        [
         "18",
         "41204672",
         "1. Thyroid. 2025 Nov 6. doi: 10.1177/10507256251393208. Online ahead of print.\n\nThe Prognostic Value of Immunohistochemical Biomarkers in Medullary Thyroid \nCarcinoma: A Systematic Review and Meta-Analysis.\n\nSchonebaum LE(1), Oudijk L(2), Stegenga MT(1), Bramer WM(3), Derakhshan A(1), \nvan Kemenade FJ(2), Visser WE(1), Peeters RP(1).\n\nAuthor information:\n(1)Department of Internal Medicine, Academic Center for Thyroid Diseases, \nErasmus MC, Erasmus University Medical Center Rotterdam, Rotterdam, The \nNetherlands.\n(2)Department of Pathology, Academic Center for Thyroid Diseases, Erasmus MC, \nErasmus University Medical Center Rotterdam, Rotterdam, The Netherlands.\n(3)Medical Library, Erasmus MC, Erasmus University Medical Center Rotterdam, \nRotterdam, The Netherlands.\n\nBackground: The prognosis and disease course of medullary thyroid carcinoma \n(MTC) can vary widely among patients. Effective risk stratification is important \nfor ensuring timely treatment and personalized follow-up. Biomarkers can enhance \nrisk stratification and guide the development of targeted anticancer therapies \nand imaging techniques. While numerous studies have explored various \nimmunohistochemical biomarkers in MTC, an overview is still lacking. This study \naimed to provide a comprehensive overview of immunohistochemical biomarkers and \ntheir role in the prognosis of MTC patients, with a primary focus on overall \nsurvival (OS). Methods: This review was preregistered in PROSPERO \n(CRD42023469437). A systematic search was performed using the online medical \ndatabases Embase, MEDLINE (Ovid), and Cochrane. Quality was assessed using an \nadapted scoring system based on the REMARK criteria and the Quality in Prognosis \nStudies tool. The primary outcome was OS. Secondary outcomes included other \ntypes of survival and associations with clinicopathological risk factors and \nrecurrence. Results: Of 2992 studies, 108 were included, investigating 170 \nunique biomarkers and with sample sizes ranging from 11 to 327 participants. The \nmajority (72%) were reported in only one article. A minority of studies were \nrated as high quality (28%). Markers of proliferation Ki-67 (Ki-67) and \nprogrammed cell death-ligand 1 (PD-L1) were significantly associated with OS \n(hazard ratio [HR]: 6.67, confidence interval [CI]: 1.43-31.18 and HR: 3.34, CI: \n1.18-9.51). Conclusions: Our systematic review provides a comprehensive \nsynthesis of the literature on immunohistochemical biomarkers in MTC and \nhighlights the need for high-quality validation studies. Our meta-analysis \nconfirms the prognostic value of Ki-67, although with varying certainty due to \nlarge differences in study quality. Furthermore, we describe the association of \nPD-L1 positivity with poorer OS, increased recurrence, and more aggressive \nclinicopathological features, which supports the rationale for further \ninvestigating the potential of anti-PD-1/PD-L1 immunotherapy for advanced MTC.\n\nDOI: 10.1177/10507256251393208\nPMID: 41204672"
        ],
        [
         "19",
         "41204580",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e44568. doi: \n10.1097/MD.0000000000044568.\n\nBibliometric and visual analysis of global research on primary liver cancer from \n2014 to 2024.\n\nWang Y(1)(2), Hui Y(2)(3), Cai C(2).\n\nAuthor information:\n(1)School of Basic Medical Sciences, Hebei University of Traditional Chinese \nMedicine, Shijiazhuang, China.\n(2)Department of Traditional Chinese Medicine, Hebei General Hospital, \nShijiazhuang, China.\n(3)Clinical College of Integrated Traditional Chinese and Western Medicine, \nTianjin University of Traditional Chinese Medicine, Tianjin, China.\n\nOBJECTIVE: To evaluate the quantity and quality of global research on primary \nliver cancer (PLC) over the past decade, visualize research hotspots, and reveal \ntrends in PLC research.\nMETHODS: A comprehensive search was conducted in the Web of Science Core \nCollection for academic publications from January 1, 2014 to January 1, 2024. \nVOSviewer and CiteSpace were used to analyze and visualize research trends and \nhotspots.\nRESULTS: A total of 1728 articles from 69 countries were included in the study, \nwith China and the United States being the leading contributors. The number of \npublications related to primary liver cancer has increased annually. Major \nresearch institutions involved in this field include Fudan University, Sun \nYat-Sen University, among others. A total of 11,001 authors from around the \nworld have published papers; among them, McGlynn, Kokudo Norihiro, and Kudo \nMasatoshi published the most articles, while Llovet JM is the most frequently \ncited author in this field. Frontiers in Oncology is the most popular journal in \nthe field of liver cancer, while Hepatology is the most co-cited journal. In \nrecent years, terms such as \"multicenter,\" \"open label,\" and \"safety\" have \ngained increasing attention and development.\nCONCLUSION: Our findings offer valuable insights and references for the \ndevelopment of key areas in PLC and contribute to future research.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000044568\nPMID: 41204580 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no funding and conflicts of \ninterest to disclose."
        ],
        [
         "20",
         "41204566",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45653. doi: \n10.1097/MD.0000000000045653.\n\nThe association between psychological status and the development of early \ngastric cancer from atrophic gastritis.\n\nLiang M(1), Wang J(1), Li R(2), Yang J(1), Liu Y(2), Zhao L(1).\n\nAuthor information:\n(1)Department of Gastroenterology, The Affiliated Hospital of Xuzhou Medical \nUniversity, Xuzhou City, Jiangsu Province, China.\n(2)Department of Nursing, The Affiliated Hospital of Xuzhou Medical University, \nXuzhou City, Jiangsu Province, China.\n\nThis study aimed to explore whether there may be a potential link between \npsychological factors and the occurrence of early-stage gastric cancer (EGC) \namong individuals diagnosed with atrophic gastritis (AG). A total of 258 \nindividuals receiving care in the Department of Gastroenterology at The \nAffiliated Hospital of Xuzhou Medical University between March 2020 and November \n2024 were included in this cross-sectional observational study, including 173 \npatients diagnosed with AG and 85 with EGC. Diagnostic confirmation was based on \nfindings from both endoscopic procedures and histological examinations. \nPsychological well-being was evaluated through the application of the Kessler \npsychological distress scale (K6) alongside the Center for epidemiologic studies \ndepression scale (CES-D). At the time of enrollment, clinical profiles, \ndemographic characteristics, and laboratory indices were recorded. Comparative \nanalyses were conducted between groups, and both univariate and multivariable \nlogistic regression models were employed to examine potential factors associated \nwith EGC. What's more, receiver operating characteristic (ROC) curve analysis \nwas utilized to explore the discriminative capacity of psychological \nassessments. Compared to the AG group, patients with EGC exhibited significantly \nhigher K6 (10.36 ± 5.54 vs 5.82 ± 5.25, P < .001) and CES-D scores (12.99 ± 7.70 \nvs 9.01 ± 7.27, P < .001), with a greater proportion exceeding clinical cutoffs \n(K6 ≥ 13: 30.59% vs 12.72%; CES-D ≥ 16: 41.18% vs 21.39%). Multivariate logistic \nregression identified Helicobacter pylori infection (OR = 3.960, P = .003), \nconcave-type lesion morphology (OR = 107.086, P < .001), K6 score (OR = 1.192, \nP < .001), and CES-D score (OR = 1.097, P = .001) as independent predictors of \nprogression from AG to EGC. ROC analysis demonstrated moderate predictive value \nfor the K6 (AUC = 0.729) and CES-D (AUC = 0.652) scores individually, with \nimproved discriminative ability when combined (AUC = 0.763). Among individuals \nwith AG, psychological distress and depressive tendencies showed a potential \nassociation with early-stage gastric malignancy. While not conclusive, these \nobservations imply that incorporating mental health screening tools, such as the \nK6 and CES-D scales, could offer supplementary value in the broader context of \nevaluating patients who might carry an increased likelihood of disease \nprogression.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045653\nPMID: 41204566 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no funding and conflicts of \ninterest to disclose."
        ],
        [
         "21",
         "41204558",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45702. doi: \n10.1097/MD.0000000000045702.\n\nReal-world study of first-line therapy with aumolertinib for elderly patients \nwith non-small cell lung cancer harboring EGFR mutation.\n\nLiu N(1), Xu S(1), Xie Y(1), Xu T(1), Fang H(1), Wang X(1), Yang L(2), Fang \nS(1).\n\nAuthor information:\n(1)Department of Respiratory Medicine, Nanjing Chest Hospital, The Affiliated \nBrain Hospital of Nanjing Medical University, Nanjing, China.\n(2)Department of Respiratory Medicine, Nanjing Gaochun People's Hospital, \nNanjing, China.\n\nElderly non-small cell lung cancer (NSCLC) patients do not always benefit from \nstandard treatments due to impaired organ function and/or multiple \ncomorbidities. Our study aimed to determine the efficacy and safety of \naumolertinib as a first-line therapy in NSCLC patients aged ≥65 and <65 years in \nclinical practice.We enrolled 100 patients with stage IIA-IVB epidermal growth \nfactor receptor-mutant NSCLC who received aumolertinib alone as the first-line \ntherapy. Efficacy and safety were compared between patients aged ≥65 and <65 \nyears in different subgroups. The primary endpoint was the objective response \nrate (ORR). The secondary endpoints included progression-free survival (PFS), \noverall survival (OS), disease control rate (DCR), and safety.Overall, ORR and \nDCR were 76% and 98%, respectively. ORR was 69.4% and 82.4% for patients \naged ≥ 65 and < 65 years, respectively (P = .27), and DCR was 98% and 98%, \nrespectively (P = .93). The median PFS was 23.2 months. The median PFS was 26.9 \nmonths and 18.3 months in the ≥65 and <65 years groups, respectively (P = .377, \nand the median OS of all patients was 31.7 months. The median OS was 33.7 months \nand 30.7 months in the ≥65 and <65 years groups, respectively (P = .851). \nAdverse events were not statistically different between the 2 groups.The \nefficacy and safety profile of aumolertinib as a first-line therapy in elderly \npatients with epidermal growth factor receptor-mutant NSCLC were similar to \nthose observed in the younger subgroup.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045702\nPMID: 41204558 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no conflicts of interest to \ndisclose."
        ],
        [
         "22",
         "41204557",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45651. doi: \n10.1097/MD.0000000000045651.\n\nEducation as a protective factor against lung cancer: A comprehensive Mendelian \nrandomization analysis.\n\nWang Y(1), Wang X(2), Sun G(1), Feng P(1).\n\nAuthor information:\n(1)Department of Basic Medicine, Changzhi Medical College, Changzhi, China.\n(2)Department of Anesthesiology, Changzhi Medical College, Changzhi, China.\n\nLung cancer remains a leading cause of global morbidity and mortality, imposing \na significant healthcare burden worldwide. To comprehensively investigate \npotential risk factors for lung cancer and provide a theoretical basis for its \nprevention and treatment, we utilized publicly available genome-wide association \nstudy data of European ancestry. We treated various traits as exposures and lung \ncancer as the outcome. Causal relationships were assessed using 5 Mendelian \nrandomization (MR) methods: inverse-variance weighted, constrained maximum \nlikelihood and model averaging, debiased inverse-variance weighted, \ncontamination mixture, and Mendelian randomization-robust adjusted profile \nscore. We subsequently performed meta-analysis and median-based MR to synthesize \nthe roles of different risk factors in lung cancer development. Following \noutlier removal via radial MR and MR analyses, we identified 88, 113, 248, and \n232 exposure datasets showing causal associations with lung adenocarcinoma, \nsmall cell lung cancer, lung cancer overall, and squamous cell lung cancer, \nrespectively. Further integration via meta-analysis and median MR revealed that \nhigher educational attainment may reduce lung cancer risk by delaying age at \nfirst sexual intercourse, increasing age at first childbirth, lowering body mass \nindex and adiposity, and reducing smoking frequency. Our comprehensive analysis \nsuggests that education may play a protective role against lung carcinogenesis, \npotentially mediated through behavioral changes such as reduced smoking and \nmodifications in physical health indicators including body mass index and \nadiposity.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045651\nPMID: 41204557 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no conflicts of interest to \ndisclose."
        ],
        [
         "23",
         "41204550",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45617. doi: \n10.1097/MD.0000000000045617.\n\nDevelopments in diagnosis and treatment of early hypopharyngeal carcinoma and \nprecancerous lesions: A review.\n\nCheng J(1)(2), Liu Y(3), Tang X(1), Zhao R(2), Bao Y(1)(2).\n\nAuthor information:\n(1)Department of Gastroenterology, The Affiliated Hospital of Southwest Medical \nUniversity, Luzhou, China.\n(2)Department of Endoscopy, Sichuan Clinical Research Center for Cancer, Sichuan \nCancer Hospital & Institute, Sichuan Cancer Center, Affiliated Cancer Hospital \nof University of Electronic Science and Technology of China, Chengdu, China.\n(3)Department of Gastroenterology and Hepatology, West China Hospital, Sichuan \nUniversity, Chengdu, China.\n\nHypopharyngeal carcinoma (HC) is one of the rarest and worst head and neck \ncancers (HNC) with insidious location, atypical early symptoms, and frequently \ndiagnosed in advanced stages. The risk factors of HC mainly include smoking and \ndrinking. Preventive interventions directed towards environmental risk factors \nare highly likely to yield far-reaching beneficial effects in reducing the \nincidence of this disease. The majority of advanced HC patients undergo \nmultimodal treatments that encompass radiotherapy, chemotherapy and surgery, \nwith poor oncological and functional outcomes. The early diagnosis and \nmanagement of HC are vital for improving the prognosis of patients. Currently, \nthere are few reviews specifically focusing on early HC, thus this review aims \nto summarize the current advancements in the diagnosis and management of early \nHC. Magnifying endoscopy with narrow-band imaging (ME-NBI) can improve the \ndetection rate of early lesions and assess the invasion depth by enhancing the \nvisualization of microvascular patterns. In recent years, artificial \nintelligence (AI) presents a promising prospect in the detection of early HC \nthrough deep learning algorithms. Larynx-preserving surgery or radiotherapy \nalone is predominantly recommended for individuals diagnosed at an early stage, \naiming to achieve optimal oncological control while preserving crucial \nfunctions. With the increasing detection rates of early HC and the development \nof surgical equipment, transoral minimally invasive surgery is becoming more \nwidely applied in HC. Its minimally invasive nature offers reduced morbidity and \nfaster recovery, making it an attractive option for eligible patients with early \nHC.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045617\nPMID: 41204550 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no conflicts of interest to \ndisclose."
        ],
        [
         "24",
         "41204543",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45769. doi: \n10.1097/MD.0000000000045769.\n\nDevelopment and validation of a prognostic model for endometrial carcinoma using \ncausal genes identified by Mendelian randomization.\n\nYang D(1), Li Y(2), Xin X(1), Ma H(1).\n\nAuthor information:\n(1)Department of Gynecology and Obstetrics, the 940th Hospital of Joint \nLogistics Support Force of Chinese PLA, Lanzhou, Gansu Province, China.\n(2)Department of Hematology, the 940th Hospital of Joint Logistics Support Force \nof Chinese PLA, Lanzhou, Gansu Province, China.\n\nEndometrial carcinoma (EC) remains an ambiguous pathogenesis. This study aimed \nto investigate the potential of causal genes in predicting EC prognosis. The \nprognostic biomarkers of EC were identified using univariate Cox regression \nanalyses based on data from The Cancer Genome Atlas Uterine Corpus Endometrial \nCarcinoma (TCGA-UCEC). Mendelian randomization (MR) analyses were conducted to \ninfer causal relationships, utilizing expression quantitative trait loci (eQTLs) \nderived from prognostic genes as exposures, and a dataset from European \npopulations with EC as outcomes. Single nucleotide polymorphisms (SNPs) that \nsignificantly influenced gene expression (eQTLs) were selected as instrumental \nvariables. The inverse variance weighted (IVW) method was employed as the \nprimary analytical approach. Sensitivity analyses were performed to ensure \nrobustness of the findings. Causal genes with potential prognostic significance \nwere further evaluated using multivariate Cox regression analysis, Kaplan-Meier \n(KM) overall survival curves, and receiver operating characteristic (ROC) curve \nanalysis. Additionally, results from gene ontology (GO) and gene set enrichment \nanalysis (GSEA) of differentially expressed genes (DEGs), along with immune \ninfiltration analyses in the high- and low-risk groups, are presented. 18 genes \nexhibiting a negative correlation with EC demonstrated a protective effect, \nwhereas 9 genes identified as risk factors for EC exerted an adverse effect on \nthe disease. A prognostic model was developed consisting of 8 genes selected \nfrom 27 genes. According to the KM overall survival curve data, ECs classified \nwith high-risk ratings exhibited significantly poor prognoses (P < .0001). The \nROC curve analysis indicated that the area under the curve (AUC) for this risk \nmodel in predicting the 1-, 3-, and 5-year EC survival rates were 0.704, 0.735, \nand 0.766, respectively. Furthermore, GO and GSEA results of DEGs in both the \nhigh- and low-risk groups revealed strong associations with pathways related to \ncell motility and immune response, among others. In addition, an analysis of \nimmune cell infiltration demonstrated significant differences between the high- \nand low-risk groups. A prognostic model for EC using causal genes identified \nusing MR has good sensitivity and specificity. These findings provide new \ninsights into ECs pathogenesis and suggest promising strategies for the \ndiagnosis and treatment of ECs.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045769\nPMID: 41204543 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no conflicts of interest to \ndisclose."
        ],
        [
         "25",
         "41204540",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45172. doi: \n10.1097/MD.0000000000045172.\n\nThe diagnostic accuracy of deep learning-based AI models in predicting lymph \nnode metastasis in T1 and T2 colorectal cancer: A systematic review and \nmeta-analysis.\n\nGuo Q(1), Wang R(2), Guo Y(3).\n\nAuthor information:\n(1)Department of Radiology, First Hospital of LanZhou University, Lanzhou, \nGansu, China.\n(2)The First Clinical Medical College of Lanzhou University, Lanzhou City, \nGansu, China.\n(3)Department of Emergency, First Hospital of LanZhou University, Lanzhou, \nGansu, China.\n\nBACKGROUND: Colorectal cancer (CRC) continues to be a leading cause of \ncancer-related mortality globally, and accurately predicting lymph node \nmetastasis (LNM) in T1 and T2 lesions is vital for informing treatment \nstrategies. This study aimed to assess the diagnostic accuracy of artificial \nintelligence (AI)-based models, particularly deep learning (DL) and machine \nlearning (ML) approaches, in predicting LNM risk in CRC.\nMETHODS: A comprehensive literature search was conducted across PubMed, EMBASE, \nWeb of Science, Cochrane Library, and Scopus databases, identifying relevant \nstudies published up to April 6, 2024. A total of 6552 articles were retrieved, \nand after screening, 12 studies involving 8540 patients were included for \nqualitative analysis, with 9 studies eligible for quantitative meta-analysis. \nThe methodological quality of the studies was evaluated using the QUADAS-2 tool.\nRESULTS: Meta-analysis yielded the following diagnostic parameters: a \nsensitivity (SEN) of 0.87 (95% CI: 0.76-0.93), specificity (SPE) of 0.69 (95% \nCI: 0.52-0.82), and AUC of 0.88 (95% CI: 0.84-0.90). The likelihood ratios were \n2.80 (95% CI: 1.74-4.50) for positive and 0.18 (95% CI: 0.10-0.34) for negative \npredictions, with a diagnostic odds ratio of 15.27 (95% CI: 6.49-35.89).\nCONCLUSION: This meta-analysis indicates that AI-based models, particularly DL \nand ML techniques, demonstrate moderate SEN and good SPE in predicting LNM in T1 \nand T2 CRC lesions. These findings highlight the potential of AI models as \nnoninvasive diagnostic tools in clinical practice.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045172\nPMID: 41204540 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no conflicts of interest to \ndisclose."
        ],
        [
         "26",
         "41204525",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45356. doi: \n10.1097/MD.0000000000045356.\n\nCXCR4 expression serves as a promising candidate target in the treatment of \nbreast cancer.\n\nLiao F(1)(2), Huang J(1)(2), Zhang Y(3).\n\nAuthor information:\n(1)Department of Breast and Thyroid Surgery, The Affiliated Hospital of Youjiang \nMedical University for Nationalities, Baise, People's Republic of China.\n(2)Youjiang Medical University For Nationalities, Baise, People's Republic of \nChina.\n(3)Department of Cardiothoracic Surgery, The Affiliated Hospital of Jiujiang \nUniversity, Jiujiang, People's Republic of China.\n\nCXC motif chemokine receptor type 4 (CXCR4) has an association with normal and \nabnormal progression. Previous studies have demonstrated the involvement of \nCXCR4 in breast cancer (BC), particularly in promoting metastatic progression. \nHowever, most of the existing evidence originates from basic research. Our study \naims to evaluate novel the expression profiles of CXCR4 in BC. Computerized \nliterature search was done on the online accessible databases, including \nONCOMINE, Kaplan-Meier plotter, TCGA database and Breast Cancer Gene-Expression \nMiner v4.3 to explore the expression profile and prognostic roles of CXCR4. \nCXCR4 overexpression was associated with BC versus normal control. Elevated \nCXCR4 expression predicted better PFS, but no survival rate benefit in BC \npatients. Moreover, there is a survival rate benefit with high expression of \nCXCR4 mRNA in the negative ER but the positive ER group. CXCR4 was more \nfrequently overexpressed in BC patients with negative expression of ER and PR \nand negative expression of HER-2. For molecular subtypes analysis, higher \nexpression of CXCR4 was associated with Basal-like and HER-2 subtypes than \nluminal A and luminal B subtypes. Moreover, we demonstrate a positive \ncorrelation between high expression of CXCR4 and low expression of LASP1 and \nEIF4A1 via gene correlation targeted analysis, which is consistent with the \nresult among the triple-negative breast cancer patients. Our results suggest \nthat CXCR4 can be used as a biological factor to predict the prognosis of BC \npatients, especially those with triple-negative breast cancer, and is also a \ntumor marker with potential value in the future treatment of BC.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045356\nPMID: 41204525 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no funding and conflicts of \ninterest to disclose."
        ],
        [
         "27",
         "41204522",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45359. doi: \n10.1097/MD.0000000000045359.\n\nFactors associated with postoperative quality of life in patients with spinal \nmetastases from lung cancer.\n\nLi Y(1)(2), Liu F(1)(2), Liu Z(1)(2), Liu X(1)(2), Zhou H(1)(2), Liu X(1)(2), \nTang Y(1)(2), Hu P(1)(2), Wei F(1)(2).\n\nAuthor information:\n(1)Department of Orthopedics, Peking University Third Hospital, Beijing, China.\n(2)Beijing Key Laboratory of Spinal Disease Research, Beijing, China.\n\nThe management of spinal metastases presents a significant challenge for spine \nsurgeons, especially in cases of lung cancer, which is associated with the \npoorest prognosis among primary cancer types. This study aimed to evaluate the \npostoperative quality of life of patients with symptomatic spinal metastases \nfrom lung cancer and identify clinical factors associated with improved outcomes \nregarding quality of life. This was a retrospective review of a prospectively \nmaintained database from November 2009 to November 2020, including 128 patients \nwho underwent surgery for symptomatic spinal metastases from lung cancer. The \nprimary outcome was the change in Karnofsky Performance Status (KPS) at 1 week \nand 6 months post-operation. Patients were dichotomized into an \"improvement\" \ngroup (KPS increased) and a \"non-improvement\" group (KPS stable or decreased). \nBinary logistic regression was used to identify independent preoperative factors \nassociated with KPS improvement. A total of 72 males and 56 females, with a mean \nage of 60 ± 10 years, were enrolled in the current study. Male sex was \nassociated with improved quality of life in the short-term post-operation (odds \nratio [OR] = 0.42, 95% confidence interval [CI] [0.716-0.962]). Conversely, the \nnumber of total bone metastatic sites was negatively associated with short-term \nimprovements in quality of life (OR = 3.66, 95% CI [1.55-8.67]). Additionally, a \nhigher number of total bone metastatic sites was linked to reduced long-term \nimprovements in quality of life, with an OR of 1.94 and a 95% CI of [1.05-3.59]. \nThe number of bone metastasis sites is closely associated with postoperative \nquality of life in patients with spinal metastases from lung cancer. Careful \npatient selection is crucial and has the potential to significantly enhance the \nquality of life for these vulnerable individuals with limited life expectancy.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045359\nPMID: 41204522 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no conflicts of interest to \ndisclose."
        ],
        [
         "28",
         "41204520",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45410. doi: \n10.1097/MD.0000000000045410.\n\nComputational identification of ECT2 as a potential pan-cancer biomarker and \ntherapeutic target through integrated genomic data analysis.\n\nChen M(1), Wang L, Song G.\n\nAuthor information:\n(1)Department of Gastrointestinal Surgery, The Second Hospital of Tianjin \nMedical University, Tianjin, China.\n\nThe ECT2 gene, encoding a guanine nucleotide exchange factor, plays crucial \nroles in cell cycle progression and cytoskeletal dynamics, implicating its \ninvolvement in various cancers. However, a comprehensive pan-cancer analysis \nintegrating genomic data is still lacking. This study employed an integrated \napproach using data from multiple cancer genomics databases to assess ECT2 \nacross various malignancies. Expression profiles of ECT2 were analyzed for \ndifferential expression across tumor stages and its association with clinical \noutcomes. Correlation analyses examined the relationship between ECT2 expression \nand immune cell infiltration levels. Pathway enrichment analysis identified \nbiological processes influenced by ECT2 dysregulation in cancer progression. \nThese methods facilitated a comprehensive exploration of ECT2's role in cancer \nbiology, revealing potential implications for diagnosis, prognosis, and therapy. \nAnalysis across 33 tumor types consistently shows elevated ECT2 expression. ECT2 \ncorrelates with tumor staging in 8 cancers and molecular subtypes in 13 cancers, \nand shows associations with immune subtypes in 22 cancers, suggesting its role \nin cancer progression and immune modulation. ECT2 demonstrates strong diagnostic \npotential (the area under the ROC curve > 0.9) in 16 cancers and correlates with \npoorer overall survival in 11 cancers. ECT2 expression correlates positively \nwith microsatellite instability in STAD, MESO, UCEC, and READ, and negatively in \nDLBC; it correlates positively with tumor mutational burden in STAD, PAAD, ACC, \nLGG, and LUAD, and negatively in THYM. ECT2 also exhibits diverse correlations \nwith immune checkpoint genes and specific immune cell types identified through \nCIBERSORT analysis. ECT2 interacts with proteins like RACGAP1, KIF23, enriched \nin pathways involving cell polarity, Ras signaling, and tight junctions, \nimpacting cancer progression and stemness in various cancer types. This study \noffers comprehensive insights into ECT2's role in cancer biology through \nintegrative bioinformatics analyses. The results advocate for ECT2 as a \npotential biomarker and therapeutic target in diverse malignancies, suggesting \navenues for personalized oncology strategies.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045410\nPMID: 41204520 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no funding and conflicts of \ninterest to disclose."
        ],
        [
         "29",
         "41204513",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45965. doi: \n10.1097/MD.0000000000045965.\n\nAn atypical presentation of NSCLC in a nonsmoker: Shoulder pain and upper \nextremity DVT: A case report.\n\nÖzdemir E(1), Yilmaz T(1), Özdemir P(2), Düzenci D(3), Özdemir E(4).\n\nAuthor information:\n(1)Department of Internal Medicine, Elaziğ Fethi Sekin City Hospital, Elaziğ, \nTurkey.\n(2)Department of Nephrology, Dr. Lütfi Kirdar City Hospital, Istanbul, Turkey.\n(3)Department of Intensive Care, Elaziğ Fethi Sekin City Hospital, Elaziğ, \nTurkey.\n(4)Department of Nephrology, Pendik State Hospital, Istanbul, Turkey.\n\nRATIONALE: Lung cancer is a leading cause of cancer-related mortality and is \noften diagnosed late due to nonspecific or extrapulmonary symptoms. Atypical \npresentations in never-smokers with adenocarcinoma may delay diagnosis. This \ncase highlights the diagnostic value of shoulder pain caused by unprovoked upper \nextremity deep vein thrombosis (UEDVT) as an early manifestation of thoracic \nmalignancy.\nPATIENT CONCERNS: A 62-year-old male nonsmoker presented with a month-long \nhistory of progressive right shoulder pain radiating to the neck, without \nrespiratory symptoms or trauma. The pain persisted despite analgesics.\nDIAGNOSES: Doppler ultrasonography showed acute thrombosis of the right \nsubclavian, axillary, brachial, and internal jugular veins, explaining the \nshoulder pain. Contrast-enhanced computed tomography (CT) revealed a right hilar \nmass compressing vascular structures, and positron emission tomography-CT \ndemonstrated mild hypermetabolism (maximum standardized uptake value = 3.1) \nwithout metastasis. Endobronchial biopsy confirmed epidermal growth factor \nreceptor-mutant invasive adenocarcinoma.\nINTERVENTIONS: The patient was hospitalized for extensive UEDVT and treated with \nlow-molecular-weight heparin, later switched to the direct oral anticoagulant. \nNo thrombectomy or thrombolysis was required. A multidisciplinary tumor board \nplanned surgical or chemoradiotherapeutic management based on resectability.\nOUTCOMES: Symptoms improved with anticoagulation, and the patient remained \nclinically stable pending oncologic therapy.\nLESSONS: Lung adenocarcinoma may rarely present with isolated shoulder pain as \nthe initial symptom. When UEDVT is identified without an evident provoking \nfactor, underlying malignancy should be thoroughly investigated. Additionally, \nepidermal growth factor receptor-mutant adenocarcinomas may demonstrate only \nmild fluorodeoxyglucose uptake on positron emission tomography-CT, emphasizing \nthat low metabolic activity does not exclude cancer.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045965\nPMID: 41204513 [Indexed for MEDLINE]\n\nConflict of interest statement: The author have no funding and conflicts of \ninterest to disclose."
        ],
        [
         "30",
         "41204507",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45499. doi: \n10.1097/MD.0000000000045499.\n\nDiagnostic value of automated breast volume scanner for breast ductal carcinoma \nin situ.\n\nTian C(1), Wu J(2), Wang Z(3).\n\nAuthor information:\n(1)Pediatrics Department, The First Affiliated Hospital of Dalian Medical \nUniversity, Dalian City, Liaoning Province, China.\n(2)Ultrasound Department, The First Affiliated Hospital of Dalian Medical \nUniversity, Dalian City, Liaoning Province, China.\n(3)Nutrition Department, The First Affiliated Hospital of Dalian Medical \nUniversity, Dalian City, Liaoning Province, China.\n\nThis study explores the differential diagnostic value of conventional \ndimensional ultrasound (US) and automated breast volume scanner (ABVS) for \nbreast ductal carcinoma in situ (DCIS) patients. A total of 986 female patients \nwho underwent breast tumor surgery in our hospital from December 2019 to \nDecember 2022 were included. Clinical, US, ABVS, and pathological information \nwere collected from all of the patients. Pathological results were used to \nseparate patients into 3 groups: benign, DCIS, and invasive ductal carcinoma \n(IDC). Single-factor and multivariate analyses were conducted to evaluate the \ncharacteristics of DCIS. Of the 986 patients with breast tumors included in this \nstudy, 498, 193, and 295 were diagnosed with benign, DCIS, and IDC tumors, \nrespectively. Compared with benign tumors, DCIS tumors were characterized by \nhigher age at onset and breast imaging-reporting and data system (BI-RADS) \ngrades, together with high rates of extension to the nipple, microcalcification, \nconvergence sign, abundant blood supply, and nipple discharge proportion. \nRelative to IDC patients, DCIS tumors exhibited lower BI-RADS grades, aspect \nratio values, and Ki-67 index values together with lower rates of irregular \nmorphology, unclear boundaries, posterior echo attenuation, convergence sign, \nenhanced peripheral echo, nipple discharge, and palpable masses, and higher \nrates of extension to the nipple. Higher microcalcification rates in DCIS tumors \nwere observed than the proportion of non-calcification in IDC cases. \nConventional US and ABVS images of DCIS tumors exhibit certain distinctive \ncharacteristics that can aid in the differential diagnosis of DCIS.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045499\nPMID: 41204507 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no conflicts of interest to \ndisclose."
        ],
        [
         "31",
         "41204503",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45857. doi: \n10.1097/MD.0000000000045857.\n\nSynchronous primary gallbladder squamous cell carcinoma and colon \nadenocarcinoma: A case report and literature review.\n\nShin MH(1), Kim YH(1), Lim SC(2).\n\nAuthor information:\n(1)Department of Surgery, School of Medicine, Chosun University, Gwangju, Korea.\n(2)Department of Pathology, School of Medicine, Chosun University, Gwangju, \nKorea.\n\nRATIONALE: Gallbladder (GB) malignancy is the most common biliary malignancy; \nhowever, squamous cell carcinoma of the gallbladder (GBSCC) is extremely rare. \nSynchronous primary GB and colon cancers are very rare, but they are still \nadenocarcinomas.\nPATIENT CONCERNS: The patient was a 68-year-old male who had symptoms of anemia \nand underwent a comprehensive examination. Irregular thickening of the GB wall \nand localized bowel wall thickening in the ascending colon were observed.\nDIAGNOSES: Colonoscopy revealed a large mass in the ascending colon, and \nadenocarcinoma was diagnosed by biopsy.\nINTERVENTIONS: Right hemicolectomy and extended cholecystectomy were \nimplemented.\nOUTCOMES: The patient was diagnosed with synchronous primary pure GBSCC and \ncolon adenocarcinoma. Anemia was found on the 10th postoperative day, and \nsyncope occurred on the 12th day; computed tomography showed a gastroduodenal \nartery aneurysm, and covered stent insertion was attempted but failed. \nConsequently, intimal dissection and common hepatic artery embolization \nresulted, leading to hepatic failure, and the patient died on the 18th day.\nLESSONS: We report a case of synchronous primary pure GBSCC and colon \nadenocarcinoma. Synchronous primary GB and colon adenocarcinomas have rarely \nbeen reported. However, synchronous primary pure GBSCC and colon adenocarcinomas \nhave not yet been reported in English literature. Here, we present a literature \nreview on the pathogenesis of GBSCC and synchronous GB and colon cancer.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045857\nPMID: 41204503 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no conflicts of interest to \ndisclose."
        ],
        [
         "32",
         "41204490",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45843. doi: \n10.1097/MD.0000000000045843.\n\nIatrogenic diaphragmatic hernia - a serious delayed complication: Three cases \nreport and literature review.\n\nJin C(1), Yang X(2), Han T(1), Yu M(1), Zhou J(1), Dai Y(1).\n\nAuthor information:\n(1)Department of Gastroenterology, The Fourth Affiliated Hospital of School of \nMedicine, and International School of Medicine, International Institutes of \nMedicine, Zhejiang University, Yiwu, China.\n(2)Department of General Surgery, The Fourth Affiliated Hospital of School of \nMedicine, and International School of Medicine, International Institutes of \nMedicine, Zhejiang University, Yiwu, China.\n\nRATIONALE: Iatrogenic diaphragmatic hernia (IDH) is rare, however, its incidence \nhas increased with the growing number of thoracic and abdominal operations. IDH \nis a serious delayed complication that is easily missed and may lead to severe \nconsequences. Recognizing and emphasizing the occurrence of IDH may help reduce \nits incidence. We report three cases of IDH from the past decade, each resulting \nfrom different operations, and managed with distinct treatments.\nPATIENT CONCERNS: The first case occurred 4 months after a left upper lobectomy. \nDue to a lack of awareness, improper placement of a gastric tube resulted in \ngastric perforation.The second patient presented with symptoms 10 years after \nendoscopic full-thickness resection of a gastric fundus gastrointestinal stromal \ntumor and was diagnosed 12 years after the operation. The cause was likely \ndiaphragmatic injury during endoscopic full-thickness resection, owing to the \nproximity of the diaphragm to the gastric fundus.The third case occurred 9 \nmonths after radiofrequency ablation (RFA) for liver cancer, which involved \npenetration through the diaphragm and caused iatrogenic damage.\nDIAGNOSES: The first patient was initially misdiagnosed at a local hospital and \nwas finally diagnosed with IDH and gastric perforation using computerized \ntomography (CT), gastroscopy, and upper gastrointestinal radiography. The second \npatient was diagnosed by CT and upper gastrointestinal radiography. The third \npatient was diagnosed by enhanced CT.\nINTERVENTIONS: Surgical repair was performed on the first and third patients. \nThe second patient declined surgery due to advanced age and having adapted to \nthe discomfort caused by IDH.\nOUTCOMES: The first patient recovered well postoperatively and showed no \nrecurrence during the 10-year follow-up period. The second patient was \nexperienced persistent discomfort, which did not worsen throughout the 5-year \nfollow-up. The third patient also recovered well postoperatively and showed no \nrecurrence during the 1-year follow-up period.\nLESSONS: We emphasize that when a patient presents with relevant symptoms and \nsignificant unilateral diaphragmatic elevation is observed on imaging, the \npossibility of a diaphragmatic hernia should be highly suspected, and early \ndiagnosis and treatment are essential. Furthermore, during operations that carry \na risk of diaphragmatic injury-particularly hepatic operations - we recommend \nmeticulous protection of the diaphragm to prevent IDH.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045843\nPMID: 41204490 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no conflicts of interest to \ndisclose."
        ],
        [
         "33",
         "41204487",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45348. doi: \n10.1097/MD.0000000000045348.\n\nDeciphering the pharmacological mechanisms of Weiweisu decoction in chronic \natrophic gastritis: Insights from network pharmacology, molecular dynamics, and \nin vivo validation.\n\nChen M(1)(2), Liu J(2), Wu S(2), Zhu X(1), Jin Y(2), Dang Y(1), Shen J(3), Chen \nH(2), Xu J(4), Bie C(2).\n\nAuthor information:\n(1)Guangzhou University of Chinese Medicine, Guangzhou, China.\n(2)Shenzhen Clinical College of Integrated Chinese and Western Medicine, \nGuangzhou University of Chinese Medicine, Shenzhen, Guangdong, China.\n(3)Shaanxi University of Chinese Medicine, Xianyang, China.\n(4)Guangzhou Medical University, Guangzhou, China.\n\nBACKGROUND: This study aims to identify the phytochemical components of Weiweisu \n(WWS) decoction, evaluate its therapeutic potential for chronic atrophic \ngastritis (CAG), and elucidate the underlying mechanisms with a focus on \nantiferroptotic effects.\nMETHODS: Liquid chromatography-mass spectrometry was used to characterize the \nphytochemical constituents of WWS. Network pharmacology was applied to predict \npotential therapeutic targets and signaling pathways. Core compounds and targets \nwere verified through molecular docking and molecular dynamics simulations. \nSubsequently, a CAG rat model was established. Enzyme-linked immunosorbent assay \nwas used to measure levels of prostaglandins (pepsinogen I/II) and tumor \nnecrosis factor-α in gastric tissue, while Fe2+, glutathione, and \nmalondialdehyde concentrations were assessed. Gene and protein expression levels \nof key targets were analyzed using quantitative real-time polymerase chain \nreaction and western blotting.\nRESULTS: A total of 1017 chemical constituents were identified via liquid \nchromatography-mass spectrometry. Network pharmacology revealed 20 \nferroptosis-related genes associated with WWS and CAG, primarily involving \ncancer-related signaling pathways. In vivo experiments showed that WWS \nameliorated gastric ulcers and mucosal atrophy, downregulated pepsinogen I/II \nand tumor necrosis factor-α levels, reduced Fe2+ and malondialdehyde \nconcentrations, and increased glutathione levels. WWS treatment also elevated \nHSP90AA1, MAPK, and mTOR expression while inhibiting GPX4 expression. Molecular \ndocking indicated strong binding affinities between key active compounds and \ncore targets. Molecular dynamics simulations confirmed the stability of these \ninteractions.\nCONCLUSION: In summary, WWS decoction can treat CAG through antiferroptosis, and \nits main mechanism may be related to the regulation of tumor signaling pathways.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045348\nPMID: 41204487 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no conflicts of interest to \ndisclose."
        ],
        [
         "34",
         "41204484",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45371. doi: \n10.1097/MD.0000000000045371.\n\nInvestigating the causal relationship between dietary habits and oral cancer \nrisk: A Mendelian randomization analysis.\n\nDing H(1), Gao Y(2), Ma S(1), Zhang C(1).\n\nAuthor information:\n(1)The Second Clinical Medical School, Henan University of Chinese Medicine, \nZhengzhou, China.\n(2)Orthopedics & Traumatology College, Henan University of Chinese Medicine, \nZhengzhou, China.\n\nThe main goal of this study was to clarify the causal link between dietary \nhabits and the occurrence of oral cancer via a Mendelian randomization (MR) \napproach. This method offers valuable insights that can inform the creation of \neffective management strategies in clinical settings. This investigation \ninvolved a comprehensive dataset that included 8,943,774 dietary samples and \n3,723,373 oral cancer cases sourced from genome-wide association studies. To \nexplore the possible causal relationship between dietary habits and the \noccurrence of oral cancer, various statistical methods, such as inverse variance \nweighted MR analysis, the weighted median approach, and MR-Egger regression \nanalysis, have been employed. The variability in the results was evaluated via \nthe Cochran Q statistic, and to address the issue of multiple comparisons, the \nMR-Egger and MR-PRESSO methods were utilized. Additionally, sensitivity analysis \nwas conducted via the leave-one-out technique to ensure the reliability of the \nfindings. Among 59 instrumental variables, inverse variance weighted analysis \nrevealed an inverse relationship between oily fish intake and oral cancer risk \n(odds ratio = 0.998; 95% confidence interval: 0.997-1.000; P = .024). \nNonsignificant results emerged from MR-Egger (P = .171) and weighted median \n(P = .483). Cochran Q testing confirmed no single-nucleotide polymorphism \nheterogeneity (P > .05), while MR-Egger (intercept = 3.776E-05, P = .389) and \nMR-PRESSO (P = .534) detected no horizontal pleiotropy. Leave-one-out analysis \ndemonstrated robustness, with no single single-nucleotide polymorphism \nsignificantly influencing outcomes. Oily fish intake may reduce the incidence of \noral cancer.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045371\nPMID: 41204484 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no conflicts of interest to \ndisclose."
        ],
        [
         "35",
         "41204478",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45927. doi: \n10.1097/MD.0000000000045927.\n\nPressurized intraperitoneal aerosol chemotherapy in peritoneal carcinomatosis \ndue to recurrent ovarian cancer.\n\nÖzcan P(1), Düzgün Ö(2).\n\nAuthor information:\n(1)Department of General Surgery, Yerköy Şehit Korgeneral Osman Erbaş State \nHospital, Yerköy, Yozgat, Türkiye.\n(2)Department of Surgical Oncology, University of Health Sciences, İstanbul \nUmraniye Training and Research Hospital, Istanbul, Türkiye.\n\nOvarian cancer is typically diagnosed at stages 3 or 4, and up to 80% of \npatients develop peritoneal metastasis. Although initial chemotherapy with \ncarboplatin and paclitaxel is effective, recurrence rates for recurrent ovarian \ncancer (ROC) reach approximately 75%. Recently, a minimally invasive approach \nknown as pressurized intraperitoneal aerosol chemotherapy (PIPAC) has been \ndeveloped, in which chemotherapy is administered as a pressurized aerosol under \nlaparoscopy. This study aimed to present the early outcomes of PIPAC for \nperitoneal carcinomatosis due to ROC. Data from ROC patients with ROC treated \nbetween January 2020 and January 2025 were retrospectively reviewed. PIPAC was \nperformed alongside systemic chemotherapy for 6-week cycles, with at least 3 \nsessions. The chemotherapy regimen consisted of doxorubicin 2.1 mg/m² and \ncisplatin 10.5 mg/m² administered via a nebulizer and injector, respectively, at \n37°C for 30 minutes. Statistical analysis was conducted using SPSS version 25.0. \nNormality was assessed with histogram plots and the Shapiro-Wilk test, as it is \nmore appropriate for small sample sizes. Categorical variables were compared \nwith the chi-squared test. The Mann-Whitney U and Kruskal-Wallis tests were used \nfor non-parametric comparisons, with post hoc analysis as needed. Spearman \ncorrelation assessed relationships between variables. Cox regression analyzed \nfactors affecting survival, and Kaplan-Meier analysis was used for overall \nsurvival. A P-value < .05 was considered statistically significant. A total of \n26 PIPAC cycles were performed in 11 ROC patients with ROC. The mean \npretreatment Peritoneal Carcinomatosis Index score was 23.18, which decreased to \n15.36 after systemic chemotherapy and PIPAC. According to the Peritoneal \nRegression Grading Score, 3 (27.3%) patients had a complete response, 4 (36.4%) \nhad a major response, 3 (27.3%) had a minor response, and 1 (9%) had no \nresponse. Two patients (18.2%) underwent secondary cytoreductive \nsurgery + hyperthermic intraperitoneal chemotherapy. The median overall survival \nwas 15.13 months. Early results indicated that PIPAC may improve survival in \npatients with platinum-resistant ROC without increasing morbidity. Approximately \n20% of patients are candidates for secondary cytoreductive surgery. The low \ntoxicity and repeatability of PIPAC when combined with systemic chemotherapy may \nimprove treatment efficacy. Larger multicenter randomized trials are needed.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045927\nPMID: 41204478 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no funding and conflicts of \ninterest to disclose."
        ],
        [
         "36",
         "41204475",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45918. doi: \n10.1097/MD.0000000000045918.\n\nIncidentalomas discoveries during staging and surveillance for colorectal cancer \npatients.\n\nLiu XR(1)(2), Chen J(3), Zhang W(1), Liu Y(4), Peng D(1), Du DL(1).\n\nAuthor information:\n(1)Department of Gastrointestinal Surgery, The First Affiliated Hospital of \nChongqing Medical University, Chongqing, China.\n(2)Department of Radiology, The Second Affiliated Hospital of Chongqing Medical \nUniversity, Chongqing, China.\n(3)Department of Radiology, Qijiang People's Hospital, Chongqing, China.\n(4)Department of Radiology, The First Affiliated Hospital of Chongqing Medical \nUniversity, Chongqing, China.\n\nThis study attempted to evaluate the incidence of incidentalomas based on \ncomputed tomography (CT) in colorectal cancer (CRC) patients. CRC patients who \nobtained plan or enhanced CT for the whole abdominal and pelvis were included at \nthe First Affiliated Hospital of Chongqing Medical University, Chongqing, China. \nIncidentalomas, including uterine tumors, adrenal gland tumors, renal cancer, \npancreatic tumors, prostatic tumors, ovarian tumors, upper-tract urothelial \ncancer, and gallbladder tumors, were assessed based on all radiology reports by \nradiologists and surgeons. Moreover, the clinical characteristics of all \npatients were collected. A total of 7053 CRC patients (mean age, 62.6 ± 12.3; \n4139 male) were finally included, 255 (3.6%) patients had an incidentaloma. The \nproportions of uterine tumors, adrenal gland tumors, renal cancer, pancreatic \ntumors, and prostatic tumors were 1.3% (92/7053), 1.1% (77/7053), 0.7% \n(46/7053), 0.1% (15/7053), and 0.1% (10/7053), respectively. Ovarian tumors, \nupper-tract urothelial cancer, and gallbladder tumors all had an incidence of \n<0.1%. The prevalence of incidentalomas in CRC patients was 3.6% (255/7053). The \nmost common incidentalomas was uterine tumor, followed by adrenal gland tumor, \nand renal cancer. These findings highlighted the importance of careful \nevaluation of abdominal and pelvic CT scans in CRC patients, as timely detection \nand management of incidentalomas might optimize treatment strategies and improve \npatient outcomes.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045918\nPMID: 41204475 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no funding and conflicts of \ninterest to disclose."
        ],
        [
         "37",
         "41204473",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45333. doi: \n10.1097/MD.0000000000045333.\n\nCausal relationship between serum bilirubin, cholelithiasis and hepatobiliary \nand pancreatic malignancy: A bidirectional and 2-sample Mendelian randomization \nstudy.\n\nLuo T(1)(2), Wang S(3), Yang S(1)(2), Zhao X(1)(2), Su A(1)(2), Qin J(1)(2), Liu \nY(4), Zhou W(1)(2).\n\nAuthor information:\n(1)Department of General Surgery, The Second Hospital of Lanzhou University & \nThe Second Clinical Medical School, Lanzhou University, Lanzhou, Gansu, China.\n(2)Gansu Province Key Laboratory of Environmental Oncology, Lanzhou, Gansu, \nChina.\n(3)Hematology Department, Second Hospital of Shanxi Medical University, Taiyuan, \nShanxi, China.\n(4)Department of Oncology, Lanzhou University Second Hospital, Lanzhou, Gansu, \nChina.\n\nSince the causal relationship between cholelithiasis and serum bilirubin levels \nand hepatobiliary and pancreatic malignancies has been inconclusive and \ninconsistent for a long time, to control confounding factors and reverse \ncausality as much as possible, this paper adopted a Mendelian randomization (MR) \nstudy to reveal further the causal relationship between different exposure \nfactors and hepatobiliary and pancreatic malignancies. This study selected 4 \nexposure factors, including cholelithiasis, serum total bilirubin, direct \nbilirubin, and indirect bilirubin, by double-sample and bidirectional MR method. \nThe results were as follows: gallbladder carcinoma, liver hepatocellular \ncarcinoma (LIHC), and pancreatic adenocarcinoma (PADC) were selected as the \noutcome, and significance level (P < 5e-08) was used as the filtering criterion \nto select genetic variation as the instrumental variable for follow-up analysis. \nThen, we calculated the odds ratio between exposure and outcome to show the \ncausal relationship between different exposure factors and outcome. A series of \nsensitivity analyses were conducted to verify the conclusions' reliability. \nThere was a causal relationship between direct bilirubin and gallbladder cancer, \nand direct bilirubin was a risk factor for gallbladder cancer (odds \nratio = 3.07, 95% confidence interval = 1.02-9.22, P = .04). We then performed a \nreverse MR analysis on this analysis to further analyze the possible reverse \ncausality between the 2 and found no reverse causality between the 2 (P = .22). \nAt the same time, other exposure factors in the study did not show a causal \nrelationship with the outcome. This study showed that serum direct bilirubin is \na risk factor for gallbladder cancer. The conclusion was statistically \nsignificant (P = .04), and reverse MR excluded the reverse causal association \nbetween the 2. Cholelithiasis, serum total bilirubin, and indirect bilirubin \nwere not causally associated with gallbladder cancer, and none of the exposure \nfactors in the study showed a causal association with hepatocellular carcinoma \nand pancreatic cancer.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045333\nPMID: 41204473 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no conflicts of interest to \ndisclose."
        ],
        [
         "38",
         "41204468",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45474. doi: \n10.1097/MD.0000000000045474.\n\nDevelopment and validation of a nomogram to predict overall survival in elderly \npatients with pancreatic cancer without surgical treatment.\n\nLi K(1), Xu R(1), Yan J(2), Guo M(1), Zhang H(1).\n\nAuthor information:\n(1)Department of General Surgery, Linyi People's Hospital, Linyi, Shandong \nProvince, China.\n(2)Department of Gastroenterology, Linyi People's Hospital, Linyi, Shandong \nProvince, China.\n\nElderly patients with pancreatic cancer (PC) not treated by surgery have \nextremely low overall survival (OS), and the aim of this study is to investigate \nthe prognostic factors of elderly patients with PC not treated by surgery and to \nestablish and validate a nomogram to predict their OS. Data on PC patients over \n65 years of age who were not treated surgically were downloaded from the \nSurveillance, Epidemiology, and End Results database from 2010 to 2015. Patients \nfrom 2010 to 2014 were randomized into a training group (n = 2044) and a \nvalidation group (n = 877). Patients from 2015 (n = 662) were used for external \nvalidation. Cox analysis was performed to determine prognostic factors, and a \nnomogram was constructed to predict 1-, 2-, and 3-year OS in elderly PC patients \nwithout surgical treatment. The nomogram was validated using the area under the \nreceiver operating characteristic curve (AUC) and the calibration curve. \nPatients were categorized into low- and high-risk groups based on the median \nrisk score for Kaplan-Meier survival analysis. Age, tumor size, grade, \nradiotherapy, and chemotherapy were independent prognostic factors affecting OS, \nand a nomogram was constructed. In the training cohort, the AUCs of the \npredicted nomogram for 1, 2, and 3 years were 0.725, 0.738, and 0.741, and in \nthe validation cohort, the AUCs of the predicted nomogram for 1, 2, and 3 years \nwere 0.716, 0.733, and 0.731, while in the test cohort, the AUCs of the \npredicted nomogram for 1, 2, and 3 years were in the test cohort were 0.777, \n0.794, and 0.813, respectively. The calibration curves showed that the actual OS \nat 1, 2, and 3 years was in good agreement with the predicted OS. In addition, \nthe OS of patients in the low-risk group was significantly better than that of \nhigh-risk patients. We developed a new nomogram to assess the prognosis of \nelderly patients with non-operatively treated PC and to develop clinical \ntreatment strategies.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045474\nPMID: 41204468 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no conflicts of interest to \ndisclose."
        ],
        [
         "39",
         "41204467",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45743. doi: \n10.1097/MD.0000000000045743.\n\nAssociation between systemic immune-inflammatory index and body mass index in \ncancer patients: A cross-sectional study from NHANES 2013 to 2018.\n\nZheng J(1), Zheng P(1), Hong W(1), Chen T(1), Liu J(1), Gu J(2).\n\nAuthor information:\n(1)The First School of Clinical Medicine, Zhejiang Chinese Medical University, \nHangzhou, Zhejiang, China.\n(2)Department of Oncology, The First Affiliated Hospital of Zhejiang Chinese \nMedical University (Zhejiang Provincial Hospital of Traditional Chinese \nMedicine), Hangzhou, Zhejiang, China.\n\nA novel integrative biomarker, the systemic immune-inflammatory index (SII), has \nbeen understudied in cancer research. This study aimed to examine potential \nassociations between body mass index (BMI) and SII in cancer patients. It \nincluded cancer patients aged 20 to 79 from the National Health and Nutrition \nExamination Survey (NHANES) conducted between 2013 and 2018. A cross-sectional \nanalysis comparing SII and BMI was performed using EmpowerStats software \n(version 3.4.3) and R packages. We utilized multivariate regression analysis to \nexamine the association between SII and BMI in cancer patients. To further \nexplore the relationship between the 2, we employed threshold effect analysis, \nstratified analysis, and smoothed curve fitting. The study included 1067 cancer \npatients, with a mean SII value of 553.32 ± 390.44. Our multivariate regression \nanalysis revealed a positive association exists between BMI and SII in the fully \nadjusted model, with this relationship observed across all 3 groups. The \nthreshold value between log-transformed SII (lgSII) and BMI, as identified by \nthe segmented linear regression model, was 3.04 (×103 cells/μL). However, no \ninflection point was observed in female patients following stratification by \ngender. Our findings suggest that SII may reflect the systemic inflammatory \nstatus associated with obesity. However, its predictive value requires further \nvalidation in prospective studies.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045743\nPMID: 41204467 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no conflicts of interest to \ndisclose."
        ],
        [
         "40",
         "41204466",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45672. doi: \n10.1097/MD.0000000000045672.\n\nAnalysis of the effect of chemotherapy on overall survival of elderly patients \nwith non-small cell lung cancer based on the propensity score matching approach.\n\nLi F(1), Li F(2), Zhao D(1), Lu H(1).\n\nAuthor information:\n(1)Ward 2, Department of Medical Oncology, Lixin People's Hospital of Bozhou \nCity, Bozhou, Anhui, China.\n(2)Ward 1, Department of Medical Oncology, Affiliated Hospital of Qinghai \nUniversity, Xining, Qinghai, China.\n\nWe conducted a retrospective analysis of elderly stage I-III non-small cell lung \ncancer (NSCLC) in the hope of providing more evidence for adjuvant chemotherapy \nin elderly NSCLC patients. Data from NSCLC patients ≥ 70 years of age during \n2010-2015 from the Surveillance, Epidemiology, and End Results database were \nused to analyze the data. Patients were divided into chemotherapy and \nnon-chemotherapy groups. Propensity score matching was performed to construct a \nbalanced cohort of chemotherapy and non-chemotherapy. Data from the matched 2 \ngroups were subjected to Kaplan-Meier analysis, and multivariate COX regression \nanalysis was performed to evaluate the effect of chemotherapy on overall \nsurvival. Subgroup analyses were performed to determine the specific benefit \npopulation. A total of 18,126 eligible patients were enrolled in the study. \nAmong these, 4776 patients (26.35%) received chemotherapy, while 13,350 patients \n(73.65%) did not. Utilizing propensity score matching, 6974 patients were \nsuccessfully matched, achieving baseline equilibrium between the chemotherapy \nand non-chemotherapy groups. In the matched dataset, multivariate Cox regression \nanalysis indicated a 26% reduction in the risk of mortality for the chemotherapy \ngroup compared to the non-chemotherapy group (hazard ratio = 0.74, 95% \nconfidence interval = 0.70-0.78, P < .001). Furthermore, Kaplan-Meier analysis \nof the matched data demonstrated a survival advantage for the chemotherapy group \nrelative to the non-chemotherapy group, with a statistically significant \ndifference observed. Additionally, subgroup analyses revealed that specific \ndemographics, including Black patients, those with higher grades of tumor \ndifferentiation, and patients classified as stage II-III, T2-T3, and N1-N2, were \nmore likely to benefit from chemotherapy. Through the findings of this study, it \nwas demonstrated that chemotherapy can enhance the prognosis of elderly patients \nwith NSCLC. This is particularly evident in patients who are Black, possess a \nhigh tumor cell grade classification, are in advanced stages of the disease, and \nexhibit multiple peripheral lymph node metastases, as they are likely to derive \nsignificant benefits from this treatment.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045672\nPMID: 41204466 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no funding and conflicts of \ninterest to disclose."
        ],
        [
         "41",
         "41204462",
         "1. Medicine (Baltimore). 2025 Nov 7;104(45):e45951. doi: \n10.1097/MD.0000000000045951.\n\nPorcelain gallbladder secondary to chronic cholelithiasis with filled \ngallstones: A case report and a minimal literature review.\n\nPan XF(1), Aizezi B(2), Jia Z(3).\n\nAuthor information:\n(1)Department of General Surgery, The Third People's Hospital of Yuhang \nDistrict, Hangzhou, China.\n(2)Department of Thoracic Surgery, Xiang'an Hospital of Xiamen University, \nSchool of Medicine, Xiamen University, Xiamen, Fujian Province, China.\n(3)Department of Hepatobiliary Surgery, Affiliated Hangzhou First People's \nHospital, School of Medicine, West Lake University, Hangzhou, China.\n\nRATIONALE: Porcelain gallbladder (PG) is an academic term to define a special as \nwell as rare condition of some gallbladder diseases. So far, some issues still \nremain a discussion, such as etiopathogenesis, clinical features, correlation \nwith gallbladder cancer, and surgical indication. We described a typical PG \ncase, and reviewed some English literature as well as relevant cases from our \ninstitutions, aiming to further illustrate some issues mentioned above.\nPATIENT CONCERNS: Herein, we describe a 54-year-old obesity woman with \nhypertension and prediabetes who presented with recurrent abdominal pain and \nindigestion for 35 years, with diagnosis of chronic cholelithiasis.\nDIAGNOSES: She insisted on nonsurgical management for many years until a ring, \nhigh-dense shadow classically associated with PG, was found on computed \ntomography scan in the latest visit of the Emergency Department.\nINTERVENTIONS: Therefore, she underwent a laparoscopic cholecystectomy.\nOUTCOMES: The patient recovered uneventfluly after surgery, and the post \noprative pathology confirmed a PG without any cancer cells under microscopy. \nAfter a month of follow-up, she was very satisfied with the clinical outcomes, \nwith no surgery-related complications.\nLESSONS: PG has a potential though low risk of gallbladder cancerization. If \npatients' condition permit, minimally invasive cholecystectomy is still a \nprioritized choice due to its safety and reliability as well.\n\nCopyright © 2025 the Author(s). Published by Wolters Kluwer Health, Inc.\n\nDOI: 10.1097/MD.0000000000045951\nPMID: 41204462 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors have no conflicts of interest to \ndisclose."
        ],
        [
         "42",
         "41204446",
         "1. Scand J Public Health. 2025 Nov 7:14034948251384432. doi: \n10.1177/14034948251384432. Online ahead of print.\n\nStrategic communication for NCD prevention: Enhancing visibility, engagement and \npolicy action in Europe.\n\nKarlsdóttir S(1), Munch AT(2), Snorradóttir H(1).\n\nAuthor information:\n(1)The Icelandic Directorate of Health, Reykjavik, Iceland.\n(2)The Norwegian Directorate of Health, Oslo, Norway.\n\nBACKGROUND: JA PreventNCD represents a collaborative European Union initiative \naimed at reducing the burden of non-communicable diseases (NCDs) across Europe \nby targeting common risk factors and addressing health inequalities.\nAIMS: This article outlines the comprehensive communication and dissemination \nwork developed to enhance the Joint Action on Cancer and other NCDs prevention: \nAction on Health Determinants (JA PreventNCD) project's visibility, impact and \nstakeholder engagement.\nMETHODS: The communication and dissemination of the project is designed to be \nboth strategic and inclusive. The approach centres on key objectives: supporting \nthe JA PreventNCD goals of shaping future policies and advancing the prevention \nof NCDs by addressing communication challenges; increasing awareness; and \nenhancing stakeholder involvement. Using a mix of traditional and digital \nplatforms, including press releases, newsletters, policy briefs, scientific \npublications, social media and interactive events, the strategy facilitates \ntargeted engagement with policymakers, healthcare professionals, non-government \norganisations and the public. A strong emphasis on collaboration while \nrecognising each country's unique challenges and opportunities in communicating \nNCD prevention.\nCONCLUSIONS: By adhering to principles of inclusivity and accessibility, the \ncommunication work strives to engage diverse audiences effectively, encourage \ndialogue, support policy change and set the agenda. Ongoing evaluation measures \nensure that the approach remains responsive, evidence based and impactful, \naligning closely with the overarching goals of JA PreventNCD.\n\nDOI: 10.1177/14034948251384432\nPMID: 41204446"
        ],
        [
         "43",
         "41204424",
         "1. Nucleosides Nucleotides Nucleic Acids. 2025 Nov 7:1-16. doi: \n10.1080/15257770.2025.2584085. Online ahead of print.\n\nAssessment of immunohistochemical expression of oxidative stress marker 8-OHdG \nin oral lichen planus.\n\nMohideen K(1), Krithika C(2), Jeddy N(3), Ramadoss R(4), Alqahtani SS(1), Chole \nR(1), Indu Bharkavi SK(5).\n\nAuthor information:\n(1)Department of Oral and Maxillofacial Surgery and Diagnostic Sciences, Faculty \nof Dentistry, Najran University, Najran, Kingdom of Saudi Arabia.\n(2)Meenakshi Academy of Higher Education and Research, Chennai, Tamil Nadu, \nIndia.\n(3)Department of Oral Pathology, Thai Moogambigai Dental College and Hospital, \nDr. M.G.R. Educational and Research Institute, Chennai, Tamil Nadu, India.\n(4)Department of Oral Biology, Saveetha Dental College and Hospitals, Chennai, \nTamil Nadu, India.\n(5)Department of Oral Pathology and Microbiology, Sathyabama Dental College and \nHospital, Sathyabama Institute of Science and Technology, Chennai, Tamil Nadu, \nIndia.\n\nOral lichen planus (OLP) is a chronic inflammatory condition that has been \nclinically linked with the risk of developing oral cancer. The present study \naimed to determine the oxidative stress in oral lichen planus (OLP) by assessing \nthe immunohistochemical (IHC) expression of 8-hydroxydeoxyguanosine (8-OHdG) in \nOLP tissue samples and comparing it with that of normal oral mucosa. The study \ngroup consisted of 30 formalin-fixed, paraffin-embedded (FFPE) tissue blocks \nfrom oral lichen planus (OLP) cases, while 10 normal oral mucosa samples served \nas the control group. Tissue sections of 5-micron thickness were prepared and \nimmunostained with the 8-hydroxydeoxyguanosine (8-OHdG) antibody. The stained \nslides were examined under a light microscope, and statistical analysis was \nconducted using Fisher's exact test. A significant increase in oxidative DNA \ndamage marker 8-OHdG expression (p < 0.05) was observed within the basal and \nsuprabasal epithelial layers of lichen planus tissue samples. In contrast, no \nimmunoreactivity was detected in the normal oral mucosa. Excessive production of \nreactive oxygen species (ROS) during chronic inflammation is believed to play a \ncrucial role in inducing DNA damage. The mutagenic marker \n8-hydroxydeoxyguanosine (8-OHdG) in oral lichen planus tissue highlights its \npotential as a biomarker for assessing the risk of inflammation-driven \ncarcinogenesis. Further research, incorporating various oxidative stress markers \nand clinicopathological correlations, is essential to enhance early prediction \nof malignant transformation in oral lichen planus.\n\nDOI: 10.1080/15257770.2025.2584085\nPMID: 41204424"
        ],
        [
         "44",
         "41204416",
         "1. J Appl Clin Med Phys. 2025 Nov;26(11):e70325. doi: 10.1002/acm2.70325.\n\nQuantitative CT reconstruction kernel harmonization for multi-site lung cancer \nscreening.\n\nJordan DW(1), Misseldine RE(2).\n\nAuthor information:\n(1)Department of Radiology, University Hospitals Cleveland Medical Center, Case \nWestern Reserve University School of Medicine, VA Northeast Ohio Healthcare \nSystem, Cleveland, Ohio, USA.\n(2)Radiation Safety, VA Northeast Ohio Healthcare System, Cleveland, Ohio, USA.\n\nBACKGROUND: Published reference CT protocols for lung cancer screening are not \noptimized to produce uniform image appearance across different scanner \nmanufacturers and models or to conform to quantitative imaging profiles for \nrobust small lung nodule size and volume measurements, which are important in \nclinical management of screen-detected nodules.\nPURPOSE: This study used widely available phantoms and software to identify lung \ncancer screening CT reconstructions that enable accurate and reproducible nodule \nsize measurements and to match reconstructions across scanner manufacturers and \nmodels to provide a consistent image appearance to interpreting physicians.\nMETHODS: ACR CT accreditation phantom scans were used to measure the modulation \ntransfer function (MTF) and noise power spectrum (NPS) for various \nreconstruction kernels for six CT scanner models from three manufacturers. A \nreference kernel was chosen, and other kernels were matched based on agreement \nbetween MTF and NPS for each candidate kernel. Kernels were validated for \nconformance to the Quantitative Imaging Biomarkers Alliance (QIBA) Small Lung \nNodule profile using a commercial phantom and conformance testing software.\nRESULTS: Medium sharp kernels with similar MTF and NPS curves and QIBA-compliant \nimaging performance were identified for the six scanner models. One medium \nsmooth kernel with iterative reconstruction and one common lung kernel did not \nconform to the QIBA profile.\nCONCLUSIONS: MTF and NPS comparisons to a validated reference CT protocol can be \nused to identify candidate scanner-specific reconstructions appropriate for lung \ncancer screening that conform to the QIBA protocol, thus producing accurate and \nreproducible nodule size measurements, and that should provide uniform image \nappearance to the interpreting radiologist.\n\n© 2025 The Author(s). Journal of Applied Clinical Medical Physics published by \nWiley Periodicals LLC on behalf of American Association of Physicists in \nMedicine.\n\nDOI: 10.1002/acm2.70325\nPMID: 41204416 [Indexed for MEDLINE]"
        ],
        [
         "45",
         "41204401",
         "1. BMC Cancer. 2025 Nov 7;25(1):1728. doi: 10.1186/s12885-025-15109-5.\n\nFactors affecting post-traumatic growth in hematopoietic stem cell \ntransplantation patients in South Korea: a descriptive survey study.\n\nBae K(1), Jo E(2).\n\nAuthor information:\n(1)Department of Clinical Research Design & Evaluation, Samsung Advanced \nInstitute for Health Sciences & Technology (SAIHST), Sungkyunkwan University, \nSeoul, South Korea.\n(2)Department of Nursing, Kyungnam University, Changwon, South Korea. \neunjinjo@kyungnam.ac.kr.\n\nBACKGROUND: While most studies on hematopoietic stem cell transplantation (HSCT) \nhave focused on its negative impact, recent research has highlighted the concept \nof post-traumatic growth (PTG), where individuals experience positive \npsychological changes after a traumatic event. However, research on PTG in HSCT \nrecipients remains limited. This study aimed to examine PTG as the primary \noutcome among HSCT recipients and identify key psychosocial and clinical \npredictors.\nMETHODS: Adult HSCT recipients within 1 year of transplantation at a university \nhospital in Seoul were enrolled from March 2018 to March 2022. Participants \ncompleted structured questionnaires, and treatment-related data were extracted \nfrom electronic medical records. Data were analyzed using descriptive \nstatistics, t-tests, analysis of variance, Pearson's correlation, and multiple \nregression analysis.\nRESULTS: Overall, 106 patients were enrolled. The average PTG score was 73.75, \nwith higher scores observed in autologous transplant recipients compared to \nthose who received transplants from sibling and unrelated donor transplants. \nSignificant factors influencing PTG included marital status (β=-2.91, p < .001), \npositive changes after transplantation (β=-0.196, p < .009), rumination \n(β = 0.298, p < .001), transplant type (β = 0.268, p < .001), and social support \n(β = 0.191, p < .006), with the regression model explaining 53% of the variance \n(F = 20.773, p < .001).\nCONCLUSIONS: Our findings underscore the need for tailored interventions that \naddress key factors influencing PTG in HSCT recipients. They also highlight the \nessential role of oncology nurses in assessing psychosocial needs and \nimplementing individualized, evidence-based care strategies to support \npsychological recovery and promote PTG.\n\n© 2025. The Author(s).\n\nDOI: 10.1186/s12885-025-15109-5\nPMID: 41204401 [Indexed for MEDLINE]\n\nConflict of interest statement: Declarations. Ethics approval and consent to \nparticipate: Our study was approved by St. Mary’s Hospital, The Catholic \nUniversity of Korea Human Research Ethics Committee(approval no. KC16QISI0447). \nAll participants gave written informed consent to participate. All procedures \nperformed in studies involving human participants were in accordance with the \nethical standards of the institutional and/or national research committee and \nwith the 1964 Helsinki Declaration and its later amendments or comparable \nethical standards. Consent for publication: Not Applicable. Competing interests: \nThe authors declare no competing interests."
        ],
        [
         "46",
         "41204391",
         "1. Cardiooncology. 2025 Nov 7;11(1):103. doi: 10.1186/s40959-025-00399-2.\n\nLongitudinal echocardiographic monitoring and cardiovascular outcomes in adult \nsurvivors of childhood cancer.\n\nGrunblatt E(1), Everitt IK(2), Kinahan K(3), Didwania A(1)(3), Winter JN(3), \nMeng Z(4), Baldridge AS(4), Akhter N(4), Rigolin VH(5).\n\nAuthor information:\n(1)Department of Medicine, Feinberg School of Medicine, Northwestern University, \n676 N. St. Clair, Suite 2330, Chicago, IL, 60611, USA.\n(2)Division of Cardiology, Johns Hopkins University School of Medicine, 1800 \nOrleans St, Baltimore, MD, 21287, USA.\n(3)Robert H. Lurie Comprehensive Cancer Center, Feinberg School of Medicine, \nNorthwestern University, 676 N. St. Clair, Suite 800, Chicago, IL, 60611, USA.\n(4)Department of Medicine, Division of Cardiovascular Medicine, Feinberg School \nof Medicine, Northwestern University, 676 N. St. Clair, Suite 600, Chicago, IL, \n60611, USA.\n(5)Department of Medicine, Division of Cardiovascular Medicine, Feinberg School \nof Medicine, Northwestern University, 676 N. St. Clair, Suite 600, Chicago, IL, \n60611, USA. v-rigolin@northwestern.edu.\n\nBACKGROUND: Adult survivors of childhood cancers are thought to be at high risk \nfor late-onset cardiovascular complications of prior anti-cancer treatments. \nWhile cancer therapy-related cardiac dysfunction (CTRCD) has previously been \nobserved in this population, data are limited regarding the utility of \nlongitudinal echocardiographic monitoring.\nMETHODS: 124 adult survivors of childhood cancer who were previously treated \nwith anthracyclines and/or radiation therapy as children were included in this \nlongitudinal cohort study. Participants were enrolled in Northwestern Medicine's \nsurvivorship clinic: Survivors Taking Action and Responsibility (STAR) Program \nand followed for up to10 years between 2009 and 2022. Serial echocardiography \nwas performed at baseline then at 1, 3, 5, and 7-10 years post-enrollment. \nClinical data was collected by chart review. Major adverse cardiovascular events \n(MACE) and CTRCD were adjudicated according to ACC/AHA MACE criteria and the ESC \nCardio-Oncology guidelines respectively.\nRESULTS: Over 10 years of follow-up in the STAR Program, 17.7% (22/124) of \nparticipants developed MACE, with the highest incidence observed in patients \ntreated with radiation alone. 10.5% (13/124) of patients developed CTRCD by left \nventricular ejection fraction (LVEF) criteria while 36.3% (45/124) developed \nCTRCD by global longitudinal strain (GLS) criteria. New incidence of MACE and \nCTRCD occurred as late as 7-10 years post-enrollment. Participants who developed \nCTRCD showed subsequent improvement in both LVEF and GLS by the end of 10 years \nof follow-up and treatment in the STAR Program.\nCONCLUSIONS: Survivors of childhood cancer are at high risk for both MACE and \nCTRCD even many years after initial diagnosis. Patients who received treatment \nfor pediatric cancer, then subsequently enrolled in the STAR Program as adults \nand received appropriate cardiovascular monitoring and treatment showed \nimprovement in both LVEF and GLS by the end of follow-up, underscoring the \nimportance of a dedicated survivorship clinic for adult survivors of pediatric \ncancer.\n\n© 2025. The Author(s).\n\nDOI: 10.1186/s40959-025-00399-2\nPMID: 41204391\n\nConflict of interest statement: Declarations. Ethics approval and consent to \nparticipate: This study was approved by the Northwestern University \nInstitutional Review Board. Consent for publication: Not applicable. Competing \ninterests: JNW receives research funding from Merck and has honoraria from \nMerck, BMS, and Genentech. NA is on the speakers’ bureau for Zoll. VHR owns \nstock in Merck/Schering-Plough, Intuitive Surgical, Organon & Co., Pfizer, \nViatris, BMS, Amgen, Solventum, Regeneron, Edwards Life Sciences, and \nAstraZeneca. The other authors report no disclosures or conflict of interest."
        ],
        [
         "47",
         "41204384",
         "1. Biomark Res. 2025 Nov 7;13(1):143. doi: 10.1186/s40364-025-00860-5.\n\nArginine methylation in cancer: mechanisms and therapeutic implications.\n\nXu Y(#)(1), Wu Q(#)(2)(3), Zhang Y(#)(4), Gu Y(1), Zhu H(5), Fu X(6)(7), Li \nA(8), Li Y(9).\n\nAuthor information:\n(1)Department of Pathology, Ruijin Hospital, Shanghai Jiaotong University School \nof Medicine, Shanghai, 200025, China.\n(2)Department of Gynecology, Xiangya Hospital, Central South University, \nChangsha, 410008, China.\n(3)National Clinical Research Center for Geriatric Disorders, Xiangya Hospital, \nChangsha, 410008, China.\n(4)Department of Pathology, Zhongshan Hospital, Fudan University, Shanghai, \n200032, China.\n(5)Department of Pathology, Shanghai Pudong New District Zhoupu Hospital, \nShanghai, 200120, China.\n(6)National Clinical Research Center for Geriatric Disorders, Xiangya Hospital, \nChangsha, 410008, China. jessicafu0225@163.com.\n(7)Department of Pathology, Xiangya Hospital, Central South University, \nChangsha, 410008, China. jessicafu0225@163.com.\n(8)Department of Pathology, Ruijin Hospital, Shanghai Jiaotong University School \nof Medicine, Shanghai, 200025, China. annlaq@163.com.\n(9)Department of Pathology, Ruijin Hospital, Shanghai Jiaotong University School \nof Medicine, Shanghai, 200025, China. yimin_li_0107@163.com.\n(#)Contributed equally\n\nArginine methylation is a critical post-translational modification that \nmodulates protein stability, enzymatic activity, and subcellular localization, \nthereby shaping cell fate decisions and maintaining cellular homeostasis. As the \nprincipal enzymes catalyzing this modification, protein arginine \nmethyltransferases (PRMTs) participate in key biological processes, including \ntranscriptional and post-transcriptional regulation as well as signal \ntransduction. Dysregulated PRMT activity has been increasingly linked to tumor \ninitiation, progression, and therapeutic resistance. This review summarizes PRMT \nclassification, structural and functional characteristics, and upstream \nregulatory mechanisms, offering a framework for understanding their diverse \nroles in cancer biology and therapeutic relevance. We further discuss the \nmechanistic contributions of PRMTs to multiple cancer hallmarks and highlight \nrecent advances in the development of PRMT inhibitors. Finally, we examine \ncurrent strategies for clinical translation, with particular emphasis on \ncombination approaches involving chemotherapy, targeted therapy, and \nimmunotherapy, thereby offering a foundation for advancing PRMT-targeted \nprecision oncology.\n\n© 2025. The Author(s).\n\nDOI: 10.1186/s40364-025-00860-5\nPMID: 41204384\n\nConflict of interest statement: Declarations. Ethics approval and consent to \nparticipate: Not applicable. Consent for publication: Not applicable. Competing \ninterests: The authors declare no competing interests."
        ],
        [
         "48",
         "41204349",
         "1. J Transl Med. 2025 Nov 7;23(1):1246. doi: 10.1186/s12967-025-07291-8.\n\nMulti-omics analyses identify mannose phosphate isomerase-centered \nhypoxia-induced angiogenesis signature in colorectal cancer.\n\nLiu S(#)(1), Zhang Y(#)(2)(3), Meng Y(1), Huang Q(1), Feng Z(1), Wen L(1), Yang \nX(2)(3), Zhang Y(1), Qiu L(1), Wang Z(2)(3), Zhang B(4), Chen Z(4), Han J(5)(6).\n\nAuthor information:\n(1)Department of Biotherapy, Cancer Center and State Key Laboratory of \nBiotherapy, and Frontiers Science Center for Disease-Related Molecular Network, \nWest China Hospital, Sichuan University, Chengdu, 610041, China.\n(2)Colorectal Cancer Center, West China Hospital, Sichuan University, Chengdu, \n610041, China.\n(3)Department of General Surgery, West China Hospital, Sichuan University, \nChengdu, 610041, China.\n(4)Laboratory of Gastrointestinal Tumor Epigenetics and Genomics, Department of \nGeneral Surgery, Frontiers Science Center for Disease-Related Molecular Network, \nWest China Hospital, Sichuan University, Chengdu, 610041, China.\n(5)Department of Biotherapy, Cancer Center and State Key Laboratory of \nBiotherapy, and Frontiers Science Center for Disease-Related Molecular Network, \nWest China Hospital, Sichuan University, Chengdu, 610041, China. \nhjunhong@scu.edu.cn.\n(6)Tianfu Jincheng Laboratory, Frontiers Medical Center, Chengdu, 610041, China. \nhjunhong@scu.edu.cn.\n(#)Contributed equally\n\nBACKGROUND: Hypoxia and angiogenesis are crucial hallmarks of cancer that play \nkey roles in the development and progression of colorectal cancer (CRC). \nHowever, the transcriptional mechanism underlying hypoxia induced angiogenesis \n(HIA) remain elusive. This study aimed to explore the regulatory networks, \nmolecular mechanisms, and prognostic value of HIA-related genes.\nMETHODS: We collected multi-omics data, including chromatin immunoprecipitation \nsequencing (ChIP-seq), bulk RNA-seq, single cell RNA-seq, spatial \ntranscriptomics, and microarray data from CRC patients and cell lines. \nComputational methods, including single sample gene set enrichment analysis \n(ssGSEA), signature-related gene analysis (SRGA), consensus clustering, and \nothers, were utilized to explore the correlation between hypoxia and \nangiogenesis, identify the HIA-related genes, and establish the risk scoring \nsystem based on HIA-related genes. The role of mannose phosphate isomerase (MPI) \nwas validated using quantitative real-time PCR (RT-qPCR), Co-immunoprecipitation \n(Co-IP), western blot, colony formation, tube formation assay, and subcutaneous \nxenograft tumor models in vitro and in vivo.\nRESULTS: We identified 12 HIA-related genes that are transcriptionally activated \nby hypoxia-inducible factors (HIFs) and functionally implicated in angiogenesis \nin CRC. Based on the differentially expressed genes among HIA-related CRC \nsubtypes, we constructed a prognostic scoring system termed HIAscore. Patients \nwith high HIAscore was correlated with poor survival, aggressive phenotype, and \nimmunosuppressive tumor microenvironment. Spatial analysis revealed \nsequestration regions between epithelial cells with higher HIAscore and T/I/NK \ncells, hindering their infiltration. Particularly, MPI was found to interact \nwith lactate dehydrogenase A (LDHA), and promote proliferation and angiogenesis \nof CRC through phosphorylation and activation of Janus kinase 2/signal \ntransducers and activators of transcription 3 (JAK2/STAT3) signaling pathways.\nCONCLUSIONS: This study depicts the transcriptional landscape linking hypoxia \nand angiogenesis in CRC, and identifies MPI as a novel regulator of this \nprocess.\n\n© 2025. The Author(s).\n\nDOI: 10.1186/s12967-025-07291-8\nPMID: 41204349 [Indexed for MEDLINE]\n\nConflict of interest statement: Declarations. Ethics approval and consent to \nparticipate: All animal experimental procedures were in accordance with \nprotocols approved by the Experimental Animal Ethics Committee of West China \nhospital, Sichuan University (Ethics record number 20220107010). Consent for \npublication: Not applicable. Competing interests: The authors declare that they \nhave no known competing financial interests or personal relationships that could \nhave appeared to influence the work reported in this paper."
        ],
        [
         "49",
         "41204345",
         "1. J Nanobiotechnology. 2025 Nov 7;23(1):706. doi: 10.1186/s12951-025-03728-w.\n\nCinobufagin loaded Prussian blue-like nanoparticles for chemo/gas therapy of \nmultidrug resistant cancer.\n\nQiu W(#)(1), Fan J(#)(1)(2), Tao W(1), Liang J(3), Xiao C(1), Chen Y(4), Tong \nC(5), Liu B(6).\n\nAuthor information:\n(1)Hunan Research Center of the Basic Discipline for Cell Signaling, College of \nBiology, Hunan University, Changsha, 410082, China.\n(2)Hunan Provincial Key Laboratory of the Research and Development of Novel \nPharmaceutical Preparations, Changsha Medical University, Changsha, 410219, \nChina.\n(3)Qingdao Hospital, University of Health and Rehabilitation Sciences (Qingdao \nMunicipal Hospital), Qingdao, 266011, China.\n(4)School of Medicine, Hunan University of Chinese Medicine, Changsha, 410208, \nP. R. China.\n(5)Hunan Research Center of the Basic Discipline for Cell Signaling, College of \nBiology, Hunan University, Changsha, 410082, China. sw_tcy@hnu.edu.cn.\n(6)Hunan Research Center of the Basic Discipline for Cell Signaling, College of \nBiology, Hunan University, Changsha, 410082, China. binliu2001@hotmail.com.\n(#)Contributed equally\n\nDOI: 10.1186/s12951-025-03728-w\nPMID: 41204345\n\nConflict of interest statement: Declarations. Ethics approval and consent to \nparticipate: All animal procedures were conducted in accordance with the \nDeclaration of Helsinki and approved by the Ethics Committee of Hunan University \n(Approval No. SYXK-2023-0010). Consent for publication: All the authors listed \nhave approved the manuscript that is enclosed. Competing interests: The authors \ndeclare no competing interests."
        ]
       ],
       "shape": {
        "columns": 2,
        "rows": 1203
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>pmid</th>\n",
       "      <th>abstract</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>41204822</td>\n",
       "      <td>1. Cell Cycle. 2025 Nov 8:1-23. doi: 10.1080/1...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>41204804</td>\n",
       "      <td>1. Int J Environ Health Res. 2025 Nov 8:1-12. ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>41204791</td>\n",
       "      <td>1. Cancer Med. 2025 Nov;14(21):e71372. doi: 10...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>41204774</td>\n",
       "      <td>1. IUBMB Life. 2025 Nov;77(11):e70075. doi: 10...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>41204772</td>\n",
       "      <td>1. Am J Med Genet A. 2025 Nov 8:e64305. doi: 1...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1198</th>\n",
       "      <td>41193040</td>\n",
       "      <td>1. J Am Coll Radiol. 2025 Nov;22(11S):S492-S50...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1199</th>\n",
       "      <td>41193032</td>\n",
       "      <td>1. Am J Clin Nutr. 2025 Nov;122(5):1195-1203. ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1200</th>\n",
       "      <td>41193016</td>\n",
       "      <td>1. West J Emerg Med. 2025 Sep 25;26(5):1211-12...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1201</th>\n",
       "      <td>41192978</td>\n",
       "      <td>1. J Gene Med. 2025 Nov;27(11):e70047. doi: 10...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1202</th>\n",
       "      <td>41192973</td>\n",
       "      <td>1. BMJ Support Palliat Care. 2025 Nov 4:spcare...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1203 rows × 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "          pmid                                           abstract\n",
       "0     41204822  1. Cell Cycle. 2025 Nov 8:1-23. doi: 10.1080/1...\n",
       "1     41204804  1. Int J Environ Health Res. 2025 Nov 8:1-12. ...\n",
       "2     41204791  1. Cancer Med. 2025 Nov;14(21):e71372. doi: 10...\n",
       "3     41204774  1. IUBMB Life. 2025 Nov;77(11):e70075. doi: 10...\n",
       "4     41204772  1. Am J Med Genet A. 2025 Nov 8:e64305. doi: 1...\n",
       "...        ...                                                ...\n",
       "1198  41193040  1. J Am Coll Radiol. 2025 Nov;22(11S):S492-S50...\n",
       "1199  41193032  1. Am J Clin Nutr. 2025 Nov;122(5):1195-1203. ...\n",
       "1200  41193016  1. West J Emerg Med. 2025 Sep 25;26(5):1211-12...\n",
       "1201  41192978  1. J Gene Med. 2025 Nov;27(11):e70047. doi: 10...\n",
       "1202  41192973  1. BMJ Support Palliat Care. 2025 Nov 4:spcare...\n",
       "\n",
       "[1203 rows x 2 columns]"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "irrelevant_df"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "5092b356",
   "metadata": {},
   "source": [
    "## **Task 4:**\n",
    "\n",
    "Implement the chosen retrieval system using the programming language of their choice. If the information retrieval system is based on machine learning techniques, the student must split the existing datasets (relevant and non-relevant documents) into three distinct groups (training, validation, and testing) to carry out the model training."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ec80f4b4",
   "metadata": {},
   "source": [
    "**CHOSEN RETRIEVAL SYSTEM:** BioBERT-based Binary Text Classifier"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "ce84273a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Relevant articles shape: (1203, 13)\n",
      "Irrelevant articles shape: (1203, 2)\n"
     ]
    }
   ],
   "source": [
    "# Loading datasets\n",
    "relevant_df = pd.read_csv(ELSEVIER_ABSTRACTS)\n",
    "relevant_df = relevant_df[relevant_df['abstract'].notnull()]\n",
    "irrelevant_df = pd.read_csv(IRRELEVANT_PUBLICATIONS)\n",
    "\n",
    "print(\"Relevant articles shape:\", relevant_df.shape)\n",
    "print(\"Irrelevant articles shape:\", irrelevant_df.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "a8708cb8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Class distribution:\n",
      "relevance\n",
      "1    1203\n",
      "0    1202\n",
      "Name: count, dtype: int64\n"
     ]
    },
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "abstract",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "relevance",
         "rawType": "int64",
         "type": "integer"
        }
       ],
       "ref": "3fb9a5a0-40c5-4175-b025-ab1f4bf7529f",
       "rows": [
        [
         "0",
         "1. J Agric Food Chem. 2007 Jun 27;55(13):5156-66. doi: 10.1021/jf070467u. Epub\n2007  Jun 6.\n\nPolyphenol composition and antioxidant activity in strawberry purees; impact of \nachene level and storage.\n\nAaby K(1), Wrolstad RE, Ekeberg D, Skrede G.\n\nAuthor information:\n(1)Matforsk AS, Norwegian Food Research Institute, Osloveien 1, N-1430 Aas, \nNorway. kjersti.aaby@matforsk.no\n\nIn this study the impact of achenes on polyphenolic compounds, ascorbic acids, \nand antioxidant activities in strawberry purees at production and after storage \nat 6 and 22 degrees C for 8 and 16 weeks was investigated. Strawberry purees \nwere made from flesh, berry, and achene-enriched homogenate and contained 0, \n1.2, and 2.9% achenes, respectively. At production, strawberry purees made from \nflesh contained more anthocyanins, p-coumaroyl glycosides, and ascorbic acids, \nwhereas increasing achene levels caused increasing levels of ellagic acid \nderivatives, proanthocyanidins, flavonols, total phenolics (TP), and antioxidant \nactivities. In addition, the anthocyanins, TP, and ferric reducing ability power \n(FRAP) in purees with more achenes were better retained during storage. Ascorbic \nacids and anthocyanins declined rapidly during storage, whereas other \npolyphenols and antioxidant activities were more stable; that is, the \ncontributions from anthocyanins and ascorbic acids to TP and antioxidant \nactivities decreased. The findings that achenes contributed significantly to \npolyphenol content and stability of strawberry purees may be interesting in a \nnutritional and, thus, commercial, perspective.\n\nDOI: 10.1021/jf070467u\nPMID: 17550269 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "1",
         "1. Free Radic Biol Med. 2002 Dec 15;33(12):1693-702. doi: \n10.1016/s0891-5849(02)01137-1.\n\nUptake and metabolism of epicatechin and its access to the brain after oral \ningestion.\n\nAbd El Mohsen MM(1), Kuhnle G, Rechner AR, Schroeter H, Rose S, Jenner P, \nRice-Evans CA.\n\nAuthor information:\n(1)Antioxidant Research Group, Wolfson Centre for Age-Related Diseases, GKT \nSchool of Biomedical Sciences, King's College, London, England, UK.\n\nEpicatechin is a flavan-3-ol that is commonly present in green teas, red wine, \ncocoa products, and many fruits, such as apples. There is considerable interest \nin the bioavailability of epicatechin after oral ingestion. In vivo studies have \nshown that low levels of epicatechin are absorbed and found in the circulation \nas glucuronides, methylated and sulfated forms. Recent research has demonstrated \nprotective effects of epicatechin and one of its in vivo metabolites, \n3'-O-methyl epicatechin, against neuronal cell death induced by oxidative \nstress. Thus, we are interested in the ability of ingested epicatechin to cross \nthe blood brain barrier and target the brain. Rats were administered 100 mg/kg \nbody weight/d epicatechin orally for 1, 5, and 10 d. Plasma and brain extracts \nwere analyzed by HPLC with photodiode array detection and LC-MS/MS. This study \nreports the presence of the epicatechin glucuronide and 3'-O-methyl epicatechin \nglucuronide formed after oral ingestion in the rat brain tissue.\n\nDOI: 10.1016/s0891-5849(02)01137-1\nPMID: 12488137 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "2",
         "1. J Agric Food Chem. 2003 Apr 9;51(8):2174-80. doi: 10.1021/jf021043x.\n\nComposition and stability of anthocyanins in blue-grained wheat.\n\nAbdel-Aal el-SM(1), Hucl P.\n\nAuthor information:\n(1)Food Research Program, Agriculture and Agri-Food Canada, Guelph, Ontario, \nCanada. abdelaale@agr.gc.ca\n\nWheat grain is recognized as a good source of potentially health-enhancing \ncomponents such as dietary fiber, phenolics, tocopherols, and carotenoids. \nAnthocyanins, another group of bioactive compounds, are found in blue and purple \nwheat grains. In the present study, a blue aleurone spring wheat line \"Purendo \n38\" with relatively high content of total anthocyanins was used to investigate \nthe composition and stability of anthocyanins over three crop years. Commercial \ncultivars of purple (Konini) and red (Katepwa) wheats were included in the \nstudy. Separation of anthocyanins by high-performance liquid chromatography \n(HPLC) showed that each wheat had a distinct anthocyanin profile. Four major \nanthocyanins were separated from blue wheat extracts as compared to five \nanthocyanins in purple wheat. Cyanidin 3-glucoside was the predominant \nanthocyanin in purple wheat, whereas it was the second major anthocyanin in blue \nwheat. The predominant anthocyanin in blue wheat, making up approximately 41% of \nthe total anthocyanin content, remains to be structurally unidentified. Blue \nwheat anthocyanins were thermally most stable at pH 1. Their degradation was \nslightly lower at pH 3 as compared to pH 5. Increasing the temperature from 65 \nto 95 degrees C increased degradation of blue wheat anthocyanins. Addition of \nSO(2) during heating of blue wheat had a stabilizing effect on anthocyanin \npigments. The optimal SO(2) concentrations were 500-1000 ppm for whole meals and \n1000-3000 ppm for isolated anthocyanins. Further studies are underway to \nidentify and verify individual anthocyanins in blue wheat and their potential \nend uses.\n\nDOI: 10.1021/jf021043x\nPMID: 12670152 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "3",
         "1. J Agric Food Chem. 2006 Jun 28;54(13):4696-704. doi: 10.1021/jf0606609.\n\nAnthocyanin composition in black, blue, pink, purple, and red cereal grains.\n\nAbdel-Aal el-SM(1), Young JC, Rabalski I.\n\nAuthor information:\n(1)Food Research Program, Agriculture and Agri-Food Canada, 93 Stone Road West, \nGuelph, Ontario, Canada N1G 5C9. abdelaale@agr.gc.ca\n\nAnthocyanin pigments from a wide variety of edible and ornamental black, blue, \npink, purple, red, and white wheat, barley, corn, rice, and wild rice were \nidentified and quantified to evaluate their potential as natural colorants or \nfunctional food ingredients. The total anthocyanin contents varied significantly \nand exhibited a range of 7-3276 microg/g. Some grains, such as red rice and \nblack rice, contained a limited number of pigments, whereas others, such as \nblue, pink, purple, and red corns, had complex anthocyanin profiles. Of the 42 \nanthocyanin compounds observed, 9 were characterized by comparison of the \nspectroscopic and chromatographic properties with those of authentic standards. \nThe remaining compounds were tentatively identified on the basis of \nspectroscopic properties and electrospray ionization mass spectra. The most \nabundant anthocyanins were cyanidin 3-glucoside in black and red rices and in \nblue, purple, and red corns, pelargonidin 3-glucoside in pink corn, and \ndelphinidin 3-glucoside in blue wheat.\n\nDOI: 10.1021/jf0606609\nPMID: 16787017 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "4",
         "A survey of resveratrol content in wines of the four Aragón designations of origin (Borja, Cariñena, Calatayud and Somontano) was carried out. Concentrations of trans- and cis-resveratrol isomers of 98 commercial wines of the four designations of origin, from several vintages, were analyzed by means of reverse-phase high-performance liquid chromatography. Concentration of trans-resveratrol ranged from 0.32 to 4.44 mg/l in red wines and from 0.12 to 2.80 mg/l in rosé wines. cis-Resveratrol levels ranged from 0.20 to 5.84 mg/l in red wines and from 0.02 to 3.17 mg/l in rosé wines. The grape variety influenced resveratrol contents in wines from the different regions. The highest trans/cis ratios were found in Somontano (4.50) and Calatayud wines (2.90), both of the Tempranillo variety. However, a discriminant analysis applied to the concentrations did not show significant differences between young red wines nor between rosé wines of the four designations of origin. © 2004 Elsevier Ltd. All rights reserved.",
         "1"
        ],
        [
         "5",
         "1. J Agric Food Chem. 1999 Oct;47(10):4184-8. doi: 10.1021/jf990317m.\n\nHPLC method for the quantification of procyanidins in cocoa and chocolate \nsamples and correlation to total antioxidant capacity.\n\nAdamson GE(1), Lazarus SA, Mitchell AE, Prior RL, Cao G, Jacobs PH, Kremers BG, \nHammerstone JF, Rucker RB, Ritter KA, Schmitz HH.\n\nAuthor information:\n(1)Analytical and Applied Sciences Group, Mars, Incorporated, 800 High Street, \nHackettstown, New Jersey 07840, USA.\n\nMonomeric and oligomeric procyanidins present in cocoa liquors and chocolates \nwere separated and quantified in four different laboratories using a \nnormal-phase high-performance liquid chromatography (HPLC) method with \nfluorescence detection. Procyanidin standards through decamers were obtained by \nextraction from cocoa beans, enrichment by Sephadex LH-20 gel permeation \nchromatography, and final purification by preparative normal-phase HPLC. The \npurity of each oligomeric fraction was assessed using HPLC coupled to mass \nspectrometry. A composite standard was then prepared, and calibration curves \nwere generated for each oligomeric class using a quadratic fit of area sum \nversus concentration. Results obtained by each of the laboratories were in close \nagreement, which suggests this method is reliable and reproducible for \nquantification of procyanidins. Furthermore, the procyanidin content of the \nsamples was correlated to the antioxidant capacity measured using the ORAC assay \nas an indicator for potential biological activity.\n\nDOI: 10.1021/jf990317m\nPMID: 10552788 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "6",
         "This study sought to evaluate the effect of steam cooking on the vitamin C, total phenolics, total flavonoid and antioxidant properties (1,1-diphenyl-2-picrylhydrazyl [DPPH] free radical and 2,2-azinobis(3-ethylbenzo-thiazoline-6-sulfonate) [ABTS*] radical scavenging abilities, reducing property and Fe2+ chelating ability) of some tropical green leafy vegetables: Talinum triangulare, Ocimum gratissimum, Amaranthus hybridus, Telfairia occidentalis, Ipomea batata, Cnidoscolus aconitifolius, Baselia alba and Senecio biafrae leaves. The results of the study revealed that cooking causes a significant (P<0.05) decrease in the vitamin C (raw [321.4-842.0], cooked [198.2-638.4mg/100g]) content. Conversely, there was a significant (P<0.05) increase in the total phenol (raw [146.9-693.8], cooked [272.9-1037.5mg/100g]), total flavonoid (raw [8.2-53.0], cooked [12.9-57.4mg/100g]), DPPH radical scavenging ability (raw [15.7-61.8], cooked [52.8-92.7%]), reducing property (raw: 28.3-61.8; cooked: 43.9-71.6mg/100 g ascorbic acid equivalent [AAE]), Fe2+ chelating ability (raw [17.4-75.4], cooked [22.8-89.2%]) and ABTS* scavenging ability (raw [17.4-87.3], cooked [57.5-113.2mmol/100g trolox equivalent antioxidant capacity]). In view of this, it could be concluded that steam cooking decreases the vitamin C content of all the vegetables, while it increased the phenolic content and antioxidant activities. Practical Applications: Consumption of fruits and vegetables has been known to be a practical approach toward the prevention and management of several degenerative diseases. Most green leafy vegetables are not eaten in their unprocessed form but are rather cooked by steaming before consumption in tropical Africa. It is well reported that vitamin C is lost during the cooking of vegetables, but recently, phenolics have been shown to be the main phytochemicals present in foods and that they play crucial roles in the management of several degenerative diseases. Cooking is an indispensable prerequisite in order to obtain safe and high-quality food products. Cooked vegetables have much better hygienic quality, and due to chemical reactions during cooking, they become more digestible and have an increased nutritional value. However, steam cooking may affect the antioxidant status of tropical green leafy vegetables due to the release of more phenolic compounds and destruction or creation of redox-active metabolites. © 2011 Wiley Periodicals, Inc.",
         "1"
        ],
        [
         "7",
         "1. Am J Clin Nutr. 1991 Dec;54(6):1093-100. doi: 10.1093/ajcn/54.6.1093.\n\nUrinary excretion of lignans and isoflavonoid phytoestrogens in Japanese men and \nwomen consuming a traditional Japanese diet.\n\nAdlercreutz H(1), Honjo H, Higashi A, Fotsis T, Hämäläinen E, Hasegawa T, Okada \nH.\n\nAuthor information:\n(1)Department of Clinical Chemistry, University of Helsinki, Meilahti Hospital, \nFinland.\n\nEpidemiologic studies revealed low mortality in hormone-dependent cancer in \nJapanese women and men consuming a traditional diet. We previously found that \ncertain diphenolic food components, lignans and isoflavonoids, which are \nconverted to biologically active hormone-like substances by intestinal \nmicroflora, may be cancer-protective agents. Therefore, we studied urinary \nexcretion of these compounds (enterolactone, enterodiol, daidzein, equol, and \nO-desmethylangolensin) in 10 women and 9 men in a rural village south of Kyoto, \nJapan. The subjects consumed a typical low-fat diet with much rice and soy \nproducts, fish, and vegetables. An isotope-dilution gas chromatographic-mass \nspectrometric method was used for the assays. The urinary excretion of lignans \nwas low but that of the isoflavonoids was very high. The excretion of \nisoflavonoids correlated with soybean-product intake. The low mortality in \nbreast and prostate cancer of Japanese women and men, respectively, may be due \nto the high intake of soybean products.\n\nDOI: 10.1093/ajcn/54.6.1093\nPMID: 1659780 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "8",
         "1. J Agric Food Chem. 2025 Jan 15;73(2):1725-1738. doi: 10.1021/acs.jafc.4c08975.\n Epub 2024 Dec 30.\n\nDynamic Metabolomic Changes in the Phenolic Compound Profile and Antioxidant \nActivity in Developmental Sorghum Grains.\n\nD'almeida CTDS(1)(2), Morel MH(3), Terrier N(4), Mameri H(3), Simões Larraz \nFerreira M(1)(2).\n\nAuthor information:\n(1)Laboratory of Bioactives (LABBIO), Food and Nutrition Graduate Program \n(PPGAN), Federal University of the State of Rio de Janeiro (UNIRIO), Rio de \nJaneiro 22290-240, Brazil.\n(2)Center of Innovation in Mass Spectrometry, Laboratory of Protein \nBiochemistry, UNIRIO, Rio de Janeiro 22290-240, Brazil.\n(3)UMR 1208 IATE, Univ Montpellier, INRAE, L'Institut-Agro Montpellier, \nMontpellier F-34060, France.\n(4)AGAP Institute, Univ. Montpellier, INRAE, CIRAD, Montpellier F-34398, France.\n\nPhenolic compounds (PC) were analyzed by UHPLC-ESI-QTOF-MSE in two sorghum \ngenotypes, harvested in two growing seasons (GS) at five distinct days after \nflowering (DAF) to evaluate how genotype/GS influences the PC synthesis and \nantioxidant capacity during grain growth. Total phenolic contents were strongly \ncorrelated with antioxidant capacity (r > 0.9, p < 0.05). Globally, 97 PC were \nannotated, including 20 PC found irrespective of the grain developmental stage \nand genotype/GS. The phenolic profile clearly differs between stages: phenolic \nacids were the most abundant class in early stages (50%), and flavonoid \naccumulation becomes predominant in late ones (3/5 of total ion abundance). \nDimeric and trimeric tannins were identified even in 10DAF grains. Chemometry \nrevealed great PC variability between genotypes (27%) and important biomarkers \nof GS differentiation (e.g., ferulic acid). This work can input open databases \nof PC and paves the way to understand biosynthetic pathways of PC in sorghum and \nfuture sorghum selection.\n\nDOI: 10.1021/acs.jafc.4c08975\nPMCID: PMC11741113\nPMID: 39811928 [Indexed for MEDLINE]\n\nConflict of interest statement: The authors declare no competing financial \ninterest.",
         "1"
        ],
        [
         "9",
         "1. J Agric Food Chem. 2005 Mar 23;53(6):2297-306. doi: 10.1021/jf048456d.\n\nPhytochemicals and antioxidant activity of milled fractions of different wheat \nvarieties.\n\nAdom KK(1), Sorrells ME, Liu RH.\n\nAuthor information:\n(1)Department of Food Science, Cornell University, Ithaca, New York 14853-7201, \nUSA.\n\nThe health-promoting effects of whole-grain consumption have been attributed in \npart to their unique phytochemical contents and profiles that complement those \nfound in fruits and vegetables. Wheat is an important component of the human \ndiet; however, little is known about the phytochemical profiles and total \nantioxidant activities of milled fractions of different wheat varieties. The \nobjectives of this study were to investigate the distribution of phytochemicals \n(total phenolics, flavonoids, ferulic acid, and carotenoids) and to determine \nhydrophilic and lipophilic antioxidant activity in milled fractions (endosperm \nand bran/germ) of three different wheat varieties, two of which were grown in \ntwo environments. Grain samples of each of the wheat varieties were milled into \nendosperm and bran/germ fractions. Each fraction was extracted and analyzed for \ntotal phenolics, ferulic acid, flavonoids, carotenoid contents, and hydrophilic \nand lipophilic antioxidant activities. Total phenolic content of bran/germ \nfractions (2867-3120 micromol of gallic acid equiv/100 g) was 15-18-fold higher \n(p < 0.01) than that of respective endosperm fractions. Ferulic acid content \nranged from 1005 to 1130 micromol/100 g in bran/germ fractions and from 15 to 21 \nmicromol/100 g in the endosperm fractions. The bran/germ fraction flavonoid \ncontent was 740-940 micromol of catechin equiv/100 g. On average, bran/germ \nfractions of wheat had 4-fold more lutein, 12-fold more zeaxanthin, and 2-fold \nmore beta-cryptoxanthin than the endosperm fractions. Hydrophilic antioxidant \nactivity of bran/germ samples (7.1-16.4 micromol of vitamin C equiv/g) was \n13-27-fold higher than that of the respective endosperm samples. Similarly, \nlipophilic antioxidant activity was 28-89-fold higher in the bran/germ fractions \n(1785-4669 nmol of vitamin E equiv/g). Hydrophilic antioxidant activity \ncontribution to the total antioxidant activity (hydrophilic + lipophilic) was \n>80%. In whole-wheat flour, the bran/germ fraction contributed 83% of the total \nphenolic content, 79% of the total flavonoid content, 51% of the total lutein, \n78% of the total zeaxanthin, 42% of the total beta-cryptoxanthin, 85% of the \ntotal hydrophilic antioxidant activity, and 94% of the total lipophilic \nantioxidant activity. Our results showed that different milled fractions of \nwheat have different profiles of both hydrophilic and lipophilic phytochemicals. \nThese findings provide information necessary for evaluating contributions to \ngood health and disease prevention from whole-wheat consumption.\n\nDOI: 10.1021/jf048456d\nPMID: 15769171 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "10",
         "The relationship between antioxidant capacity and levels of various antioxidants in rice bran and brown rice powder was evaluated. Three different varieties of Venezuelan rice, namely, Cimarrón, Zeta 15 and FONAIAP-1, were studied using ferric reducing antioxidant power (FRAP), 2,2′-azinobis-3-ethylbenzotiazoline-6-sulphonic acid (ABTS), and oxygen radical absorbance capacity (ORAC) to measure antioxidant capacity. The results showed that rice varieties contained different levels and combinations of total polyphenols, γ-oryzanol, α- and γ-tocopherols and α-, γ- and δ-tocotrienols. Compared to brown rice powder, rice bran contained most of the antioxidants and had correspondingly higher values of antioxidant capacity. Principal components analysis and multiple regression on the data indicate that FRAP was sensitive to polyphenols and total tocotrienols, while ORAC was sensitive to polyphenols and total tocopherols. ABTS was the least sensitive of all assays tested. Thus, results from antioxidant capacity assays must be interpreted with caution particularly in complex systems and that further study is necessary to define more precisely the nature of the relevant chemical reactions. © 2006 Elsevier Ltd. All rights reserved.",
         "1"
        ],
        [
         "11",
         "1. J Agric Food Chem. 2010 Sep 22;58(18):10101-8. doi: 10.1021/jf102222t.\n\nEvaluation of phenolic profile and antioxidant properties of Pardina lentil as \naffected by industrial dehydration.\n\nAguilera Y(1), Dueñas M, Estrella I, Hernández T, Benitez V, Esteban RM, \nMartín-Cabrejas MA.\n\nAuthor information:\n(1)Instituto de Ciencias de la Alimentación (CIAL), Departamento de Química \nAgrícola, Facultad de Ciencias, C/Nicolás Cabrera 9, Campus Universidad Autónoma \nde Madrid (UAM), 28049-Madrid, Spain.\n\nThis study presents the effects of soaking, cooking, and industrial dehydration \ntreatments on phenolic profile and also on antioxidant properties in Pardina \nlentil. HPLC-PAC and HPLC-MS (ESI) analysis identified a total of 35 phenolic \ncompounds in raw and processed lentil flours, corresponding to catechins and \nprocyanidins (69% of the total of identified phenolic compounds), flavonols \n(17%), flavones, and flavanones (5%), and hydroxybenzoic and hydroxycinnamic \ncompounds (5 and 4%, respectively). During the industrial process, catechins and \nprocyanidins, flavonols, flavones, and flavanones decreased, while \nhydroxybenzoic compounds exhibited an important increase. In addition, raw \nlentils showed high values of the antioxidant activity (66.97 μmol Trolox/g); \nalthough the thermal processing promotes decreased, the levels of antioxidant \nactivity were still relevant. Thus, the significant occurrence of bioactive \nphenolic compounds along with the interesting antioxidant capacity of dehydrated \nlentil flours make them useful for daily inclusion in the human diet as \nready-to-use for special meals to specific populations.\n\nDOI: 10.1021/jf102222t\nPMID: 20735138 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "12",
         "A reverse-phase high-performance liquid chromatographic technique with isocratic elution has been developed to separate and quantitate the major phenolic compounds of the hydroalcoholic extracts of olive oils. Hydroxytyrosol, tyrosol, caffeic acid, p-hydroxyphenylacetic acid and homovanillic acid were analyzed on a μBonapak C18 column with an acetonitrile/water/acetic acid (20:90:0.18, vol/vol/vol) mixture as a mobile phase. Electrochemical detection provided selectivity as well as sensitivity. The method was applied to the analysis of the most important phenolic compounds in olive oils. © 1993 AOCS Press.",
         "1"
        ],
        [
         "13",
         "The concentrations of isoflavones, especially daidzein and genistein, were determined by high performance liquid chromatography in 4 soybean cultivars and 26 soybean products. The total isoflavone content of the soybean cultivars was in the range of 525-986 mg per kg, and for soy products it was 32.9-795 mg per kg. Amongst the soybean products, the isoflavone content decreased in the order: soy sprouts, soy seeds, soy flour, soy milk, soy meals and soy sauce. Significant differences in the concentration of genistein and daidzein were observed between the commercial soy products and also within the soybean cultivars. The antioxidant activity of soybean and soy products correlated well with total phenolic content (TPC) and total isoflavones (TI), whereas TPC showed higher correlation with TI. © 2008 Elsevier Ltd. All rights reserved.",
         "1"
        ],
        [
         "14",
         "1. J Agric Food Chem. 2005 Sep 21;53(19):7592-9. doi: 10.1021/jf050579q.\n\nComparison of antioxidant activity, anthocyanins, carotenoids, and phenolics of \nthree native fresh and sun-dried date (Phoenix dactylifera L.) varieties grown \nin Oman.\n\nAl-Farsi M(1), Alasalvar C, Morris A, Baron M, Shahidi F.\n\nAuthor information:\n(1)Plant Research Center, Ministry of Agriculture and Fisheries, Al-Khoud, \nMuscat, Oman.\n\nFresh and sun-dried dates of three native varieties from Oman, namely, Fard, \nKhasab, and Khalas, were examined for their antioxidant activity and total \ncontents of anthocyanins, carotenoids, and phenolics, as well as free and bound \nphenolic acids. All results are expressed as mean value +/- standard deviation \n(n = 3) on a fresh weight basis. Fresh date varieties were found to be a good \nsource of antioxidants (11687-20604 micromol of Trolox equiv/g), total contents \nof anthocyanins (0.24-1.52 mg of cyanidin 3-glucoside equiv/100 g), carotenoids \n(1.31-3.03 mg/100 g), phenolics (134-280 mg of ferulic acid equiv/100 g), free \nphenolic acids (2.61-12.27 mg/100 g), and bound phenolic acids (6.84-30.25 \nmg/100 g). A significant (p < 0.05) amount of antioxidants and carotenoids was \nlost after sun-drying of dates, whereas the total content of phenolics and free \nand bound phenolic acids increased significantly (p < 0.05). Anthocyanins were \ndetected only in fresh dates. Date varieties had different levels and patterns \nof phenolic acids. Four free phenolic acids (protocatechuic acid, vanillic acid, \nsyringic acid, and ferulic acid) and nine bound phenolic acids (gallic acid, \nprotocatechuic acid, p-hydroxybenzoic acid, vanillic acid, caffeic acid, \nsyringic acid, p-coumaric acid, ferulic acid, and o-coumaric acid) were \ntentatively identified. Of the date varieties studied, Khalas, which is \nconsidered to be premium quality, had higher antioxidant activity, total \ncarotenoids, and bound phenolic acids than other varieties. These results \nsuggest that all date varieties serve as a good source of natural antioxidants \nand could potentially be considered as a functional food or functional food \ningredient, although some of their antioxidant constituents are lost during \nsun-drying.\n\nDOI: 10.1021/jf050579q\nPMID: 16159191 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "15",
         "Three native sun-dried date varieties from Oman (namely Mabseeli, Um-sellah, and Shahal), as well as their syrups and by-products (press cake and seed) were examined for their proximate composition, dietary fibre, total phenolics, and total antioxidant activity. Carbohydrate was the predominant component in all date varieties, syrups, and their by-products, followed by moisture, along with small amounts of protein, fat, and ash. Press cakes had the highest protein content, ranging in concentration from 3.62 g/100 g in Shahal to 5.23 g/100 g in Mabseeli, whereas fat was the highest in seeds and ranged from 5.02 g/100 g in Mabseeli to 5.90 g/100 g in Um-sellah. Seeds and press cakes were found to be good sources of dietary fibre, varied between 77.75 and 80.15 g/100 g fresh weight and between 25.39 and 33.81 g/100 g fresh weight, respectively. Among dates, syrups, and their by-products, seeds had the highest contents of total phenolics (3102-4430 mg of gallic acid equivalents/100 g fresh weight) and antioxidant activity (580-929 μmol of Trolox equivalents/g fresh weight). The results obtained suggest that date by-products (particularly seeds) serve as a good source of natural antioxidants and could potentially be considered as a functional food or functional food ingredient. © 2007 Elsevier Ltd. All rights reserved.",
         "1"
        ],
        [
         "16",
         "The effects of microwave cooking and other traditional cooking methods such as boiling and autoclaving on the nutritional composition and anti-nutritional factors of chickpeas (Cicer arietinum L.) were studied. Cooking treatments caused significant (P < 0.05) decreases in fat, total ash, carbohydrate fractions (reducing sugars, sucrose, raffinose and stachyose, while verbascose was completely eliminated after cooking treatments), antinutritional factors (trypsin inhibitor, haemagglutinin activity, tannins, saponins and phytic acid), minerals and B-vitamins. Cooking treatments decreased the concentrations of lysine, tryptophan, total aromatic and sulfur-containing amino acids. However, cooked chickpeas were still higher in lysine, isoleucine and total aromatic amino acid contents than the FAO/WHO reference. The losses in B-vitamins and minerals in chickpeas cooked by microwaving were smaller than those cooked by boiling and autoclaving. In-vitro protein digestibility, protein efficiency ratio and essential amino acid index were improved by all cooking treatments. The chemical score and limiting amino acid of chickpeas subjected to the various cooking treatments varied considerably, depending on the type of treatment. Based on these results, microwave cooking is recommended for chickpea preparation, not only for improving nutritional quality (by reducing the level of antinutritional and flatulence factors as well as increasing in-vitro protein digestibility and retention rates of both B-vitamins and minerals), but also for reducing cooking time. © 2006 Elsevier Inc. All rights reserved.",
         "1"
        ],
        [
         "17",
         "Nocino is an after-dinner liqueur made with green, unripe walnuts, appreciated both for its appealing flavour and aroma, and for its properties as a tonic and digestive aid. There is a large number of recipes for the production of nocino and the industrial production coexists with preparations at home. The liqueur has never been scientifically investigated; thus the aim of this work was to examine various kinds of it. Three sets of samples were considered: Commercial brands, home-made nocino and, among the latter, samples made for years with the same recipe and aged for different times, up to 25 years. In the commercial products, the phenolic profiles and the antioxidant activities were widely variable. The antioxidant activity proved to be directly correlated with the content of total phenols, total tannins and non-tannin phenolics. Study of samples aged for different lengths of time showed that the characteristics considered remained substantially unchanged during the aging process. © 2004 Elsevier Ltd. All rights reserved.",
         "1"
        ],
        [
         "18",
         "1. J Agric Food Chem. 2001 Mar;49(3):1410-6. doi: 10.1021/jf000595h.\n\nComparison of volatiles, phenolics, sugars, antioxidant vitamins, and sensory \nquality of different colored carrot varieties.\n\nAlasalvar C(1), Grigor JM, Zhang D, Quantick PC, Shahidi F.\n\nAuthor information:\n(1)Food Research Center, University of Lincolnshire and Humberside, Brayford \nPool, Lincoln LN6 7TS, United Kingdom. calasalvar@lincoln.ac.uk\n\nFour different colored carrots, orange, purple with orange core, yellow, and \nwhite, were examined for their content of phenolics, antioxidant vitamins, and \nsugars as well as their volatiles and sensory responses. A total of 35 volatiles \nwere identified in all carrots, 27 positively. White carrot contained the \nhighest content of volatiles, followed by orange, purple, and yellow. In total, \n11, 16, 10, and 9 phenolic compounds were determined for the first time in \norange, purple, yellow, and white carrots, respectively. Of these, chlorogenic \nacid was the most predominant phenolic compound in all carrot varieties. \nDifferences (p < 0.05) in relative sweetness, the contents of vitamin C and \nalpha- and beta-carotenes, and certain flavor characteristics were observed \namong the colored carrot varieties examined. Purple carrots contained 2.2 and \n2.3 times more alpha- and beta-carotenes (trace in yellow; not detected in \nwhite) than orange carrots, respectively. Purple carrot may be used in place of \nother carrot varieties to take advantage of its nutraceutical components.\n\nDOI: 10.1021/jf000595h\nPMID: 11312873 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "19",
         "1. J Agric Food Chem. 2010 Aug 11;58(15):8674-9. doi: 10.1021/jf101039f.\n\nEffects of roasting on taste-active compounds of Turkish hazelnut varieties ( \nCorylus avellana L.).\n\nAlasalvar C(1), Pelvan E, Amarowicz R.\n\nAuthor information:\n(1)TUBITAK Marmara Research Center, Food Institute, P.O. Box 21, 41470 \nGebze-Kocaeli, Turkey. cesarettin.alasalvar@mam.gov.tr\n\nThe effect of roasting on taste-active components of 18 native hazelnut \nvarieties, grown in the Giresun province of Turkey, was assessed. Samples were \nexamined for their sugars, organic acids, condensed tannins, and free phenolic \nacids. Six sugars (fructose, glucose, sucrose, myo-inositol, raffinose, and \nstachyose), seven organic acids (oxalic, maleic, citric, malic, lactic, \nsuccinic, and acetic), and one phenolic acid (gallic acid) were positively \nidentified in natural and roasted hazelnut varieties; among these, sucrose, \nmalic acid, and gallic acid predominated, respectively. Total sugars among \nhazelnut varieties ranged from 1.99 to 4.94 g/100 g, organic acids from 0.96 to \n2.72 g/100 g, condensed tannins from 3.99 to 40.56 mg of catechin equivalents/g, \nand gallic acid from 0.159 to 0.871 mg/100 g. Differences existed in the sugar \nand organic acid contents between natural and roasted hazelnut varieties, but \nthey did not follow any particular trend. Significant losses (p < 0.05) in \ncondensed tannins ( approximately 97.3%) and gallic acid ( approximately 66.7%) \nwere noted when the hazelnuts were roasted. The present work suggests that \nroasting resulted in significant loss in condensed tannins and gallic acid due \nto the removal of the brown skin. The effect of roasting on sugars and organic \nacids was not noteworthy.\n\nDOI: 10.1021/jf101039f\nPMID: 20681657 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "20",
         "Anthocyanins (ACs) are phenolic compounds that are distributed widely in fruits and vegetables. Apart from imparting color to plants, ACs also have an array of health-promoting benefits. In this research, the amounts of major ACs of 15 pomegranate (Punica granatum L.) varieties obtained from Yazd province were determined. The major ACs detected in the studied varieties were as follows: delphinidin 3-glucoside (2.19-16.29 mg/L), delphinidin 3,5-diglucoside (2.36-63.07 mg/L), pelargonidin 3-glucoside (0.26-1.36 mg/L), pelargonidin 3,5-diglucoside (0.01-8.11 mg/L), cyanidin 3-glucoside (5.78-30.38 mg/L), and cyanidin 3,5-diglucoside (4.39-166.32 mg/L). The effect of storage time of unprocessed and pasteurized juices on ACs content of four selected varieties was also studied. Average degradation percentage of each AC was between 23.0 and 83.0% during 10 days at 4 °C. Moreover, in pasteurized juices average degradation of ACs was 42.8 ± 0.5% after 10 weeks storage at 4 °C. © 2007 Springer-Verlag.",
         "1"
        ],
        [
         "21",
         "Comparative effects of extrusion cooking and conventional processing methods on protein content and reduction of antinutritional factor (phytic acid, condensed tannins, polyphenols, trypsin, chymotrypsin, α-amylase inhibitors and haemagglutinating activity) levels in Vicia faba and Phaseolus vulgaris seeds were studied. In vitro protein and starch digestibilities were assessed. P. vulgaris seeds showed highest levels of condensed tannins, chymotrypsin and α-amylase inhibitory activities and haemagglutinating activity. Dehulling significantly increased protein content and greatly reduced condensed tannin and polyphenol levels in both legumes. Extrusion was the best method to abolish trypsin, chymotrypsin, α-amylase inhibitors and haemagglutinating activity without modifying protein content. Furthermore, this thermal treatment was most effective in improving protein and starch digestibilities when compared with dehulling, soaking and germination. Copyright (C) 1999.",
         "1"
        ],
        [
         "22",
         "The protective effect on health of wine consumed in moderation has been widely studied in recent years. This has been attributed to the content in polyphenols and to the antioxidant activity of these compounds. The products derived from wine, such as vinegar and brandy, also contain polyphenols and could possess a certain antioxidant activity, similarly contributing to a protective effect for health. This paper describes the study of the antioxidant power of diverse Sherry brandies and vinegars, by means of an electrochemical method previously devised. The total polyphenolic content is studied using the method of Folin-Ciocalteu, and some polyphenols are identified and quantified by means of HPLC. A close correlation between the antioxidant power and the total polyphenolic content of the samples has been found. Considering the polyphenols individually, the compounds that are better correlated with the antioxidant power of the samples are not necessarily those that are present in higher concentrations. For brandies, the contact with wood has an important contribution to their antioxidant power. © 2004 Elsevier Ltd. All rights reserved.",
         "1"
        ],
        [
         "23",
         "1. J Chromatogr A. 2004 Oct 29;1054(1-2):175-80. doi:\n10.1016/j.chroma.2004.07.092.\n\nDevelopment of a rapid method based on solid-phase extraction and liquid \nchromatography with ultraviolet absorbance detection for the determination of \npolyphenols in alcohol-free beers.\n\nGarcía AA(1), Grande BC, Gándara JS.\n\nAuthor information:\n(1)Nutrition and Bromatology Group, Analytical and Food Chemistry Department, \nFaculty of Food Science and Technology, University of Vigo, Ourense Campus, \n32004 Ourense, Spain.\n\nAn analytical method based on solid-phase extraction (SPE) and followed by \nliquid chromatographic separation and ultraviolet detection (HPLC-UV) is \nproposed for the determination of 10 phenolic compounds which participate on \nbeer stability and sensory properties in alcohol-free beers. Acetonitrile was \nfound to be the most appropriate solvent for the elution of polyphenolic \ncompounds adsorbed on C18 cartridges. The performance of the method was assessed \nby the evaluation of parameters such as absolute recovery (generally higher than \n60%), repeatability (lower than 10%), linearity (r2 higher than 0.993) and \nlimits of quantitation (ranging from 1 to 37 microg/L); no matrix effects were \nobserved. The polyphenol content of different Spanish alcohol-free beers is \npresented. Five phenolic compounds such as protocatechuic, p-coumaric, ferulic, \ncaffeic acids, and (+)-catechin were identified at levels lower than 10 mg/L.\n\nDOI: 10.1016/j.chroma.2004.07.092\nPMID: 15553142 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "24",
         "1. J Agric Food Chem. 2001 Aug;49(8):3761-7. doi: 10.1021/jf010021s.\n\nDetermination of polyphenolic profiles of Basque cider apple varieties using \naccelerated solvent extraction.\n\nAlonso-Salces RM(1), Korta E, Barranco A, Berrueta LA, Gallo B, Vicente F.\n\nAuthor information:\n(1)Department of Analytical Chemistry, Faculty of Sciences, Universidad del País \nVasco/ Euskal Herriko Unibertsitatea, P.O. Box 644, E-48080 Bilbao, Spain.\n\nPolyphenols in the peel and pulp of 15 Basque cider apple varieties were \ndetermined by accelerated solvent extraction followed by reversed phase \nhigh-performance liquid chromatography with diode array detection. It was \nobserved that the polyphenolic composition in apple peel depended on varieties, \nwhereas the main classes of phenolic compounds in apple pulp were flavan-3-ols \nand hydroxycinnamic acids in all cases, representing both together between 86 \nand 95% of total polyphenols assayed.\n\nDOI: 10.1021/jf010021s\nPMID: 11513662 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "25",
         "1. J Agric Food Chem. 2004 May 19;52(10):2938-52. doi: 10.1021/jf035416l.\n\nPolyphenolic profiles of Basque cider apple cultivars and their technological \nproperties.\n\nAlonso-Salces RM(1), Barranco A, Abad B, Berrueta LA, Gallo B, Vicente F.\n\nAuthor information:\n(1)Departamento de Química Analítica, Facultad de Ciencias, Universidad del País \nVasco/Euskal Herriko Unibertsitatea, P.O. Box 644, E-48080 Bilbao, Spain.\n\nThe polyphenolic compositions of 31 Basque cider apple cultivars were determined \nin pulp, peel, and juice by high-performance liquid chromatography with diode \narray detection analysis of crude extracts and after thiolysis. Total \npolyphenols are distributed in a wide concentration range depending on the \ncultivar. Procyanidins are the class of polyphenols that present major \nconcentrations in apple. Their average degrees of polymerization range from 4 to \n8 depending on the cultivar. Apple cultivars were technologically classified \ninto bitter and nonbitter categories using different classification systems \nobtained by applying several pattern recognition techniques, such as principal \ncomponent analysis, K-nearest neighbors, soft independent modeling of class \nanalogy, partial least-squares, and multilayer feed-forward-artificial neural \nnetworks, to apple pulp, peel, or juice data (individual polyphenol \nconcentrations, total procyanidin content, and the average degree of \npolymerization of procyanidins). Bitter apple cultivars present higher contents \nof flavan-3-ols and/or dihydrochalcones than nonbitter cultivars. Detailed \nknowledge of the polyphenolic profile of each apple cultivar affords information \nabout their susceptibility to oxidation, their sensory properties (bitterness, \nastringency), and their possible influence on the characteristics and quality of \nthe final product (juice, cider) when apples are processed.\n\nDOI: 10.1021/jf035416l\nPMID: 15137839 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "26",
         "Polyphenolic profiles of cider apple cultivars were studied in order to differentiate fruits according to their maturity state (ripe or unripe). Thiolysis and direct solvent extracts of freeze-dried apple pulps and peels were analysed by HPLC-DAD. Univariate data treatment did not achieve the mentioned target; thus a multivariate approach was considered. For each apple tissue data set, several chemometric techniques were applied to the most discriminant variables. Cluster analysis allowed a preliminary study of the data structure. Then, supervised pattern recognition procedures, namely linear discriminant analysis, K-nearest neighbours, soft independent modelling of class analogy, and multilayer feed-forward artificial neural networks (MLF-ANN), were used to develop decision rules to classify samples in the established categories. Excellent results were afforded by MLF-ANN applied to the concentrations of total procyanidins and (+)-catechin and the average degree of polymerisation of procyanidins in apple pulp, with success in the prediction ability of 97% and 99% for unripe and ripe categories, respectively. © 2004 Elsevier Ltd. All rights reserved.",
         "1"
        ],
        [
         "27",
         "In this study, the influences of processing and pasteurization on color values and total phenolic compounds of pomegranate juices were investigated. Pomegranate juices were produced by different clarification techniques (conventional fining, conventional fining together with polyvinylpolypyrrolidone [PVPP], ultrafiltration). Nonclarified juice was also produced as control. Clarification methods, heat treatments and methods and heat treatments together significantly (P < 0.05) affected the color values of pomegranate juices. Moreover, conventional fining together with PVPP treatment was found to be the most effective method to remove phenolic compounds. © Copyright 2005, Blackwell Publishing.",
         "1"
        ],
        [
         "28",
         "1. J Agric Food Chem. 2000 Dec;48(12):6292-7. doi: 10.1021/jf000849z.\n\nInfluence of jam processing on the radical scavenging activity and phenolic \ncontent in berries.\n\nAmakura Y(1), Umino Y, Tsuji S, Tonogai Y.\n\nAuthor information:\n(1)National Institute of Health Sciences, Osaka Branch, 1-1-43, Hoenzaka, \nChuo-ku, Osaka 540-0006, Japan. amkura@nihs.gov.jp\n\nSix selected phenolic aglycons (caffeic and ellagic acids, kaempferol, \nquercetin, myricetin, and morin) in nine types of berries, and their changes as \ninfluenced by jam processing, have been evaluated using optimized HPLC with \ndiode-array detection. The berry samples, fresh and after jam processing, were \nanalyzed, and the total amounts of selected phenolics as aglycons were \nidentified and determined by acid hydrolysis. Their contents in fresh and jam \nsamples did not indicate appreciable changes; therefore, the influence of jam \nprocessing on these selected phenolics in berries was suggested to be small, and \nwas mostly present in berries as several conjugated forms that were \nglycosylated, esterified, etc., in the samples. The total phenolic content of \neach sample was also determined by the Folin-Ciocalteu method. The three samples \nof each berry, namely fresh, jam, and acid hydrolysate of the berry, had similar \ntotal phenolic contents. On the other hand, the scavenging effect on the \n1,1-diphenyl-2-picrylhydrazyl (DPPH) radical was measured, and acid hydrolysates \nshowed stronger activity than that of the fresh and jam-processed samples for \nall of the berry types.\n\nDOI: 10.1021/jf000849z\nPMID: 11312801 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "29",
         "Extracts of polyphenolic compounds were obtained from seeds of faba bean, broad bean, adzuki bean, red bean, pea, red lentil and green lentil using 80% (v/v) acetone. The total antioxidant activity (TAA) of extracts was determined and their astringency (Sensation Astringency Indices - SAI) investigated using the sensory scaling method. The content of tannins in extracts was expressed as absorbance values at 500 and 550 nm after color developed with vanillin/HCl reagent and after n-butanol-HCl hydrolysis, respectively. A statistically significant correlation was found between the TAA values and total phenolics (P = 0.01), TAA and tannins determined by vanillin method (P = 0.05), TAA and tannins determined after n-butanol-HCl hydrolysis (P = 0.05), SAI values and tannins determined with vanillin method (P = 0.10), SAI values and tannins determined after n-butanol-HCl hydrolysis (P = 0.05), SAI values and total phenolics (P = 0.01), total phenolics and tannin content determined by vanillin method (P = 0.01), total phenolics and tannin content determined after n-butanol-HCl hydrolysis (P = 0.05), tannins determined with vanillin method and tannins determined after n-butanol hydrolysis (P = 0.02). © Copyright 2004, Blackwell Publishing.",
         "1"
        ],
        [
         "30",
         "1. Clin Pharmacol Ther. 1996 Jul;60(1):34-40. doi: 10.1016/S0009-9236(96)90164-2.\n\nFlavanone absorption after naringin, hesperidin, and citrus administration.\n\nAmeer B(1), Weintraub RA, Johnson JV, Yost RA, Rouseff RL.\n\nAuthor information:\n(1)Food Science and Human Nutrition Department, University of Florida, \nGainesville.\n\nDisposition of citrus flavonoids was evaluated after single oral doses of pure \ncompounds (500 mg naringin and 500 mg hesperidin) and after multiple doses of \ncombined grapefruit juice and orange juice and of once-daily grapefruit. \nCumulative urinary recovery indicated low bioavailability ( < 25%) of naringin \nand hesperidin. The aglycones naringenin and hesperitin were detected in urine \nand plasma by positive chemical ionization-collisionally activated dissociation \ntandem mass spectrometry (PCI-CAD MS/MS). After juice administration, PCI-CAD \nMS/MS detected naringenin, hesperitin, and four related flavanones, tentatively \nidentified as monomethoxy and dimethoxy derivatives. These methoxyflavanones \nappear to be absorbed from juice. Absorbed citrus flavanones may undergo \nglucuronidation before urinary excretion.\n\nDOI: 10.1016/S0009-9236(96)90164-2\nPMID: 8689809 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "31",
         "This study aimed to evaluate the effect of different blanching times on the antioxidant properties (antioxidant and free radical scavenging activities) and phenolic content of selected cruciferous vegetables. The study revealed that a 10-min blanching time had a significant effect (p < 0.05) on the antioxidant properties and phenolic content of all the vegetables except for cabbage and mustard cabbage. The loss of antioxidant activity was highest in Chinese cabbage (40%) after 15 min of blanching, followed by cabbage (27%), Chinese white cabbage (19%), mustard cabbage (9%) and red cabbage (4%). Red cabbage had lost a total of 40% scavenging activity after 15 min of blanching followed by Chinese cabbage (38%), cabbage (36%), mustard cabbage (23%) and Chinese white cabbage (11%). Only Chinese cabbage showed an increase (p < 0.05) in total phenolic content after 15 min of blanching compared with other vegetables. Minimal heat treatment through blanching process is recommended to prevent the major loss of antioxidant properties and phenolic content in selected cruciferous vegetables. © 2005 Society of Chemical Industry.",
         "1"
        ],
        [
         "32",
         "Phenolic compounds and polyphenol oxidase (PPO) activity in the fruits of 11 apple cultivars before and after bruising were studied in relation to their susceptibility to browning. The degree of browning has been determined by the simultaneous measurements of soluble (absorbance at 400nm) and insoluble (lightness) brown pigments. The sum of these two expressing the extent of browning, closely correlated with the amount of phenolics (hydroxycinnamic derivatives and flavan‐3‐ols) degraded. Maturity did not appear to greatly influence the development of browning. Copyright © 1992, Wiley Blackwell. All rights reserved",
         "1"
        ],
        [
         "33",
         "The phenolic composition and the degree of browning of pears were determined for nine cultivars at different stages of maturity. 5′-Caffeoylquinic acid and (-)-epicatechin were the two major phenolics found. Phenolic content and susceptibility of pears to browning were high in peel. Differences in phenolic composition and in the degree of browning were more influenced by the cultivar than by the maturity stage of the fruit. At harvest, the degree of browning was closely correlated with the initial amount of hydroxycinnamic esters and flavanols. After 4 days of storage at room temperature, the variation in enzymatic browning depended on the cultivar and the phenolic content decreased for all cultivars. Williams cultivar pears stored in air accumulated phenolics more than fruits stored under 1% CO2/1% O2 and 3% CO2/3% O2. © 1995, American Chemical Society. All rights reserved.",
         "1"
        ],
        [
         "34",
         "1. J Agric Food Chem. 2000 Oct;48(10):4565-70. doi: 10.1021/jf0001684.\n\nEllagic acid, vitamin C, and total phenolic contents and radical scavenging \ncapacity affected by freezing and frozen storage in raspberry fruit.\n\nde Ancos B(1), González EM, Cano MP.\n\nAuthor information:\n(1)Department of Plant Foods Science and Technology, Instituto del Frío, CSIC, \nCuidad Universitaria s/n, 28040 Madrid, Spain.\n\nThe ellagic acid, total phenolic, and vitamin C contents in four raspberry \ncultivars (Heritage, Autumn Bliss, Rubi, and Zeva) grown in Spain were detected \nand quantified by HPLC in fresh, just frozen, and stored fruits at -20 degrees C \nfor a one year period. Ellagic acid [207-244 mg kg(-)(1) of fresh weight (fw)], \ntotal phenolic (137-1776 mg kg(-)(1) of fw), and vitamin C (221-312 mg kg(-)(1) \nof fw) contents in raw material were higher in the late cultivars Zeva and Rubi \nthan in the early cultivars Autumn Bliss and Heritage. The freezing process \nslightly affected the values of extracted ellagic acid, total phenolic, and \nvitamin C content. At the end of long-term frozen storage (12 months), no \nsignificant change of total phenolic content extracted was observed, but \nsignificant decreases of 14-21% in ellagic acid and of 33-55% in vitamin C were \nquantified. Free radical scavenging capacity measured as antiradical efficiency \n(AE) depends on the seasonal period of harvest. Late cultivars, Rubi (6.1 x \n10(-)(4)) and Zeva (10.17 x 10(-)(4)), showed higher AE than early cultivars, \nHeritage (4.02 x 10(-)(4)) and Autumn Bliss (4.36 x 10(-)(4)). The freezing \nprocess produced a decrease of AE values in the four cultivars ranging between 4 \nand 26%. During the frozen storage, the AE values reached after the freezing \nprocess remained unchanged.\n\nDOI: 10.1021/jf0001684\nPMID: 11052701 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "35",
         "1. J Agric Food Chem. 2000 Mar;48(3):873-9. doi: 10.1021/jf990747c.\n\nFrozen storage effects on anthocyanins and volatile compounds of raspberry \nfruit.\n\nde Ancos B(1), Ibañez E, Reglero G, Cano MP.\n\nAuthor information:\n(1)Department of Plant Foods Science and Technology, Instituto del Frío, CSIC, \nCiudad Universitaria s/n, 28040 Madrid, Spain.\n\nThe quantitative and qualitative evolution of the anthocyanins and volatile \ncompounds of four raspberry cultivars (cvs. Heritage, Autumn Bliss, Zeva, and \nRubi) growing in Spain were analyzed raw, just frozen, and during long-term \nfrozen storage at -20 degrees C for a 1 year period. HS-SPME coupled with GC-MS \nand HPLC techniques were employed to study the evolution of the volatile \ncompounds and the individual anthocyanins, respectively. The volatile aroma \ncomposition changes produced by the freezing process and long-term frozen \nstorage were minimal. Only a significant increase in extraction capacity was \nobtained for alpha-ionone (27%) and for caryophyllene (67%) in Heritage at 12 \nmonths of storage. The stability of anthocyanins to freezing and frozen storage \ndepends on the seasonal period of harvest. Heritage and Autumn Bliss (early \ncultivars) were less affected by processing and long-term frozen storage (1 \nyear), and the total pigment extracted showed the tendency to increase 17 and \n5%, respectively. Rubi and Zeva (late cultivars) suffered a decreased trend on \nthe total anthocyanin content of 4% for Rubi and 17.5% for Zeva. Cyanidin \n3-glucoside most easily suffered the degradative reactions that take place \nduring processing and the storage period.\n\nDOI: 10.1021/jf990747c\nPMID: 10725166 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "36",
         "1. J Agric Food Chem. 2000 Aug;48(8):3106-11. doi: 10.1021/jf000354+.\n\nPotential antioxidants in beer assessed by ESR spin trapping.\n\nAndersen ML(1), Outtrup H, Skibsted LH.\n\nAuthor information:\n(1)Food Chemistry, Department of Dairy and Food Science, Royal Veterinary and \nAgricultural University, Frederiksberg C, Denmark. mola@kvl.dk\n\nA number of potential antioxidants have been evaluated for their effect on \nformation of radicals in beer using the electron spin resonance (ESR) lag phase \nmethod. Sulfite was found to be the only compound that was able to delay the \nformation of radicals, whereas phenolic compounds such as phenolic acids, \ncatechin, epicatechin, and proanthocyanidin dimers had no effect on the \nformation of radicals. Ascorbate, cystein, and cysteamin were on the other hand \nfound to be prooxidants. It is suggested that antioxidants must be able to \neither scavenge peroxides or trap metal ions in order to be effective in beer. \nThe effectiveness of sulfite is suggested to be a consequence of its \ntwo-electron nonradical producing reaction with peroxides.\n\nDOI: 10.1021/jf000354+\nPMID: 10956077 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "37",
         "1. Int J Vitam Nutr Res. 2003 Mar;73(2):152-9. doi: 10.1024/0300-9831.73.2.152.\n\nInfluence of cooking process on phenolic marker compounds of vegetables.\n\nAndlauer W(1), Stumpf C, Hubert M, Rings A, Fürst P.\n\nAuthor information:\n(1)Institute for Biological Chemistry & Nutrition, University of Hohenheim, \nD-70593 Stuttgart, Germany. Wilfried.Andlauer@hevs.ch\n\nPhenolic compounds are secondary plant metabolites which have long been \nassociated with flavor and color characteristics of fruits and vegetables. These \nphenolic compounds attract great interest due to their postulated health \nprotecting properties. However, adequate intakes and absorption rate of phenolic \ncompounds are necessary for these beneficial effects. Until now, little is known \nabout alterations of phenolic compounds content by the cooking process. In the \npresent study, the influence of different volumes of cooking water on the amount \nof selected phenolic marker compounds resting in the vegetables was assessed. In \nzucchini, rutin was quantified as a marker for flavonoid glycosides. Chlorogenic \nacid, representative of phenolic acids was analyzed in carrots. In beans, rutin \nand quercitrin, both belonging to flavonoid glycosides, were investigated. In \npotatoes, chlorogenic and caffeic acid were determined. The cooking of zucchini, \nbeans and carrots with smaller amounts of water resulted in significant higher \ncontent of phenolic phytochemicals in the vegetables compared to cooking with \nlarger water volumes. For potatoes, which showed great variations in content of \nphenolic acids after cooking, no significant differences in phenolic acids was \nobserved. It can be concluded from these observations, that real intakes of \nphenolic compounds from cooked vegetables are lower and that the amounts \nconsumed are therefore overestimated.\n\nDOI: 10.1024/0300-9831.73.2.152\nPMID: 12747223 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "38",
         "Seven varietal port wine grapes (Tinta Roriz, Tinta Barroca, Tinta Amarela, Tinta Cao, Touriga Francesa, Touriga Nacional and Rufete) were analysed in order to determine differences in flavonol compositions. For this purpose, a SPE-HPLC method was applied. Although no qualitative variations were found, there are quantitative differences in the individual concentrations of the phenolic compounds. In Tinta Cao and Rufete varieties, (-)-epicatechin was the major compound. All samples showed significant amounts of kaempferol 3-glucoside and isorhamnetin 3-glucoside, with the exception of Rufete variety, where these compounds were present in traces. Copyright © 2001 Elsevier Science Ltd.",
         "1"
        ],
        [
         "39",
         "1. J Sci Food Agric. 2015 Mar 30;95(5):1103-15. doi: 10.1002/jsfa.7007. Epub 2014\n Dec 30.\n\nSoluble and cell wall-bound phenolic acids and ferulic acid dehydrodimers in rye \nflour and five bread model systems: insight into mechanisms of improved \navailability.\n\nDynkowska WM(1), Cyran MR, Ceglińska A.\n\nAuthor information:\n(1)Department of Plant Biochemistry and Physiology, Plant Breeding and \nAcclimatization Institute - National Research Institute, Radzikow, PL-05-870, \nBlonie, Poland.\n\nBACKGROUND: The bread-making process influences bread components, including \nphenolics that significantly contribute to its antioxidant properties. Five \nbread model systems made from different rye cultivars were investigated to \ncompare their impact on concentration of ethanol-soluble (free and ester-bound) \nand insoluble phenolics.\nRESULTS: Breads produced by a straight dough method without acid addition (A) \nand three-stage sourdough method with 12 h native starter preparation (C) \nexhibited the highest, genotype-dependent concentrations of free phenolic acids. \nDough acidification by direct acid addition (method B) or by gradual production \nduring prolonged starter fermentation (24 and 48 h, for methods D and E) \nconsiderably decreased their level. However, breads B were enriched in soluble \nester-bound fraction. Both direct methods, despite substantial differences in \ndough pH, caused a similar increase in the amount of insoluble ester-bound \nfraction. The contents of phenolic fractions in rye bread were positively \nrelated to activity level of feruloyl esterase and negatively to those of \narabinoxylan-hydrolysing enzymes in wholemeal flour.\nCONCLUSION: The solubility of rye bread phenolics may be enhanced by application \nof a suitable bread-making procedure with respect to rye cultivar, as the \nmechanisms of this process are also governed by a response of an individual \ngenotype with specific biochemical profile.\n\n© 2014 The Authors. Journal of the Science of Food and Agriculture published by \nJohn Wiley & Sons Ltd on behalf of Society of Chemical Industry.\n\nDOI: 10.1002/jsfa.7007\nPMCID: PMC4368412\nPMID: 25410263 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "40",
         "1. J Agric Food Chem. 2000 Jul;48(7):2837-42. doi: 10.1021/jf991266w.\n\nContent of phenolic acids and ferulic acid dehydrodimers in 17 rye (Secale \ncereale L.) varieties.\n\nAndreasen MF(1), Christensen LP, Meyer AS, Hansen A.\n\nAuthor information:\n(1)Department of Fruit, Vegetable and Food Science, Danish Institute of \nAgricultural Sciences, Kirstinebjergvej 10, DK-5792 Aarslev, Denmark. \nmette.andreasen@agrsci.dk\n\nThe contents of pnenolic acids and ferulic acid dehydrodimers were quantified by \nHPLC analysis after alkaline hydrolysis in kernels of 17 rye (Secale cereale L.) \nvarieties grown in one location in Denmark during 1997 and 1998. Significant \nvariations (P < 0.05) with regard to the concentration of the analyzed \ncomponents were observed among the different rye varieties and also between \ndifferent harvest years. However, the content of phenolic acids in the analyzed \nrye varieties was narrow compared to cereals such as wheat and barley. The \nconcentration of ferulic acid, the most abundant phenolic acid ranged from 900 \nto 1170 microgram g(-1) dry matter. The content in sinapic acid ranged from 70 \nto 140 microgram g(-1) dry matter, p-coumaric acid ranged from 40 to 70 \nmicrogram g(-1) dry matter, and caffeic, p-hydroxybenzoic, protocatechuic, and \nvanillic acids were all detected in concentrations less than 20 microgram g(-1) \ndry matter. The most abundant ferulic acid dehydrodimer 8-O-4 -DiFA was \nquantified in concentrations from 130 to 200 microgram g(-1) dry matter followed \nby 8,5 -DiFA benzofuran form (50-100 microgram g(-1) dry matter), 5,5 -DiFA \n(40-70 microgram g(-1) dry matter), and 8,5 -DiFA (20-40 microgram g(-1) dry \nmatter).\n\nDOI: 10.1021/jf991266w\nPMID: 11032481 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "41",
         "1. J Agric Food Chem. 2008 May 14;56(9):3111-7. doi: 10.1021/jf0728754. Epub 2008\n Apr 16.\n\nFlavanol and flavonol contents of cocoa powder products: influence of the \nmanufacturing process.\n\nAndres-Lacueva C(1), Monagas M, Khan N, Izquierdo-Pulido M, Urpi-Sarda M, \nPermanyer J, Lamuela-Raventós RM.\n\nAuthor information:\n(1)Departament de Nutrició i Bromatologia, XARTA, INSA Facultat de Farmàcia, \nUniversitat de Barcelona, Av. Joan XXIII s/n, Barcelona 08028, Spain.\n\nMajor brands of cocoa powder products present in the Spanish market were \nanalyzed for monomeric flavanols [(+)-catechin and (-)-epicatechin] and \nflavonols [quercetin-3-glucuronide, quercetin-3-glucoside (isoquercitrin), \nquercetin-3-arabinoside, and quercetin]. In addition, the influence of the \nmanufacturing process of cocoa powder products, in particular, the \nalkalinization treatment ( Dutching), on the original content of these \nflavonoids has been studied. (-)-Epicatechin was in the range of 116.02-730.26 \nmicrog/g, whereas (+)-catechin was in the range of 81.40-447.62 microg/g in the \ncommercial cocoa products studied. Among flavonols, quercetin-3-arabinoside and \nisoquercitrin were the major flavonols in the cocoa powder products studied, \nranging from 2.10 to 40.33 microg/g and from 3.97 to 42.74 microg/g, \nrespectively, followed by quercetin-3-glucuronide (0.13-9.88 microg/g) and \nquercetin aglycone (0.28-3.25 microg/g). To our knowledge, these results are the \nfirst quantitative data in relation to the content of individualized flavonol \nderivatives in commercial cocoa powder products. The alkalinization treatment \nresulted in 60% loss of the mean total flavonoid content. Among flavanols, \n(-)-epicatechin presented a larger decline (67%, as a mean percentage \ndifference) than (+)-catechin (38%), probably because of its epimerization into \n(-)-catechin, a less bioavailable form of catechin. A decline was also confirmed \nfor di-, tri-, and tetrameric procyanidins. In the case of flavonols, quercetin \npresented the highest loss (86%), whereas quercetin-3-glucuronide, \nquercetin-3-arabinoside, and isoquercitrin showed a similar decrease (58, 62, \nand 61%, respectively). It is concluded that the large decrease found in the \nflavonoid content of natural cocoa powder, together with the observed change in \nthe monomeric flavanol profile that results from the alkalinization treatment, \ncould affect the antioxidant properties and the polyphenol biovailability of \ncocoa powder products.\n\nDOI: 10.1021/jf0728754\nPMID: 18412367 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "42",
         "Commonly used spices (40) in cooking and therapeutic preparations in India were evaluated for antioxidant activity (AOA), total phenols and flavonoids in raw and cooked in open (5 min, 1:10, 100°C). Among these spices, AOA was highest for white mustard (91.6%) followed by black mustard (91.0%) while the total phenols (141 mg/g) and flavonoids (60 mg) were highest in clove. The results revealed a significant reduction in AOA, polyphenols and flavonoids contents on cooking of large number of spices.",
         "1"
        ],
        [
         "43",
         "Red raspberry (Rubus idaeus L.) is an economically important berry crop that contains numerous phenolic compounds with potential health benefits. It is known that the content of phenolics is affected by processing factors, but limited information is available on the influence of cultural factors or genotype. To clarify this issue, phenolic compounds were analysed from a diverse range of raspberry cultivars grown under northern European conditions, in Finland. The content of phenolic compounds varied widely and significantly between cultivars. The quercetin content ranged from 0.32 (yellow cultivar) to 1.55 mg/100 g fresh weight (fw) (cv. Balder). The ellagic acid content varied from 38 (cv. Gatineau and cv. Nova) to 118 mg/100 g fw (cv. Ville). The total anthocyanin content varied from close to 0 (yellow cultivars) to 51 mg/100 g fw (cv. Gatineau). The content of total phenolics varied from 192 (cv. Gatineau) to 359 mg/100 g fw (cv. Ville). In addition, environment had a significant effect on the content of quercetin. Thus, breeding material should be evaluated for their potential health benefits over several regions in northern raspberry breeding. © 2005 Elsevier Inc. All rights reserved.",
         "1"
        ],
        [
         "44",
         "1. J Agric Food Chem. 1999 Oct;47(10):4150-5. doi: 10.1021/jf9812230.\n\nEffect of various compounds on virgin olive oil stability measured by Rancimat.\n\nAparicio R(1), Roda L, Albi MA, Gutiérrez F.\n\nAuthor information:\n(1)Instituto de la Grasa, Avenida Padre García Tejero 4, E-41012 Sevilla, Spain. \naparicio@cica.es\n\nPhenolic and orthodiphenolic compounds together with carotenoids, tocopherols, \npigments, and fatty acids were tested for their antioxidant effect in 79 samples \nof virgin olive oils cv. Picual and Hojiblanca. A linear regression based on the \noleic/linoleic ratio and the contents of phenols and tocopherols showed a good \ncorrelation (adjusted-R(2) = 0.91) with the stability measured by Rancimat, \nlater verified with an external text set (adjusted-R(2) = 0.95). A tentative \nstudy on the percentage contribution of each chemical variable to stability is \ndiscussed. The contribution of phenolic and orthodiphenolic compounds was around \n51%, the composition of fatty acids 24%, and in less percentage \nalpha-tocopherol, carotenoids, and chlorophylls. No effect, or very little, was \nshown by beta- and gamma-tocopherols.\n\nDOI: 10.1021/jf9812230\nPMID: 10552782 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "45",
         "1. J Agric Food Chem. 2004 Mar 10;52(5):1124-31. doi: 10.1021/jf0499525.\n\nFlavonoids in vegetable foods commonly consumed in Brazil and estimated \ningestion by the Brazilian population.\n\nArabbi PR(1), Genovese MI, Lajolo FM.\n\nAuthor information:\n(1)Laboratório de Química, Bioquímica e Biologia Molecular de Alimentos, \nDepartamento de Alimentos e Nutrição Experimental, FCF, Universidade de São \nPaulo, Av. Prof. Lineu Prestes 580, Bloco 14, 05508-900 São Paulo, SP, Brazil.\n\nThe objective of this work was to quantify the flavonoids present in foods most \ncommonly consumed by the Brazilian population. The predominant flavonoids found \nin largest abundance in all of the analyzed vegetables were glycosides of \nquercetin. In lettuce, a small amount of luteolin was also detected. In sweet \npepper, quercetin and luteolin were both present. White onion [48-56 mg/100 g of \nfresh weight (FW), expressed as aglycon], red onion (40-100 mg/100 g of FW), red \nlettuce (67-67.2 mg/100 g of FW), arugula (41-118 mg/100 g of FW), and chicory \n(18-38 mg/100 g of FW) were highest in total flavonoids. In fruits, the highest \nconcentrations of flavonoids were found in the peel (125-170 mg/100 g of FW) and \npulp (35-44 mg/100 g of FW) of oranges and in some apple varieties (14-36 mg/100 \ng of FW). Variability in flavonoid content due to time of harvesting was high \nfor leafy vegetables and red onions. The estimated ingestion by Brazilian \npopulation ranged from 60 to 106 mg/day.\n\nDOI: 10.1021/jf0499525\nPMID: 14995109 [Indexed for MEDLINE]",
         "1"
        ],
        [
         "46",
         "A reversed-phase high-performance liquid chromatography procedure is proposed for the determination of 10 phenolic compounds (eriodictyol 7-O-rutinoside, eriodictyol 7-O-glucoside, luteolin 7-O-rutinoside, luteolin 7-O-glucoside, hesperetin 7-O-rutinoside, apigenin 7-O-rutinoside, rosmarinic acid, 5,6-dihydroxy-7,8,3′,4′-tetramethoxyflavone, pebrellin and gardenin B) in peppermint. The chromatographic separation was achieved using a reversed-phase Spherisorb ODS 2 (5 μm particle size; 25.0×0.46 cm) column. Of the several extractive solvents tried, ethanol was the best for qualitative and quantitative analysis. Best resolution was obtained using a gradient of water-phosphoric acid (999:1) and acetonitrile. Fourteen samples were subjected to quantification and showed a common composition pattern. Copyright © 2001 Elsevier Science Ltd.",
         "1"
        ],
        [
         "47",
         "Selected Greek regional, aged red wines were assessed for their polyphenolic composition using well-established spectrophotometric methodology, and tested for antioxidant potential by three different assays. The total polyphenol (TP) concentration was found to vary from 1217 to 3772 mg L-1 gallic acid equivalents, total anthocyanin (TA) content ranged from 44.3 to 360.1 mg L-1, and total flavanol (TF) from 291.1 to 664.8 mg L-1 catechin equivalents. The antiradical activity was evaluated using DPPH?O, and varied from 0.73 to 1.37mM Trolox® equivalents, while reducing power values were from 5.29 to 10.80mM ascorbic acid equivalents, using a modified FRAP assay. The hydroxyl free radical scavenging activity was determined with the deoxyribose method, and ranged from 38.5 to 59.9% decrease in A532 relative to control. The examinations showed that TP and TF may provide a significant contribution to the overall antioxidant status of wines, but TA appears to be a less important factor in this respect. Reducing power was found to be highly associated with the antiradical activity of wines. © 2002 Elsevier Science Ltd. All rights reserved.",
         "1"
        ],
        [
         "48",
         "The main objective of this research was to determine the extent to which irrigation practices affect the partitioning of phenolic compounds between olive paste, pomace, olive oil and wastewater. The current paper also aimed to study the effect of technological natural micro-talc (NMT) addition during the oil extraction process on the partitioning of the phenolic compounds between solid and liquid phases. The results obtained in this study showed that irrigation applied to olive trees let to a considerable decrease in the phenol content of the olive paste. The water status of the trees affected the phenol synthesis in the olive fruit, and consequently the phenol content of the olive paste, more than the partitioning of the phenolic compounds during the olive oil extraction process. The most remarkable point of the phenol partitioning was related to the simple phenols. While in the samples from non-irrigated trees the greater proportion of these phenols partitioned into the pomace, in samples from irrigated trees most of them were lost in the wastewater. After comparison of the results obtained from the experiments with and without NMT addition, it was concluded that the use of that coadjuvant did not significantly alter either the phenolic profile of the oil phase obtained or the content of the individual phenolic compounds. © 2006 WILEY-VCH Verlag GmbH & Co. KGaA.",
         "1"
        ],
        [
         "49",
         "The transfer of phenolic compounds of Olea europaea L. cv. Arbequina variety during olive oil extraction in relation to ripening stage was investigated. The parameters of oil extraction by the Abencor system are shown together with mass balances of the products and by products from the olive oil extraction in relation to olive paste. The phenolic compounds in olive paste, pomace, oil and wastewater were identified and measured by HPLC. Throughout the study, the concentrations of simple phenols, secoiridoids and flavonoids were higher in the olive paste and pomace phases than in oil and wastewater phases. High concentrations of 4-(acetoxyethyl)-1,2-dihydroxybenzene (3,4-DHPEA-AC) and secoiridoid derivatives such as the dialdehydic form of elenolic acid linked to 3,4-DHPEA (hydroxytyrosol) or p-HPEA (tyrosol) (3,4-DHPEA-EDA, p-HPEA-EDA, where EDA is elenolic acid dialdehyde) and an isomer of oleuropein aglycone (3,4-DHPEA-EA, where EA is elenolic acid aldehyde) were found in olive oil, together with lignan compounds. It was observed that 3,4-DHPEA-EDA was the most abundant polyphenol present in the wastewater phase. This indicates that biotransformation occurred during olive extraction, especially in the crushing and malaxation operations, and reflects the possible chemical changes that lead to the formation of new compounds. Moreover, the distribution of compounds showed their affinities toward different phases. © 2005 Society of Chemical Industry.",
         "1"
        ]
       ],
       "shape": {
        "columns": 2,
        "rows": 2405
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>abstract</th>\n",
       "      <th>relevance</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1. J Agric Food Chem. 2007 Jun 27;55(13):5156-...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1. Free Radic Biol Med. 2002 Dec 15;33(12):169...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1. J Agric Food Chem. 2003 Apr 9;51(8):2174-80...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1. J Agric Food Chem. 2006 Jun 28;54(13):4696-...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>A survey of resveratrol content in wines of th...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2400</th>\n",
       "      <td>1. J Am Coll Radiol. 2025 Nov;22(11S):S492-S50...</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2401</th>\n",
       "      <td>1. Am J Clin Nutr. 2025 Nov;122(5):1195-1203. ...</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2402</th>\n",
       "      <td>1. West J Emerg Med. 2025 Sep 25;26(5):1211-12...</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2403</th>\n",
       "      <td>1. J Gene Med. 2025 Nov;27(11):e70047. doi: 10...</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2404</th>\n",
       "      <td>1. BMJ Support Palliat Care. 2025 Nov 4:spcare...</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2405 rows × 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                               abstract  relevance\n",
       "0     1. J Agric Food Chem. 2007 Jun 27;55(13):5156-...          1\n",
       "1     1. Free Radic Biol Med. 2002 Dec 15;33(12):169...          1\n",
       "2     1. J Agric Food Chem. 2003 Apr 9;51(8):2174-80...          1\n",
       "3     1. J Agric Food Chem. 2006 Jun 28;54(13):4696-...          1\n",
       "4     A survey of resveratrol content in wines of th...          1\n",
       "...                                                 ...        ...\n",
       "2400  1. J Am Coll Radiol. 2025 Nov;22(11S):S492-S50...          0\n",
       "2401  1. Am J Clin Nutr. 2025 Nov;122(5):1195-1203. ...          0\n",
       "2402  1. West J Emerg Med. 2025 Sep 25;26(5):1211-12...          0\n",
       "2403  1. J Gene Med. 2025 Nov;27(11):e70047. doi: 10...          0\n",
       "2404  1. BMJ Support Palliat Care. 2025 Nov 4:spcare...          0\n",
       "\n",
       "[2405 rows x 2 columns]"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Adding target variable 'relevance' \n",
    "relevant_df['relevance'] = 1\n",
    "irrelevant_df['relevance'] = 0\n",
    "\n",
    "# Combining relevant and irrelevant datasets and maintaining only abstract and relevance columns\n",
    "features = ['abstract', 'relevance']\n",
    "combined_df = pd.concat([relevant_df[features], irrelevant_df[features]], ignore_index=True)\n",
    "\n",
    "# Remove any rows where the abstract is missing (e.g., API fetch failed)\n",
    "combined_df.dropna(subset=['abstract'], inplace=True)\n",
    "combined_df.reset_index(drop=True, inplace=True)\n",
    "\n",
    "# Saving\n",
    "combined_df.to_csv(COMBINED_PUBLICATIONS, index=False)\n",
    "\n",
    "print(\"Class distribution:\")\n",
    "print(combined_df['relevance'].value_counts())\n",
    "\n",
    "combined_df"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ec449e83",
   "metadata": {},
   "source": [
    "Following Fine-tuning of BERT for text classification tasks: https://huggingface.co/docs/transformers/en/tasks/sequence_classification"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "84b8bad6",
   "metadata": {},
   "source": [
    "- Train-Test-Validation Split: 80%-10%-10%"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "b988a960",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Training size: 1924\n",
      "Validation size: 240\n",
      "Test size: 241\n"
     ]
    }
   ],
   "source": [
    "RANDOM_STATE = 42\n",
    "\n",
    "train_df, test_df = train_test_split(combined_df,\n",
    "                                     test_size=0.2,\n",
    "                                     stratify=combined_df[\"relevance\"],\n",
    "                                     random_state=RANDOM_STATE)\n",
    "\n",
    "val_df, test_df = train_test_split(test_df,\n",
    "                                   test_size=0.5,\n",
    "                                   stratify=test_df[\"relevance\"],\n",
    "                                   random_state=RANDOM_STATE)\n",
    "\n",
    "print(f\"Training size: {len(train_df)}\")\n",
    "print(f\"Validation size: {len(val_df)}\")\n",
    "print(f\"Test size: {len(test_df)}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "05b0e10b",
   "metadata": {},
   "source": [
    "- Convert Pandas DataFrame to HuggingFace Dataset"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "f67eea05",
   "metadata": {},
   "outputs": [],
   "source": [
    "train_dataset = Dataset.from_pandas(train_df)\n",
    "val_dataset = Dataset.from_pandas(val_df)\n",
    "test_dataset = Dataset.from_pandas(test_df)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e430ee07",
   "metadata": {},
   "source": [
    "- Tokenization of abstracts using BioBERT tokenizer"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "bbf38265",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Map: 100%|██████████| 1924/1924 [00:00<00:00, 2420.84 examples/s]\n",
      "Map: 100%|██████████| 240/240 [00:00<00:00, 3332.99 examples/s]\n",
      "Map: 100%|██████████| 241/241 [00:00<00:00, 3349.68 examples/s]\n"
     ]
    }
   ],
   "source": [
    "BERT_MODEL_NAME = \"dmis-lab/biobert-v1.1\"\n",
    "tokenizer = AutoTokenizer.from_pretrained(BERT_MODEL_NAME)\n",
    "\n",
    "def tokenize(examples):\n",
    "    return tokenizer(examples[\"abstract\"], \n",
    "                     padding=\"max_length\", \n",
    "                     truncation=True,\n",
    "                     max_length=512 # Maximum length for BERT models\n",
    "                    )\n",
    "\n",
    "train_dataset = train_dataset.map(tokenize, batched=True)\n",
    "val_dataset = val_dataset.map(tokenize, batched=True)\n",
    "test_dataset = test_dataset.map(tokenize, batched=True)\n",
    "\n",
    "# Renaming the target column to 'labels' as expected by HuggingFace Trainer\n",
    "train_dataset = train_dataset.rename_column(\"relevance\", \"labels\")\n",
    "val_dataset = val_dataset.rename_column(\"relevance\", \"labels\")\n",
    "test_dataset = test_dataset.rename_column(\"relevance\", \"labels\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d4c95d8c",
   "metadata": {},
   "source": [
    "- Loading BioBERT model for binary text classification (relevant vs irrelevant)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "90a8a12b",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Some weights of BertForSequenceClassification were not initialized from the model checkpoint at dmis-lab/biobert-v1.1 and are newly initialized: ['classifier.bias', 'classifier.weight']\n",
      "You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.\n"
     ]
    }
   ],
   "source": [
    "id2label = {0: \"irrelevant\", 1: \"relevant\"}\n",
    "label2id = {\"irrelevant\": 0, \"relevant\": 1}\n",
    "\n",
    "model = AutoModelForSequenceClassification.from_pretrained(BERT_MODEL_NAME, \n",
    "                                                           num_labels=2,\n",
    "                                                           id2label=id2label,\n",
    "                                                           label2id=label2id)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "554adc57",
   "metadata": {},
   "source": [
    "- Defining evaluation metrics"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "fb597090",
   "metadata": {},
   "outputs": [],
   "source": [
    "def compute_metrics(eval_pred):\n",
    "    logits, labels = eval_pred\n",
    "    predictions = np.argmax(logits, axis=-1)\n",
    "\n",
    "    accuracy = accuracy_score(labels, predictions)\n",
    "    precision, recall, f1, _ = precision_recall_fscore_support(labels, \n",
    "                                                               predictions, \n",
    "                                                               average=\"binary\",\n",
    "                                                               zero_division=0)\n",
    "\n",
    "    return {\n",
    "        \"accuracy\": accuracy,\n",
    "        \"precision\": precision,\n",
    "        \"recall\": recall,\n",
    "        \"f1\": f1\n",
    "    }"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b701845f",
   "metadata": {},
   "source": [
    "- Putting the training arguments"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "62e7d046",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/didierrc/miniconda3/envs/biomed_ir/lib/python3.13/site-packages/transformers/training_args.py:1594: FutureWarning: `evaluation_strategy` is deprecated and will be removed in version 4.46 of 🤗 Transformers. Use `eval_strategy` instead\n",
      "  warnings.warn(\n"
     ]
    }
   ],
   "source": [
    "training_args = TrainingArguments(\n",
    "    output_dir=\"./biobert_pubmed_classifier\",\n",
    "\n",
    "    # Training hyperparameters\n",
    "    num_train_epochs=3,\n",
    "    learning_rate=2e-5,\n",
    "    per_device_train_batch_size=16,\n",
    "    per_device_eval_batch_size=16,\n",
    "\n",
    "    # Optimiser settings\n",
    "    weight_decay=0.01,\n",
    "    \n",
    "    # Evaluation settings\n",
    "    evaluation_strategy=\"epoch\",\n",
    "    save_strategy=\"epoch\",\n",
    "    logging_steps=100,\n",
    "\n",
    "    # Model selection    \n",
    "    load_best_model_at_end=True,\n",
    "    metric_for_best_model=\"f1\",\n",
    "    greater_is_better=True,\n",
    "\n",
    "    # Performance\n",
    "    fp16=torch.cuda.is_available(),\n",
    "    dataloader_num_workers=4,\n",
    "\n",
    "    seed=RANDOM_STATE,\n",
    "    push_to_hub=False,\n",
    "    report_to=\"none\"\n",
    ")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "35c4d0a6",
   "metadata": {},
   "source": [
    "- Actual training using Trainer API"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "744c7adc",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_5409/3549236244.py:1: FutureWarning: `tokenizer` is deprecated and will be removed in version 5.0.0 for `Trainer.__init__`. Use `processing_class` instead.\n",
      "  trainer = Trainer(\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "\n",
       "    <div>\n",
       "      \n",
       "      <progress value='11' max='363' style='width:300px; height:20px; vertical-align: middle;'></progress>\n",
       "      [ 11/363 00:23 < 15:33, 0.38 it/s, Epoch 0.08/3]\n",
       "    </div>\n",
       "    <table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       " <tr style=\"text-align: left;\">\n",
       "      <th>Epoch</th>\n",
       "      <th>Training Loss</th>\n",
       "      <th>Validation Loss</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "  </tbody>\n",
       "</table><p>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[31m---------------------------------------------------------------------------\u001b[39m",
      "\u001b[31mKeyboardInterrupt\u001b[39m                         Traceback (most recent call last)",
      "\u001b[36mCell\u001b[39m\u001b[36m \u001b[39m\u001b[32mIn[20]\u001b[39m\u001b[32m, line 11\u001b[39m\n\u001b[32m      1\u001b[39m trainer = Trainer(\n\u001b[32m      2\u001b[39m     model=model,\n\u001b[32m      3\u001b[39m     args=training_args,\n\u001b[32m   (...)\u001b[39m\u001b[32m      8\u001b[39m     callbacks=[EarlyStoppingCallback(early_stopping_patience=\u001b[32m2\u001b[39m)]\n\u001b[32m      9\u001b[39m )\n\u001b[32m---> \u001b[39m\u001b[32m11\u001b[39m \u001b[43mtrainer\u001b[49m\u001b[43m.\u001b[49m\u001b[43mtrain\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/miniconda3/envs/biomed_ir/lib/python3.13/site-packages/transformers/trainer.py:2241\u001b[39m, in \u001b[36mTrainer.train\u001b[39m\u001b[34m(self, resume_from_checkpoint, trial, ignore_keys_for_eval, **kwargs)\u001b[39m\n\u001b[32m   2239\u001b[39m         hf_hub_utils.enable_progress_bars()\n\u001b[32m   2240\u001b[39m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[32m-> \u001b[39m\u001b[32m2241\u001b[39m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43minner_training_loop\u001b[49m\u001b[43m(\u001b[49m\n\u001b[32m   2242\u001b[39m \u001b[43m        \u001b[49m\u001b[43margs\u001b[49m\u001b[43m=\u001b[49m\u001b[43margs\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m   2243\u001b[39m \u001b[43m        \u001b[49m\u001b[43mresume_from_checkpoint\u001b[49m\u001b[43m=\u001b[49m\u001b[43mresume_from_checkpoint\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m   2244\u001b[39m \u001b[43m        \u001b[49m\u001b[43mtrial\u001b[49m\u001b[43m=\u001b[49m\u001b[43mtrial\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m   2245\u001b[39m \u001b[43m        \u001b[49m\u001b[43mignore_keys_for_eval\u001b[49m\u001b[43m=\u001b[49m\u001b[43mignore_keys_for_eval\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m   2246\u001b[39m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/miniconda3/envs/biomed_ir/lib/python3.13/site-packages/transformers/trainer.py:2500\u001b[39m, in \u001b[36mTrainer._inner_training_loop\u001b[39m\u001b[34m(self, batch_size, args, resume_from_checkpoint, trial, ignore_keys_for_eval)\u001b[39m\n\u001b[32m   2498\u001b[39m update_step += \u001b[32m1\u001b[39m\n\u001b[32m   2499\u001b[39m num_batches = args.gradient_accumulation_steps \u001b[38;5;28;01mif\u001b[39;00m update_step != (total_updates - \u001b[32m1\u001b[39m) \u001b[38;5;28;01melse\u001b[39;00m remainder\n\u001b[32m-> \u001b[39m\u001b[32m2500\u001b[39m batch_samples, num_items_in_batch = \u001b[38;5;28;43mself\u001b[39;49m\u001b[43m.\u001b[49m\u001b[43mget_batch_samples\u001b[49m\u001b[43m(\u001b[49m\u001b[43mepoch_iterator\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mnum_batches\u001b[49m\u001b[43m)\u001b[49m\n\u001b[32m   2501\u001b[39m \u001b[38;5;28;01mfor\u001b[39;00m i, inputs \u001b[38;5;129;01min\u001b[39;00m \u001b[38;5;28menumerate\u001b[39m(batch_samples):\n\u001b[32m   2502\u001b[39m     step += \u001b[32m1\u001b[39m\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/miniconda3/envs/biomed_ir/lib/python3.13/site-packages/transformers/trainer.py:5180\u001b[39m, in \u001b[36mTrainer.get_batch_samples\u001b[39m\u001b[34m(self, epoch_iterator, num_batches)\u001b[39m\n\u001b[32m   5178\u001b[39m \u001b[38;5;28;01mfor\u001b[39;00m _ \u001b[38;5;129;01min\u001b[39;00m \u001b[38;5;28mrange\u001b[39m(num_batches):\n\u001b[32m   5179\u001b[39m     \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[32m-> \u001b[39m\u001b[32m5180\u001b[39m         batch_samples += [\u001b[38;5;28;43mnext\u001b[39;49m\u001b[43m(\u001b[49m\u001b[43mepoch_iterator\u001b[49m\u001b[43m)\u001b[49m]\n\u001b[32m   5181\u001b[39m     \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mStopIteration\u001b[39;00m:\n\u001b[32m   5182\u001b[39m         \u001b[38;5;28;01mbreak\u001b[39;00m\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/miniconda3/envs/biomed_ir/lib/python3.13/site-packages/accelerate/data_loader.py:577\u001b[39m, in \u001b[36mDataLoaderShard.__iter__\u001b[39m\u001b[34m(self)\u001b[39m\n\u001b[32m    574\u001b[39m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[32m    575\u001b[39m     \u001b[38;5;66;03m# But we still move it to the device so it is done before `StopIteration` is reached\u001b[39;00m\n\u001b[32m    576\u001b[39m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28mself\u001b[39m.device \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n\u001b[32m--> \u001b[39m\u001b[32m577\u001b[39m         current_batch = \u001b[43msend_to_device\u001b[49m\u001b[43m(\u001b[49m\u001b[43mcurrent_batch\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[43m.\u001b[49m\u001b[43mdevice\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mnon_blocking\u001b[49m\u001b[43m=\u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[43m.\u001b[49m\u001b[43m_non_blocking\u001b[49m\u001b[43m)\u001b[49m\n\u001b[32m    578\u001b[39m     \u001b[38;5;28mself\u001b[39m._update_state_dict()\n\u001b[32m    579\u001b[39m     next_batch = \u001b[38;5;28mnext\u001b[39m(dataloader_iter)\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/miniconda3/envs/biomed_ir/lib/python3.13/site-packages/accelerate/utils/operations.py:154\u001b[39m, in \u001b[36msend_to_device\u001b[39m\u001b[34m(tensor, device, non_blocking, skip_keys)\u001b[39m\n\u001b[32m    152\u001b[39m     device = \u001b[33m\"\u001b[39m\u001b[33mnpu:0\u001b[39m\u001b[33m\"\u001b[39m\n\u001b[32m    153\u001b[39m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[32m--> \u001b[39m\u001b[32m154\u001b[39m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43mtensor\u001b[49m\u001b[43m.\u001b[49m\u001b[43mto\u001b[49m\u001b[43m(\u001b[49m\u001b[43mdevice\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mnon_blocking\u001b[49m\u001b[43m=\u001b[49m\u001b[43mnon_blocking\u001b[49m\u001b[43m)\u001b[49m\n\u001b[32m    155\u001b[39m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mTypeError\u001b[39;00m:  \u001b[38;5;66;03m# .to() doesn't accept non_blocking as kwarg\u001b[39;00m\n\u001b[32m    156\u001b[39m     \u001b[38;5;28;01mreturn\u001b[39;00m tensor.to(device)\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/miniconda3/envs/biomed_ir/lib/python3.13/site-packages/transformers/tokenization_utils_base.py:821\u001b[39m, in \u001b[36mBatchEncoding.to\u001b[39m\u001b[34m(self, device, non_blocking)\u001b[39m\n\u001b[32m    816\u001b[39m \u001b[38;5;66;03m# This check catches things like APEX blindly calling \"to\" on all inputs to a module\u001b[39;00m\n\u001b[32m    817\u001b[39m \u001b[38;5;66;03m# Otherwise it passes the casts down and casts the LongTensor containing the token idxs\u001b[39;00m\n\u001b[32m    818\u001b[39m \u001b[38;5;66;03m# into a HalfTensor\u001b[39;00m\n\u001b[32m    819\u001b[39m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(device, \u001b[38;5;28mstr\u001b[39m) \u001b[38;5;129;01mor\u001b[39;00m is_torch_device(device) \u001b[38;5;129;01mor\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(device, \u001b[38;5;28mint\u001b[39m):\n\u001b[32m    820\u001b[39m     \u001b[38;5;28mself\u001b[39m.data = {\n\u001b[32m--> \u001b[39m\u001b[32m821\u001b[39m         k: \u001b[43mv\u001b[49m\u001b[43m.\u001b[49m\u001b[43mto\u001b[49m\u001b[43m(\u001b[49m\u001b[43mdevice\u001b[49m\u001b[43m=\u001b[49m\u001b[43mdevice\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mnon_blocking\u001b[49m\u001b[43m=\u001b[49m\u001b[43mnon_blocking\u001b[49m\u001b[43m)\u001b[49m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(v, torch.Tensor) \u001b[38;5;28;01melse\u001b[39;00m v\n\u001b[32m    822\u001b[39m         \u001b[38;5;28;01mfor\u001b[39;00m k, v \u001b[38;5;129;01min\u001b[39;00m \u001b[38;5;28mself\u001b[39m.data.items()\n\u001b[32m    823\u001b[39m     }\n\u001b[32m    824\u001b[39m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[32m    825\u001b[39m     logger.warning(\u001b[33mf\u001b[39m\u001b[33m\"\u001b[39m\u001b[33mAttempting to cast a BatchEncoding to type \u001b[39m\u001b[38;5;132;01m{\u001b[39;00m\u001b[38;5;28mstr\u001b[39m(device)\u001b[38;5;132;01m}\u001b[39;00m\u001b[33m. This is not supported.\u001b[39m\u001b[33m\"\u001b[39m)\n",
      "\u001b[31mKeyboardInterrupt\u001b[39m: "
     ]
    }
   ],
   "source": [
    "trainer = Trainer(\n",
    "    model=model,\n",
    "    args=training_args,\n",
    "    train_dataset=train_dataset,\n",
    "    eval_dataset=val_dataset,\n",
    "    tokenizer=tokenizer,\n",
    "    compute_metrics=compute_metrics,\n",
    "    callbacks=[EarlyStoppingCallback(early_stopping_patience=2)]\n",
    ")\n",
    "\n",
    "trainer.train()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d7795ee5",
   "metadata": {},
   "source": [
    "- Evaluating on the test set"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "05ba41f9",
   "metadata": {},
   "outputs": [],
   "source": [
    "predictions_output = trainer.predict(test_dataset)\n",
    "predictions = np.argmax(predictions_output.predictions, axis=-1)\n",
    "true_labels = predictions_output.label_ids\n",
    "\n",
    "# Calculate all metrics\n",
    "test_metrics = compute_metrics((predictions_output.predictions, true_labels))\n",
    "\n",
    "print(\"\\nTest Set Results:\")\n",
    "print(f\"Accuracy:  {test_metrics['accuracy']:.4f}\")\n",
    "print(f\"Precision: {test_metrics['precision']:.4f}\")\n",
    "print(f\"Recall:    {test_metrics['recall']:.4f}\")\n",
    "print(f\"F1-Score:  {test_metrics['f1']:.4f}\")\n",
    "\n",
    "# Detailed classification report\n",
    "print(\"\\nClassification Report:\")\n",
    "print(classification_report(\n",
    "    true_labels, \n",
    "    predictions,\n",
    "    target_names=['Irrelevant', 'Relevant'],\n",
    "    digits=4\n",
    "))\n",
    "\n",
    "# Confusion matrix\n",
    "print(\"\\nConfusion Matrix:\")\n",
    "cm = confusion_matrix(true_labels, predictions)\n",
    "print(cm)\n",
    "print(f\"\\nTrue Negatives:  {cm[0][0]} (correctly identified irrelevant)\")\n",
    "print(f\"False Positives: {cm[0][1]} (incorrectly marked relevant)\")\n",
    "print(f\"False Negatives: {cm[1][0]} (missed relevant papers)\")\n",
    "print(f\"True Positives:  {cm[1][1]} (correctly identified relevant)\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d50be2fb",
   "metadata": {},
   "source": [
    "- saving the trained model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4858b6d4",
   "metadata": {},
   "outputs": [],
   "source": [
    "model_save_path = './final_biobert_classifier'\n",
    "trainer.save_model(model_save_path)\n",
    "tokenizer.save_pretrained(model_save_path)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "biomed_ir",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.13.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
